<H1>Chapter DOI: 10.1186/s13073-016-0320-1<br/>Cited-By Count: 0</H1><table border="1" width="30%"><tr><td>Total References</td><td>90</td></tr><tr><td>Springer references</td><td>4</td></tr><tr><td>Non Springer references</td><td>86</td></tr><tr><td>BibStructured Count</td><td width="10%">90</td></tr><tr><td>BibUnstructured Count</td><td width="10%">0</td></tr><tr><td>DOI already available in SpringerLink</td><td>89</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>1</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>0</td></tr><tr><td>DOI Obtained For Non Springer</td><td>1</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>1</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>0</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibArticle</td><td>Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, et al. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res. 2014;42(1):D1001&#8211;6.</td><td><a href=http://dx.doi.org/10.1093/nar/gkt1229>10.1093/nar/gkt1229</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Paisan-Ruiz C, Jain S, Evans EW, Gilks WP, Simon J, van der Brug M, et al. Cloning of the gene containing mutations that cause PARK8-linked Parkinson&#8217;s disease. Neuron. 2004;44(4):595&#8211;600.</td><td><a href=http://dx.doi.org/10.1016/j.neuron.2004.10.023>10.1016/j.neuron.2004.10.023</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibArticle</td><td>Polymeropoulos MH, Lavedan C, Leroy E, Ide SE, Dehejia A, Dutra A, Pike B. Mutation in the alpha-synuclein gene identified in families with Parkinson&#8217;s disease. Science. 1997;276(5321):2045&#8211;7.</td><td><a href=http://dx.doi.org/10.1126/science.276.5321.2045>10.1126/science.276.5321.2045</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Simon-Sanchez J, Schulte C, Bras JM, Sharma M, Gibbs JR, Berg D, et al. Genome-wide association study reveals genetic risk underlying Parkinson&#8217;s disease. Nat Genet. 2009;41(12):1308&#8211;12.</td><td><a href=http://dx.doi.org/10.1038/ng.487>10.1038/ng.487</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR5</td><td>BibArticle</td><td>Freedman ML, Monteiro AN, Gayther SA, Coetzee GA, Risch A, Plass C, et al. Principles for the post-GWAS functional characterization of cancer risk loci. Nat Genet. 2011;43(6):513&#8211;8.</td><td><a href=http://dx.doi.org/10.1038/ng.840>10.1038/ng.840</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR6</td><td>BibArticle</td><td>FANTOM Consortium and the RIKEN PMI and CLST (DGT). A promoter-level mammalian expression atlas. Nature. 2014;507(7493):462&#8211;70.</td><td><a href=http://dx.doi.org/10.1038/nature13182>10.1038/nature13182</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR7</td><td>BibArticle</td><td>ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489(7414):57&#8211;74.</td><td><a href=http://dx.doi.org/10.1038/nature11247>10.1038/nature11247</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Bryois J, Buil A, Evans DM, Kemp JP, Montgomery SB, Conrad DF, et al. Cis and trans effects of human genomic variants on gene expression. PLoS Genet. 2014;10(7):e1004461.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1004461>10.1371/journal.pgen.1004461</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Ramasamy A, Trabzuni D, Guelfi S, Varghese V, Smith C, Walker R, et al. Genetic variability in the regulation of gene expression in ten regions of the human brain. Nat Neurosci. 2014;17(10):1418&#8211;28.</td><td><a href=http://dx.doi.org/10.1038/nn.3801>10.1038/nn.3801</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR10</td><td>BibArticle</td><td>GTEx Consortium. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science. 2015;348(6235):648&#8211;60.</td><td><a href=http://dx.doi.org/10.1126/science.1262110>10.1126/science.1262110</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR11</td><td>BibArticle</td><td>Andersson R, Gebhard C, Miguel-Escalada I, Hoof I, Bornholdt J, Boyd M, et al. An atlas of active enhancers across human cell types and tissues. Nature. 2014;507(7493):455&#8211;61.</td><td><a href=http://dx.doi.org/10.1038/nature12787>10.1038/nature12787</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR12</td><td>BibArticle</td><td>Gibbs JR, van der Brug MP, Hernandez DG, Traynor BJ, Nalls MA, Lai SL, et al. Abundant quantitative trait loci exist for DNA methylation and gene expression in human brain. PLoS Genet. 2010;6(5):e1000952.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1000952>10.1371/journal.pgen.1000952</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Heinzen EL, Ge D, Cronin KD, Maia JM, Shianna KV, Gabriel WN, et al. Tissue-specific genetic control of splicing: implications for the study of complex traits. PLoS Biol. 2008;6(12):e1.</td><td><a href=http://dx.doi.org/10.1371/journal.pbio.1000001>10.1371/journal.pbio.1000001</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Myers AJ, Gibbs JR, Webster JA, Rohrer K, Zhao A, Marlowe L, et al. A survey of genetic human cortical gene expression. Nat Genet. 2007;39(12):1494&#8211;9.</td><td><a href=http://dx.doi.org/10.1038/ng.2007.16>10.1038/ng.2007.16</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Pardo LM, Rizzu P, Francescatto M, Vitezic M, Leday GG, Sanchez JS, et al. Regional differences in gene expression and promoter usage in aged human brains. Neurobiol Aging. 2013;34(7):1825&#8211;36.</td><td><a href=http://dx.doi.org/10.1016/j.neurobiolaging.2013.01.005>10.1016/j.neurobiolaging.2013.01.005</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Alvarez JA, Emory E. Executive function and the frontal lobes: a meta-analytic review. Neuropsychol Rev. 2006;16(1):17&#8211;42.</td><td><a href=http://dx.doi.org/10.1007/s11065-006-9002-x>10.1007/s11065-006-9002-x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Shiraki T, Kondo S, Katayama S, Waki K, Kasukawa T, Kawaji H, et al. Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage. Proc Natl Acad Sci U S A. 2003;100(26):15776&#8211;81.</td><td><a href=http://dx.doi.org/10.1073/pnas.2136655100>10.1073/pnas.2136655100</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Kodzius R, Kojima M, Nishiyori H, Nakamura M, Fukuda S, Tagami M, et al. CAGE: cap analysis of gene expression. Nat Methods. 2006;3(3):211&#8211;22.</td><td><a href=http://dx.doi.org/10.1038/nmeth0306-211>10.1038/nmeth0306-211</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Takahashi H, Lassmann T, Murata M, Carninci P. 5&#8242; end-centered expression profiling using cap-analysis gene expression and next-generation sequencing. Nat Protoc. 2012;7(3):542&#8211;61.</td><td><a href=http://dx.doi.org/10.1038/nprot.2012.005>10.1038/nprot.2012.005</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559&#8211;75.</td><td><a href=http://dx.doi.org/10.1086/519795>10.1086/519795</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR21</td><td>BibArticle</td><td>International HapMap Consortium. A haplotype map of the human genome. Nature. 2005;437(7063):1299&#8211;320.</td><td><a href=http://dx.doi.org/10.1038/nature04226>10.1038/nature04226</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR22</td><td>BibArticle</td><td>Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. Genet Epidemiol. 2010;34(8):816&#8211;34.</td><td><a href=http://dx.doi.org/10.1002/gepi.20533>10.1002/gepi.20533</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR23</td><td>BibArticle</td><td>Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet. 2012;44(8):955&#8211;9.</td><td><a href=http://dx.doi.org/10.1038/ng.2354>10.1038/ng.2354</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR24</td><td>BibArticle</td><td>1000 Genomes Project Consortium, Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, et al. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491(7422):56&#8211;65.</td><td><a href=http://dx.doi.org/10.1038/nature11632>10.1038/nature11632</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR25</td><td>BibArticle</td><td>Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009;25(14):1754&#8211;60.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btp324>10.1093/bioinformatics/btp324</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR26</td><td>BibArticle</td><td>DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, Hartl C, et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet. 2011;43(5):491&#8211;8.</td><td><a href=http://dx.doi.org/10.1038/ng.806>10.1038/ng.806</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR27</td><td>BibArticle</td><td>McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 2010;20(9):1297&#8211;303.</td><td><a href=http://dx.doi.org/10.1101/gr.107524.110>10.1101/gr.107524.110</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR28</td><td>BibArticle</td><td>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009;25(16):2078&#8211;9.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btp352>10.1093/bioinformatics/btp352</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR29</td><td>BibArticle</td><td>Koboldt DC, Zhang Q, Larson DE, Shen D, McLellan MD, Lin L, et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. 2012;22(3):568&#8211;76.</td><td><a href=http://dx.doi.org/10.1101/gr.129684.111>10.1101/gr.129684.111</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR30</td><td>BibArticle</td><td>Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38(16):e164.</td><td><a href=http://dx.doi.org/10.1093/nar/gkq603>10.1093/nar/gkq603</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR31</td><td>BibArticle</td><td>Lassmann T, Hayashizaki Y, Daub CO. TagDust--a program to eliminate artifacts from next generation sequencing data. Bioinformatics. 2009;25(21):2839&#8211;40.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btp527>10.1093/bioinformatics/btp527</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR32</td><td>BibArticle</td><td>Harbers M, Kato S, de Hoon M, Hayashizaki Y, Carninci P, Plessy C. Comparison of RNA- or LNA-hybrid oligonucleotides in template-switching reactions for high-speed sequencing library preparation. BMC Genomics. 2013;14:665.</td><td><a href=http://dx.doi.org/10.1186/1471-2164-14-665>10.1186/1471-2164-14-665</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR33</td><td>BibArticle</td><td>Harrow J, Frankish A, Gonzalez JM, Tapanari E, Diekhans M, Kokocinski F, et al. GENCODE: the reference human genome annotation for The ENCODE Project. Genome Res. 2012;22(9):1760&#8211;74.</td><td><a href=http://dx.doi.org/10.1101/gr.135350.111>10.1101/gr.135350.111</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR34</td><td>BibArticle</td><td>Severin J, Lizio M, Harshbarger J, Kawaji H, Daub CO, Hayashizaki Y, et al. Interactive visualization and analysis of large-scale sequencing datasets using ZENBU. Nat Biotechnol. 2014;32(3):217&#8211;9.</td><td><a href=http://dx.doi.org/10.1038/nbt.2840>10.1038/nbt.2840</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR35</td><td>BibArticle</td><td>Shabalin AA. Matrix eQTL: ultra fast eQTL analysis via large matrix operations. Bioinformatics. 2012;28(10):1353&#8211;8.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/bts163>10.1093/bioinformatics/bts163</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR36</td><td>BibArticle</td><td>Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22(9):1790&#8211;7.</td><td><a href=http://dx.doi.org/10.1101/gr.137323.112>10.1101/gr.137323.112</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR37</td><td>BibArticle</td><td>Faulkner GJ, Kimura Y, Daub CO, Wani S, Plessy C, Irvine KM, et al. The regulated retrotransposon transcriptome of mammalian cells. Nat Genet. 2009;41(5):563&#8211;71.</td><td><a href=http://dx.doi.org/10.1038/ng.368>10.1038/ng.368</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR38</td><td>BibArticle</td><td>Shulha HP, Cheung I, Akbarian S, Weng Z. Coordinated cell type-specific epigenetic remodeling in prefrontal cortex begins before birth and continues into early adulthood. PLoS Genet. 2013;9(4):e1003433.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1003433>10.1371/journal.pgen.1003433</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR39</td><td>BibArticle</td><td>Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 2008;9(9):R137.</td><td><a href=http://dx.doi.org/10.1186/gb-2008-9-9-r137>10.1186/gb-2008-9-9-r137</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR40</td><td>BibArticle</td><td>Iyer MK, Niknafs YS, Malik R, Singhal U, Sahu A, Hosono Y, et al. The landscape of long noncoding RNAs in the human transcriptome. Nat Genet. 2015;47(3):199&#8211;208.</td><td><a href=http://dx.doi.org/10.1038/ng.3192>10.1038/ng.3192</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR41</td><td>BibArticle</td><td>Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics. 2010;26(6):841&#8211;2.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btq033>10.1093/bioinformatics/btq033</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR42</td><td>BibArticle</td><td>Zeller T, Wild P, Szymczak S, Rotival M, Schillert A, Castagne R, et al. Genetics and beyond--the transcriptome of human monocytes and disease susceptibility. PLoS One. 2010;5(5):e10693.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0010693>10.1371/journal.pone.0010693</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR43</td><td>BibArticle</td><td>McKenzie M, Henders AK, Caracella A, Wray NR, Powell JE. Overlap of expression quantitative trait loci (eQTL) in human brain and blood. BMC Med Genomics. 2014;7:31.</td><td><a href=http://dx.doi.org/10.1186/1755-8794-7-31>10.1186/1755-8794-7-31</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR44</td><td>BibArticle</td><td>Battle A, Mostafavi S, Zhu X, Potash JB, Weissman MM, McCormick C, et al. Characterizing the genetic basis of transcriptome diversity through RNA-sequencing of 922 individuals. Genome Res. 2014;24(1):14&#8211;24.</td><td><a href=http://dx.doi.org/10.1101/gr.155192.113>10.1101/gr.155192.113</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR45</td><td>BibArticle</td><td>Qureshi IA, Mehler MF. Emerging roles of non-coding RNAs in brain evolution, development, plasticity and disease. Nat Rev Neurosci. 2012;13(8):528&#8211;41.</td><td><a href=http://dx.doi.org/10.1038/nrn3234>10.1038/nrn3234</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR46</td><td>BibArticle</td><td>Rinn JL, Kertesz M, Wang JK, Squazzo SL, Xu X, Brugmann SA, et al. Functional demarcation of active and silent chromatin domains in human HOX loci by noncoding RNAs. Cell. 2007;129(7):1311&#8211;23.</td><td><a href=http://dx.doi.org/10.1016/j.cell.2007.05.022>10.1016/j.cell.2007.05.022</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR47</td><td>BibArticle</td><td>Wutz A. Gene silencing in X-chromosome inactivation: advances in understanding facultative heterochromatin formation. Nat Rev Genet. 2011;12(8):542&#8211;53.</td><td><a href=http://dx.doi.org/10.1038/nrg3035>10.1038/nrg3035</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR48</td><td>BibArticle</td><td>Carrieri C, Cimatti L, Biagioli M, Beugnet A, Zucchelli S, Fedele S, et al. Long non-coding antisense RNA controls Uchl1 translation through an embedded SINEB2 repeat. Nature. 2012;491(7424):454&#8211;7.</td><td><a href=http://dx.doi.org/10.1038/nature11508>10.1038/nature11508</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR49</td><td>BibArticle</td><td>Francescatto M, Vitezic M, Heutink P, Saxena A. Brain-specific noncoding RNAs are likely to originate in repeats and may play a role in up-regulating genes in cis. Int J Biochem Cell Biol. 2014;54:331&#8211;7.</td><td><a href=http://dx.doi.org/10.1016/j.biocel.2014.06.014>10.1016/j.biocel.2014.06.014</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR50</td><td>BibArticle</td><td>Mercer TR, Dinger ME, Sunkin SM, Mehler MF, Mattick JS. Specific expression of long noncoding RNAs in the mouse brain. Proc Natl Acad Sci U S A. 2008;105(2):716&#8211;21.</td><td><a href=http://dx.doi.org/10.1073/pnas.0706729105>10.1073/pnas.0706729105</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR51</td><td>BibArticle</td><td>Tanaka K. Yeast osmosensors Hkr1 and Msb2 activate the Hog1 MAPK cascade by different mechanisms. Sci Signal. 2014;7(314):ra21.</td><td><a href=http://dx.doi.org/10.1126/scisignal.2004780>10.1126/scisignal.2004780</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR52</td><td>BibArticle</td><td>Cao L, Wang Z, Zhu C, Zhao Y, Yuan W, Li J, et al. ZNF383, a novel KRAB-containing zinc finger protein, suppresses MAPK signaling pathway. Biochem Biophys Res Commun. 2005;333(4):1050&#8211;9.</td><td><a href=http://dx.doi.org/10.1016/j.bbrc.2005.05.193>10.1016/j.bbrc.2005.05.193</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR53</td><td>BibArticle</td><td>Hamshere ML, Bennett P, Williams N, Segurado R, Cardno A, Norton N, et al. Genomewide linkage scan in schizoaffective disorder: significant evidence for linkage at 1q42 close to DISC1, and suggestive evidence at 22q11 and 19p13. Arch Gen Psychiatry. 2005;62(10):1081&#8211;8.</td><td><a href=http://dx.doi.org/10.1001/archpsyc.62.10.1081>10.1001/archpsyc.62.10.1081</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR54</td><td>BibArticle</td><td>Funk AJ, McCullumsmith RE, Haroutunian V, Meador-Woodruff JH. Abnormal activity of the MAPK- and cAMP-associated signaling pathways in frontal cortical areas in postmortem brain in schizophrenia. Neuropsychopharmacology. 2012;37(4):896&#8211;905.</td><td><a href=http://dx.doi.org/10.1038/npp.2011.267>10.1038/npp.2011.267</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR55</td><td>BibArticle</td><td>Popadin K, Gutierrez-Arcelus M, Dermitzakis ET, Antonarakis SE. Genetic and epigenetic regulation of human lincRNA gene expression. Am J Hum Genet. 2013;93(6):1015&#8211;26.</td><td><a href=http://dx.doi.org/10.1016/j.ajhg.2013.10.022>10.1016/j.ajhg.2013.10.022</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR56</td><td>BibArticle</td><td>Nicolae DL, Gamazon E, Zhang W, Duan S, Dolan ME, Cox NJ. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. PLoS Genet. 2010;6(4):e1000888.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1000888>10.1371/journal.pgen.1000888</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR57</td><td>BibArticle</td><td>Lisak DA, Schacht T, Enders V, Habicht J, Kiviluoto S, Schneider J, et al. The transmembrane Bax inhibitor motif (TMBIM) containing protein family: Tissue expression, intracellular localization and effects on the ER CA(2+)-filling state. Biochim Biophys Acta. 2015;1853(9):2104&#8211;14.</td><td><a href=http://dx.doi.org/10.1016/j.bbamcr.2015.03.002>10.1016/j.bbamcr.2015.03.002</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR58</td><td>BibArticle</td><td>Shukla A, Tapadia MG. Differential localization and processing of apoptotic proteins in Malpighian tubules of Drosophila during metamorphosis. Eur J Cell Biol. 2011;90(1):72&#8211;80.</td><td><a href=http://dx.doi.org/10.1016/j.ejcb.2010.08.010>10.1016/j.ejcb.2010.08.010</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR59</td><td>BibArticle</td><td>Rojas-Rivera D, Hetz C. TMBIM protein family: ancestral regulators of cell death. Oncogene. 2015;34(3):269&#8211;80.</td><td><a href=http://dx.doi.org/10.1038/onc.2014.6>10.1038/onc.2014.6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR60</td><td>BibArticle</td><td>Qiu W, Wu B, Wang X, Buchanan ME, Requeiro MD, Hartman DJ, et al. PUMA-mediated intestinal epithelial apoptosis contributes to ulcerative colitis in humans and mice. J Clin Invest. 2011;121(5):1722&#8211;32.</td><td><a href=http://dx.doi.org/10.1172/JCI42917>10.1172/JCI42917</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR61</td><td>BibArticle</td><td>Verhaaren BF, de Boer R, Vernooij MW, Rivadeneira F, Uitterlinden AG, Hofman A, et al. Replication study of chr17q25 with cerebral white matter lesion volume. Stroke. 2011;42(11):3297&#8211;9.</td><td><a href=http://dx.doi.org/10.1161/STROKEAHA.111.623090>10.1161/STROKEAHA.111.623090</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR62</td><td>BibArticle</td><td>Tabara Y, Igase M, Okada Y, Nagai T, Uetani E, Kido T, et al. Association of Chr17q25 with cerebral white matter hyperintensities and cognitive impairment: the J-SHIPP study. Eur J Neurol. 2013;20(5):860&#8211;2.</td><td><a href=http://dx.doi.org/10.1111/j.1468-1331.2012.03879.x>10.1111/j.1468-1331.2012.03879.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR63</td><td>BibArticle</td><td>Fornage M, Debette S, Bis JC, Schmidt H, Ikram MA, Dufouil C, et al. Genome-wide association studies of cerebral white matter lesion burden: the CHARGE consortium. Ann Neurol. 2011;69(6):928&#8211;39.</td><td><a href=http://dx.doi.org/10.1002/ana.22403>10.1002/ana.22403</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR64</td><td>BibArticle</td><td>Wong JJ, Ritchie W, Ebner OA, Selbach M, Wong JW, Huang Y, et al. Orchestrated intron retention regulates normal granulocyte differentiation. Cell. 2013;154(3):583&#8211;95.</td><td><a href=http://dx.doi.org/10.1016/j.cell.2013.06.052>10.1016/j.cell.2013.06.052</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR65</td><td>BibArticle</td><td>Vandrovcova J, Anaya F, Kay V, Lees A, Hardy J, de Silva R. Disentangling the role of the tau gene locus in sporadic tauopathies. Curr Alzheimer Res. 2010;7(8):726&#8211;34.</td><td><a href=http://dx.doi.org/10.2174/156720510793611619>10.2174/156720510793611619</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR66</td><td>BibArticle</td><td>Jun G, Ibrahim-Verbaas CA, Vronskaya M, Lambert JC, Chung J, Naj AC, et al. A novel Alzheimer disease locus located near the gene encoding tau protein. Mol Psychiatry. 2016;21(1):108&#8211;17.</td><td><a href=http://dx.doi.org/10.1038/mp.2015.23>10.1038/mp.2015.23</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR67</td><td>BibArticle</td><td>Koolen DA, Kramer JM, Neveling K, Nillesen WM, Moore-Barton HL, Elmslie FV, et al. Mutations in the chromatin modifier gene KANSL1 cause the 17q21.31 microdeletion syndrome. Nat Genet. 2012;44(6):639&#8211;41.</td><td><a href=http://dx.doi.org/10.1038/ng.2262>10.1038/ng.2262</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR68</td><td>BibArticle</td><td>Zollino M, Orteschi D, Murdolo M, Lattante S, Battaglia D, Stefanini C, et al. Mutations in KANSL1 cause the 17q21.31 microdeletion syndrome phenotype. Nat Genet. 2012;44(6):636&#8211;8.</td><td><a href=http://dx.doi.org/10.1038/ng.2257>10.1038/ng.2257</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR69</td><td>BibArticle</td><td>Steinberg KM, Antonacci F, Sudmant PH, Kidd JM, Campbell CD, Vives L, et al. Structural diversity and African origin of the 17q21.31 inversion polymorphism. Nat Genet. 2012;44(8):872&#8211;80.</td><td><a href=http://dx.doi.org/10.1038/ng.2335>10.1038/ng.2335</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR70</td><td>BibArticle</td><td>Boettger LM, Handsaker RE, Zody MC, McCarroll SA. Structural haplotypes and recent evolution of the human 17q21.31 region. Nat Genet. 2012;44(8):881&#8211;5.</td><td><a href=http://dx.doi.org/10.1038/ng.2334>10.1038/ng.2334</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR71</td><td>BibArticle</td><td>Myers AJ, Pittman AM, Zhao AS, Rohrer K, Kaleem M, Marlowe L, et al. The MAPT H1c risk haplotype is associated with increased expression of tau and especially of 4 repeat containing transcripts. Neurobiol Dis. 2007;25(3):561&#8211;70.</td><td><a href=http://dx.doi.org/10.1016/j.nbd.2006.10.018>10.1016/j.nbd.2006.10.018</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR72</td><td>BibArticle</td><td>International Parkinson Disease Genomics Consortium, Nalls MA, Plagnol V, Hernandez DG, Sharma M, Sheerin UM. Imputation of sequence variants for identification of genetic risks for Parkinson&#8217;s disease: a meta-analysis of genome-wide association studies. Lancet. 2011;377(9766):641&#8211;9.</td><td><a href=http://dx.doi.org/10.1016/S0140-6736(10)62345-8>10.1016/S0140-6736(10)62345-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR73</td><td>BibArticle</td><td>Hayesmoore JB, Bray NJ, Cross WC, Owen MJ, O&#8217;Donovan MC, Morris HR. The effect of age and the H1c MAPT haplotype on MAPT expression in human brain. Neurobiol Aging. 2009;30(10):1652&#8211;6.</td><td><a href=http://dx.doi.org/10.1016/j.neurobiolaging.2007.12.017>10.1016/j.neurobiolaging.2007.12.017</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR74</td><td>BibArticle</td><td>Trabzuni D, Wray S, Vandrovcova J, Ramasamy A, Walker R, Smith C, et al. MAPT expression and splicing is differentially regulated by brain region: relation to genotype and implication for tauopathies. Hum Mol Genet. 2012;21(18):4094&#8211;103.</td><td><a href=http://dx.doi.org/10.1093/hmg/dds238>10.1093/hmg/dds238</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR75</td><td>BibArticle</td><td>Ramasamy A, Trabzuni D, Gibbs JR, Dillman A, Hernandez DG, Arepalli S, et al. Resolving the polymorphism-in-probe problem is critical for correct interpretation of expression QTL studies. Nucleic Acids Res. 2013;41(7):e88.</td><td><a href=http://dx.doi.org/10.1093/nar/gkt069>10.1093/nar/gkt069</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR76</td><td>BibArticle</td><td>Latourelle JC, Dumutriu A, Hadzi TC, Beach TG, Myers RH. Evaluation of Parkinson disease risk variants as expression-QTLs. PLoS One. 2012;7(10):e46199.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0046199>10.1371/journal.pone.0046199</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR77</td><td>BibArticle</td><td>Satake W, Nakabayashi Y, Mizuta I, Hirota Y, Ito C, Kubo M, et al. Genome-wide association study identifies common variants at four loci as genetic risk factors for Parkinson&#8217;s disease. Nat Genet. 2009;41(12):1303&#8211;7.</td><td><a href=http://dx.doi.org/10.1038/ng.485>10.1038/ng.485</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR78</td><td>BibArticle</td><td>Nalls MA, Pankratz N, Lill CM, Do CB, Hernandez DG, Saad M, et al. Large-scale meta-analysis of genome-wide association data identifies six new risk loci for Parkinson&#8217;s disease. Nat Genet. 2014;46(9):989&#8211;93.</td><td><a href=http://dx.doi.org/10.1038/ng.3043>10.1038/ng.3043</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR79</td><td>BibArticle</td><td>MacLeod DA, Rhinn H, Kuwahara T, Zolin A, Di Paolo G, McCabe BD, et al. RAB7L1 interacts with LRRK2 to modify intraneuronal protein sorting and Parkinson&#8217;s disease risk. Neuron. 2013;77(3):425&#8211;39.</td><td><a href=http://dx.doi.org/10.1016/j.neuron.2012.11.033>10.1016/j.neuron.2012.11.033</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR80</td><td>BibArticle</td><td>Tucci A, Nalls MA, Houlden H, Revesz T, Singleton AB, Wood NW, et al. Genetic variability at the PARK16 locus. Eur J Hum Genet. 2010;18(12):1356&#8211;9.</td><td><a href=http://dx.doi.org/10.1038/ejhg.2010.125>10.1038/ejhg.2010.125</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR81</td><td>BibArticle</td><td>Kolisek M, Sponder G, Mastrototaro L, Smorodchenko A, Launay P, Vormann J, et al. Substitution p.A350V in Na(+)/Mg(2)(+) exchanger SLC41A1, potentially associated with Parkinson&#8217;s disease, is a gain-of-function mutation. PLoS One. 2013;8(8):e71096.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0071096>10.1371/journal.pone.0071096</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR82</td><td>BibArticle</td><td>Beilina A, Rudenko IN, Kaganovich A, Civiero L, Chau H, Kalia SK, et al. Unbiased screen for interactors of leucine-rich repeat kinase 2 supports a common pathway for sporadic and familial Parkinson disease. Proc Natl Acad Sci U S A. 2014;111(7):2626&#8211;31.</td><td><a href=http://dx.doi.org/10.1073/pnas.1318306111>10.1073/pnas.1318306111</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR83</td><td>BibArticle</td><td>Stefansson H, Ophoff RA, Steinberg S, Andreassen OA, Cichon S, Rujescu D, et al. Common variants conferring risk of schizophrenia. Nature. 2009;460(7256):744&#8211;7.</td><td><a href=http://dx.doi.org/10.1038/nature08186>10.1038/nature08186</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, JournalTitle, Author_FamilyName_2, ArticleTitle, Year</td><td>CrossRef</td></tr><tr><td>CR84</td><td>BibArticle</td><td>Donohoe G, Walters J, Morris DW, Da Costa A, Rose E, Hargreaves A, et al. A neuropsychological investigation of the genome wide associated schizophrenia risk variant NRGN rs12807809. Schizophr Res. 2011;125(2-3):304&#8211;6.</td><td><a href=http://dx.doi.org/10.1016/j.schres.2010.10.019>10.1016/j.schres.2010.10.019</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR85</td><td>BibArticle</td><td>Krug A, Krach S, Jansen A, Nieratschker V, Witt SH, Shah NJ, et al. The effect of neurogranin on neural correlates of episodic memory encoding and retrieval. Schizophr Bull. 2013;39(1):141&#8211;50.</td><td><a href=http://dx.doi.org/10.1093/schbul/sbr076>10.1093/schbul/sbr076</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR86</td><td>BibArticle</td><td>Richards AL, Jones L, Moskvina V, Kirov G, Gejman PV, Levinson DF, et al. Schizophrenia susceptibility alleles are enriched for alleles that affect gene expression in adult human brain. Mol Psychiatry. 2012;17(2):193&#8211;201.</td><td><a href=http://dx.doi.org/10.1038/mp.2011.11>10.1038/mp.2011.11</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR87</td><td>BibArticle</td><td>Smith RL, Knight D, Williams H, Dwyer S, Richards A, Kirov G, et al. Analysis of neurogranin (NRGN) in schizophrenia. Am J Med Genet B Neuropsychiatr Genet. 2011;156B(5):532&#8211;5.</td><td><a href=http://dx.doi.org/10.1002/ajmg.b.31191>10.1002/ajmg.b.31191</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR88</td><td>BibArticle</td><td>Ohi K, Hashimoto R, Yasuda Y, Fukumoto M, Yamamori H, Umeda-Yano S, et al. Functional genetic variation at the NRGN gene and schizophrenia: evidence from a gene-based case-control study and gene expression analysis. Am J Med Genet B Neuropsychiatr Genet. 2012;159B(4):405&#8211;13.</td><td><a href=http://dx.doi.org/10.1002/ajmg.b.32043>10.1002/ajmg.b.32043</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR89</td><td>BibArticle</td><td>Ohi K, Hashimoto R, Yasuda Y, Fukumoto M, Yamamori H, Umeda-Yano S, et al. Influence of the NRGN gene on intellectual ability in schizophrenia. J Hum Genet. 2013;58(10):700&#8211;5.</td><td><a href=http://dx.doi.org/10.1038/jhg.2013.82>10.1038/jhg.2013.82</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR90</td><td>BibArticle</td><td>Ling KH, Hewitt CA, Beissbarth T, Hyde L, Cheah PS, Smyth GK, et al. Spatiotemporal regulation of multiple overlapping sense and novel natural antisense transcripts at the Nrgn and Camk2n1 gene loci during mouse cerebral corticogenesis. Cereb Cortex. 2011;21(3):683&#8211;97.</td><td><a href=http://dx.doi.org/10.1093/cercor/bhq141>10.1093/cercor/bhq141</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr></table>