<H1>Chapter DOI: 10.1186/s13073-016-0319-7<br/>Cited-By Count: 0</H1><table border="1" width="30%"><tr><td>Total References</td><td>113</td></tr><tr><td>Springer references</td><td>9</td></tr><tr><td>Non Springer references</td><td>104</td></tr><tr><td>BibStructured Count</td><td width="10%">101</td></tr><tr><td>BibUnstructured Count</td><td width="10%">12</td></tr><tr><td>DOI already available in SpringerLink</td><td>0</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>83</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>5</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>9</td></tr><tr><td>DOI Obtained For Non Springer</td><td>79</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>86</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>2</td></tr><tr><td>Valid DOI Enriched</td><td>76</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#DDDDDD'>Unsupervised</span> <span style='background:#BCBCBC'>Feature Learning</span></span> and <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#DDDDDD'>Deep</span>, <span style='background:#BCBCBC'>Learning Tutorial</span></span></aug>: <span style='background:#FF3300'>http://deeplearning.stanford.edu/tutorial/</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR2</td><td>BibArticle</td><td>L&#228;ngkvist M, Karlsson L, Loutfi A. A review of unsupervised feature learning and deep learning for time-series modeling. Pattern Recognit Lett. 2014;42:11&#8211;24.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.patrec.2014.01.008>10.1016/j.patrec.2014.01.008</a></td><td><b>CrossRef-Web</b></td>
<td>Year, Author_FamilyName_2, VolumeID, ArticleTitle, FirstPage</td><td>CrossRef</td></tr><tr><td>CR3</td><td>BibArticle</td><td>LeCun Y, Bengio Y, Hinton G. Deep learning. Nature. 2015;521:436&#8211;44.</td><td><a href=http://dx.doi.org/10.1038/nature14539>10.1038/nature14539</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, JournalTitle, ArticleTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Szegedy C, Toshev A, Erhan D. Deep neural networks for object detection. Adv Neural Inf Process Syst. 2013;26:2553&#8211;61.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR5</td><td>BibArticle</td><td>Lecun Y, Bottou L, Bengio Y, Haffner P. Gradient-based learning applied to document recognition. Proc IEEE. 1998;86:2278&#8211;324.</td><td><a href=http://dx.doi.org/10.1109/5.726791>10.1109/5.726791</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, VolumeID, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Margolin AA, Bilal E, Huang E, Norman TC, Ottestad L, Mecham BH, et al. Systematic analysis of challenge-driven improvements in molecular prognostic models for breast cancer. Sci Transl Med. 2013;5:181re1.</td><td><a href=http://dx.doi.org/10.1126/scitranslmed.3006112>10.1126/scitranslmed.3006112</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Year, Author_FamilyName_1, FirstPage, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Cheng WY, Yang THO, Anastassiou D. Development of a prognostic model for breast cancer survival in an open challenge environment. Sci Transl Med. 2013;5:181ra50.</td><td><a href=http://dx.doi.org/10.1126/scitranslmed.3005974>10.1126/scitranslmed.3005974</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, FirstPage, Author_FamilyName_1, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Langfelder P, Horvath S. WGCNA: an R package for weighted gene co-expression network analysis. BMC Bioinformatics. 2008;9:559.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Sherlock G. Analysis of large-scale gene expression data. Brief Bioinform. 2001;2:350&#8211;62.</td><td><a href=http://dx.doi.org/10.1093/bib/2.4.350>10.1093/bib/2.4.350</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, FirstPage, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Lee SI, Batzoglou S. ICA-based clustering of genes from microarray expression data. Adv Neural Inf Process Syst. 2004;16:675&#8211;82.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR11</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Celik</span> <span style='background:#DDDDDD'>S</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Logsdon</span> <span style='background:#DDDDDD'>BA</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Lee</span> <span style='background:#DDDDDD'>S-I.</span></span></aug> <span style='background:#CCCCFF'>Efficient dimensionality reduction for high-dimensional network estimation</span>. <span style='background:#CCFF99'>Proc of the 31st International Conference on Machine Learning</span>. <span style='background:#66FF66'>2014</span>;<span style='background:#FFCC66'>31</span>:<span style='background:#D279FF'>1953&#8211;61</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR12</td><td>BibBook</td><td>Koller D, Friedman N. Probabilistic graphical models: principles and techniques. Harvard, MA: MIT Press; 2009.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Chuang H-Y, Lee E, Liu Y-T, Lee D, Ideker T. Network-based classification of breast cancer metastasis. Mol Syst Biol. 2007;3:140.</td><td><a href=http://dx.doi.org/10.1038/msb4100180>10.1038/msb4100180</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, Author_FamilyName_1, Year, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Lee E, Chuang HY, Kim JW, Ideker T, Lee D. Inferring pathway activity toward precise disease classification. PLoS Comput Biol. 2008;4, e1000217.</td><td><a href=http://dx.doi.org/10.1371/journal.pcbi.1000217>10.1371/journal.pcbi.1000217</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, JournalTitle, ArticleTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Taylor IW, Linding R, Warde-Farley D, Liu Y, Pesquita C, Faria D, et al. Dynamic modularity in protein interaction networks predicts breast cancer outcome. Nat Biotechnol. 2009;27:199&#8211;204.</td><td><a href=http://dx.doi.org/10.1038/nbt.1522>10.1038/nbt.1522</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, FirstPage, Author_FamilyName_1, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Ravasi T, Suzuki H, Cannistraci CV, Katayama S, Bajic VB, Tan K, et al. An atlas of combinatorial transcriptional regulation in mouse and man. Cell. 2010;140:744&#8211;52.</td><td><a href=http://dx.doi.org/10.1016/j.cell.2010.01.044>10.1016/j.cell.2010.01.044</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, JournalTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Herschkowitz JI, Simin K, Weigman VJ, Mikaelian I, Usary J, Hu Z, et al. Identification of conserved gene expression features between murine mammary carcinoma models and human breast tumors. Genome Biol. 2007;8:R76.</td><td><a href=http://dx.doi.org/10.1186/gb-2007-8-5-r76>10.1186/gb-2007-8-5-r76</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, JournalTitle, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Hotelling H. Analysis of a complex of statistical variables into principal components. J Educ Psychol. 1933;24:417&#8211;41.</td><td><a href=http://dx.doi.org/10.1037/h0071325>10.1037/h0071325</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, FirstPage, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Lee S-I, Batzoglou S. Application of independent component analysis to microarrays. Genome Biol. 2003;4:R76.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Jiang D, Tang C, Zhang A. Cluster analysis for gene expression data: A survey. IEEE Trans Knowl Data Eng. 2004;16:1370&#8211;86.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1109/tkde.2004.68>10.1109/tkde.2004.68</a></td><td><b>CrossRef-Web</b></td>
<td>ArticleTitle, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR21</td><td>BibArticle</td><td>Chandrasekaran V, Parrilo PA, Willsky AS. Latent variable graphical model selection via convex optimization. Ann Stat. 2012;40:1935&#8211;67.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1214/12-AOS1020>10.1214/12-AOS1020</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR22</td><td>BibArticle</td><td>He Y, Qi Y, Kavukcuoglu K, Park H. Learning the dependency structure of latent factors. Adv Neural Inf Process Syst. 2012; 25:2366&#8211;74.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR23</td><td>BibArticle</td><td>Bell D, Berchuck A, Birrer M, Chien J, Cramer DW, Dao F, et al. Integrated genomic analyses of ovarian carcinoma. Nature. 2011;474:609&#8211;15.</td><td><a href=http://dx.doi.org/10.1038/nature10166>10.1038/nature10166</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, JournalTitle, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR24</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#DDDDDD'>INSPIRE</span> <span style='background:#BCBCBC'>web page</span></span></aug>: <span style='background:#FF3300'>http://inspire.cs.washington.edu</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR25</td><td>BibArticle</td><td>Johnson WE, Li C, Rabinovic A. Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostatistics. 2007;8:118&#8211;27.</td><td><a href=http://dx.doi.org/10.1093/biostatistics/kxj037>10.1093/biostatistics/kxj037</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR26</td><td>BibArticle</td><td>Barrett T, Wilhite SE, Ledoux P, Evangelista C, Kim IF, Tomashevsky M, et al. NCBI GEO: Archive for functional genomics data sets - Update. Nucleic Acids Res. 2013;41:D991&#8211;5.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1093/nar/gks1193>10.1093/nar/gks1193</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR27</td><td>BibArticle</td><td>Denkert C, Budczies J, Darb-Esfahani S, Gy&#246;rffy B, Sehouli J, K&#246;nsgen D, et al. A prognostic gene expression index in ovarian cancer - Validation across different independent data sets. J Pathol. 2009;218:273&#8211;80.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1002/path.2547>10.1002/path.2547</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Year, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR28</td><td>BibArticle</td><td>Bonome T, Levine DA, Shih J, Randonovich M, Pise-Masison CA, Bogomolniy F, et al. A gene signature predicting for survival in suboptimally debulked patients with ovarian cancer. Cancer Res. 2008;68:5478&#8211;86.</td><td><a href=http://dx.doi.org/10.1158/0008-5472.CAN-07-6595>10.1158/0008-5472.CAN-07-6595</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, Year, FirstPage, Author_FamilyName_1, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR29</td><td>BibArticle</td><td>Hendrix ND, Wu R, Kuick R, Schwartz DR, Fearon ER, Cho KR. Fibroblast growth factor 9 has oncogenic activity and is a downstream target of Wnt signaling in ovarian endometrioid adenocarcinomas. Cancer Res. 2006;66:1354&#8211;62.</td><td><a href=http://dx.doi.org/10.1158/0008-5472.CAN-05-3694>10.1158/0008-5472.CAN-05-3694</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, FirstPage, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR30</td><td>BibArticle</td><td>Mok SC, Bonome T, Vathipadiekal V, Bell A, Johnson ME, Wong KK, et al. A gene signature predictive for outcome in advanced ovarian cancer identifies a survival factor: microfibril-associated glycoprotein 2. Cancer Cell. 2009;16:521&#8211;32.</td><td><a href=http://dx.doi.org/10.1016/j.ccr.2009.10.018>10.1016/j.ccr.2009.10.018</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, ArticleTitle, FirstPage, Author_FamilyName_1, Year, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR31</td><td>BibArticle</td><td>Konstantinopoulos PA, Spentzos D, Karlan BY, Taniguchi T, Fountzilas E, Francoeur N, et al. Gene expression profile of BRCAness that correlates with responsiveness to chemotherapy and with outcome in patients with epithelial ovarian cancer. J Clin Oncol. 2010;28:3555&#8211;61.</td><td><a href=http://dx.doi.org/10.1200/JCO.2009.27.5719>10.1200/JCO.2009.27.5719</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, Year, FirstPage, Author_FamilyName_1, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR32</td><td>BibArticle</td><td>Meyniel J-P, Cottu PH, Decraene C, Stern M-H, Couturier J, Lebigot I, et al. A genomic and transcriptomic approach for a differential diagnosis between primary and secondary ovarian carcinomas in patients with a previous history of breast cancer. BMC Cancer. 2010;10:222.</td><td><a href=http://dx.doi.org/10.1186/1471-2407-10-222>10.1186/1471-2407-10-222</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR33</td><td>BibArticle</td><td>Ferriss JS, Kim Y, Duska L, Birrer M, Levine DA, Moskaluk C, Theodorescu D, Lee JK. Multi-gene expression predictors of single drug responses to adjuvant chemotherapy in ovarian carcinoma: Predicting platinum resistance. PLoS One. 2012;7, e30550.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0030550>10.1371/journal.pone.0030550</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Year, Author_FamilyName_1, JournalTitle, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR34</td><td>BibArticle</td><td>Tothill RW, Tinker AV, George J, Brown R, Fox SB, Lade S, et al. Novel molecular subtypes of serous and endometrioid ovarian cancer linked to clinical outcome. Clin Cancer Res. 2008;14:5198&#8211;208.</td><td><a href=http://dx.doi.org/10.1158/1078-0432.CCR-08-0196>10.1158/1078-0432.CCR-08-0196</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR35</td><td>BibArticle</td><td>Gautier L, Cope L, Bolstad BM, Irizarry RA. Affy - Analysis of Affymetrix GeneChip data at the probe level. Bioinformatics. 2004;20:307&#8211;15.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btg405>10.1093/bioinformatics/btg405</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, FirstPage, Author_FamilyName_1, Year, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR36</td><td>BibArticle</td><td>Maglott D, Ostell J, Pruitt KD, Tatusova T. Entrez gene: Gene-centered information at NCBI. Nucleic Acids Res. 2011;39:D52&#8211;7.</td><td><a href=http://dx.doi.org/10.1093/nar/gkq1237>10.1093/nar/gkq1237</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, FirstPage, Author_FamilyName_1, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR37</td><td>BibArticle</td><td>Dai M, Wang P, Boyd AD, Kostov G, Athey B, Jones EG, et al. Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data. Nucleic Acids Res. 2005;33, e175.</td><td><a href=http://dx.doi.org/10.1093/nar/gni179>10.1093/nar/gni179</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, ArticleTitle, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR38</td><td>BibArticle</td><td>Tibshirani R. Regression selection and shrinkage via the Lasso. J R Stat Soc B. 1994;58:267&#8211;88.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR39</td><td>BibArticle</td><td>Tibshirani R, Bien J, Friedman J, Hastie T, Simon N, Taylor J, et al. Strong rules for discarding predictors in lasso-type problems. J R Stat Soc Ser B Stat Methodol. 2012;74:245&#8211;66.</td><td><a href=http://dx.doi.org/10.1111/j.1467-9868.2011.01004.x>10.1111/j.1467-9868.2011.01004.x</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR40</td><td>BibArticle</td><td>Marquardt DW, Snee RD. Ridge regression in practice. Source Am Stat. 1975;29:3&#8211;20.</td><td><a href=http://dx.doi.org/10.1080/00031305.1975.10479105>10.1080/00031305.1975.10479105</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR41</td><td>BibArticle</td><td>Sardy S. On the practice of rescaling covariates. Int Stat Rev. 2008;76:285&#8211;97.</td><td><a href=http://dx.doi.org/10.1111/j.1751-5823.2008.00050.x>10.1111/j.1751-5823.2008.00050.x</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR42</td><td>BibArticle</td><td>de los Campos G, Hickey JM, Pong-Wong R, Daetwyler HD, Calus MPL. Whole-genome regression and prediction methods applied to plant and animal breeding. Genetics. 2013;193:327&#8211;45.</td><td><a href=http://dx.doi.org/10.1534/genetics.112.143313>10.1534/genetics.112.143313</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR43</td><td>BibArticle</td><td>Schmidt M, Niculescu-Mizil A, Murphy KP. Learning graphical model structure using L1-regularization paths. Proc AAAI Conf Artif Intell. 2007;22:1278.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR44</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Mu</span> <span style='background:#DDDDDD'>B</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>How</span> <span style='background:#DDDDDD'>JP.</span></span>, <span style='background:#BDBAD6'>Learning Sparse Gaussian Graphical Model with l0 -regularization. Tech Rep.</span></aug> <span style='background:#66FF66'>2014</span>; <span style='background:#D279FF'>1&#8211;13</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR45</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Friedman</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hastie</span> <span style='background:#DDDDDD'>T</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Tibshirani</span> <span style='background:#DDDDDD'>R.</span></span></aug> Applications of the lasso and grouped lasso to the estimation of sparse graphical models. Tech. Rep. <span style='background:#66FF66'>2010</span>; <span style='background:#D279FF'>1&#8211;22</span></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR46</td><td>BibArticle</td><td>Lee S-I, Pe&#8217;er D, Dudley AM, Church GM, Koller D. Identifying regulatory mechanisms using individual variation reveals key role for chromatin modification. Proc Natl Acad Sci U S A. 2006;103:14062&#8211;7.</td><td><a href=http://dx.doi.org/10.1073/pnas.0601852103>10.1073/pnas.0601852103</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR47</td><td>BibArticle</td><td>Lee SI, Dudley AM, Drubin D, Silver PA, Krogan NJ, Pe&#8217;er D, et al. Learning a prior on regulatory potential from eQTL data. PLoS Genet. 2009;5, e1000358.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1000358>10.1371/journal.pgen.1000358</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR48</td><td>BibArticle</td><td>Akavia UD, Litvin O, Kim J, Sanchez-Garcia F, Kotliar D, Causton HC, et al. An integrated approach to uncover drivers of cancer. Cell. 2010;143:1005&#8211;17.</td><td><a href=http://dx.doi.org/10.1016/j.cell.2010.11.013>10.1016/j.cell.2010.11.013</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage, Year, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR49</td><td>BibArticle</td><td>Segal E, Shapira M, Regev A, Pe&#8217;er D, Botstein D, Koller D, et al. Module networks: identifying regulatory modules and their condition-specific regulators from gene expression data. Nat Genet. 2003;34:166&#8211;76.</td><td><a href=http://dx.doi.org/10.1038/ng1165>10.1038/ng1165</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, Year, FirstPage, Author_FamilyName_1, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR50</td><td>BibUnstructured</td><td><span style='background:#BDBAD6'>cBio Cancer Genomics Portal</span>: <span style='background:#FF3300'>http://cbioportal.org</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR51</td><td>BibArticle</td><td>Mermel CH, Schumacher SE, Hill B, Meyerson ML, Beroukhim R, Getz G. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol. 2011;12:R41.</td><td><a href=http://dx.doi.org/10.1186/gb-2011-12-4-r41>10.1186/gb-2011-12-4-r41</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR52</td><td>BibBook</td><td>Mardia KV, Kent JT, Bibby JM. Multivariate analysis. Washington, DC: Academic Press; 1979.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.2307/3617970>10.2307/3617970</a></td><td><b>CrossRef-Web</b></td>
<td>BookTitle</td><td>CrossRef</td></tr><tr><td>CR53</td><td>BibBook</td><td>Lauritzen SL. Graphical models. Oxford: Oxford University Press; 1996.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR54</td><td>BibArticle</td><td>Friedman J, Hastie T, Tibshirani R. Sparse inverse covariance estimation with the graphical lasso. Biostatistics. 2008;9:432&#8211;41.</td><td><a href=http://dx.doi.org/10.1093/biostatistics/kxm045>10.1093/biostatistics/kxm045</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, ArticleTitle, JournalTitle, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR55</td><td>BibArticle</td><td>Tseng P. Convergence of a block coordinate descent method for nondifferentiable minimization. J Optim Theory Appl. 2001;109:475&#8211;94.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR56</td><td>BibArticle</td><td>Josse J, Chavent M, Liquet B, Husson F. Handling missing values with regularized iterative multiple correspondence analysis. J Classif. 2012;29:91&#8211;116.</td><td><a href=http://dx.doi.org/10.1007/s00357-012-9097-0>10.1007/s00357-012-9097-0</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, ArticleTitle, JournalTitle, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR57</td><td>BibArticle</td><td>Witten DM, Friedman JH, Simon N. New insights and faster computations for the graphical lasso. J Comput Graph Stat. 2011;20:892&#8211;900.</td><td><a href=http://dx.doi.org/10.1198/jcgs.2011.11051a>10.1198/jcgs.2011.11051a</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, FirstPage, Author_FamilyName_1, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR58</td><td>BibArticle</td><td>Tibshirani R. Regression shrinkage and selection via the lasso. J R Stat Soc Ser B. 1996;58:267&#8211;88.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR59</td><td>BibArticle</td><td>Harrell FE, Lee KL, Mark DB. Multivariable prognostic models: Issues in developing models, evaluating assumptions and adequacy, and measuring and reducing errors. Stat Med. 1996;15:361&#8211;87.</td><td><a href=http://dx.doi.org/10.1002/(SICI)1097-0258(19960229)15:4%3C361::AID-SIM168%3E3.0.CO;2-4>10.1002/(SICI)1097-0258(19960229)15:4%3C361::AID-SIM168%3E3.0.CO;2-4</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR60</td><td>BibArticle</td><td>Hofree M, Shen JP, Carter H, Gross A, Ideker T. Network-based stratification of tumor mutations. Nat Methods. 2013;10:1108&#8211;15.</td><td><a href=http://dx.doi.org/10.1038/nmeth.2651>10.1038/nmeth.2651</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, ArticleTitle, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR61</td><td>BibUnstructured</td><td><span style='background:#BDBAD6'>ImageJ</span>: <span style='background:#FF3300'>http://imagej.nih.gov/ij/</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR62</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Marlin</span> <span style='background:#DDDDDD'>BM</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Murphy</span> <span style='background:#DDDDDD'>K.</span></span></aug> <span style='background:#CCCCFF'>Sparse gaussian graphical models with unknown block structure</span>. <span style='background:#CCFF99'>Proc of the 26th International Conference on Machine Learning</span>. <span style='background:#66FF66'>2009</span>;<span style='background:#FFCC66'>26</span>:<span style='background:#D279FF'>705&#8211;712</span>.</td><td><a href=http://dx.doi.org/10.1145/1553374.1553465>10.1145/1553374.1553465</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, ArticleTitle, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR63</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Duchi</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Gould</span> <span style='background:#DDDDDD'>S.</span></span></aug> <span style='background:#CCCCFF'>Projected subgradient methods for learning sparse gaussians</span>. <span style='background:#CCFF99'>Proc of the Twenty-Fourth Conf on Uncertainty in Artificial Intelligence</span>. <span style='background:#66FF66'>2008</span>; <span style='background:#D279FF'>153&#8211;60</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR64</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hastie</span> <span style='background:#DDDDDD'>T</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Tibshirani</span> <span style='background:#DDDDDD'>R</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Friedman</span> <span style='background:#DDDDDD'>J.</span></span></aug>, <span style='background:#FFD9B3'>The Elements of Statistical Learning</span>. <span style='background:#C0FFC0'>New York, NY</span>: <span style='background:#FFFF49'>Springer New York Inc</span>; <span style='background:#66FF66'>2001</span>.</td><td><a href=http://dx.doi.org/10.1007/978-0-387-21606-5>10.1007/978-0-387-21606-5</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>PublisherName, Author_FamilyName_1, BookTitle, Year</td><td>CrossRef</td></tr><tr><td>CR65</td><td>BibArticle</td><td>Rand WM. Objective criteria for the evaluation of clustering methods. J Am Stat Assoc. 1971;66:846&#8211;50.</td><td><a href=http://dx.doi.org/10.1080/01621459.1971.10482356>10.1080/01621459.1971.10482356</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR66</td><td>BibArticle</td><td>Lachmann A, Xu H, Krishnan J, Berger SI, Mazloom AR, Ma&#8217;ayan A. ChEA: transcription factor regulation inferred from integrating genome-wide ChIP-X experiments. Bioinformatics. 2010;26:2438&#8211;44.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btq466>10.1093/bioinformatics/btq466</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Year, Author_FamilyName_1, FirstPage, ArticleTitle, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR67</td><td>BibArticle</td><td>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A. 2005;102:15545&#8211;50.</td><td><a href=http://dx.doi.org/10.1073/pnas.0506580102>10.1073/pnas.0506580102</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR68</td><td>BibArticle</td><td>Vastrik I, D&#8217;Eustachio P, Schmidt E, Joshi-Tope G, Gopinath G, Croft D, et al. Reactome: a knowledge base of biologic pathways and processes. Genome Biol. 2007;8:R39.</td><td><a href=http://dx.doi.org/10.1186/gb-2007-8-3-r39>10.1186/gb-2007-8-3-r39</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, JournalTitle, ArticleTitle, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR69</td><td>BibArticle</td><td>Kanehisa M, Goto S, Kawashima S, Okuno Y, Hattori M. The KEGG resource for deciphering the genome. Nucleic Acids Res. 2004;32:D277&#8211;80.</td><td><a href=http://dx.doi.org/10.1093/nar/gkh063>10.1093/nar/gkh063</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, ArticleTitle, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR70</td><td>BibArticle</td><td>Ciriello G, Miller ML, Aksoy BA, Senbabaoglu Y, Schultz N, Sander C. Emerging landscape of oncogenic signatures across human cancers. Nat Genet. 2013;45:1127&#8211;33.</td><td><a href=http://dx.doi.org/10.1038/ng.2762>10.1038/ng.2762</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR71</td><td>BibArticle</td><td>Efron B, Tibshirani R, Storey JD, Tusher V. Empirical Bayes analysis of a microarray experiment. J Am Stat Assoc. 2001;96:1151&#8211;60.</td><td><a href=http://dx.doi.org/10.1198/016214501753382129>10.1198/016214501753382129</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, FirstPage, Author_FamilyName_1, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR72</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Way</span> <span style='background:#DDDDDD'>GP</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Rudd</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Wang</span> <span style='background:#DDDDDD'>C</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hamidi</span> <span style='background:#DDDDDD'>H</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Fridley</span> <span style='background:#DDDDDD'>BL</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Konecny</span> <span style='background:#DDDDDD'>G</span></span>, et al.</aug> High-grade serous ovarian cancer subtypes are similar across populations. Biorxiv. <span style='background:#FF3300'>http://dx.doi.org/10.1101/030239</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR73</td><td>BibArticle</td><td>Wels J, Kaplan RN, Rafii S, Lyden D. Migratory neighbors and distant invaders: tumor-associated niche cells. Genes Dev. 2008;22:559&#8211;74.</td><td><a href=http://dx.doi.org/10.1101/gad.1636908>10.1101/gad.1636908</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR74</td><td>BibArticle</td><td>Le Blanc K, Mougiakakos D. Multipotent mesenchymal stromal cells and the innate immune system. Nat Rev Immunol. 2012;12:383&#8211;96.</td><td><a href=http://dx.doi.org/10.1038/nri3209>10.1038/nri3209</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, JournalTitle, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR75</td><td>BibArticle</td><td>Heuvers ME, Aerts JG, Cornelissen R, Groen H, Hoogsteden HC, Hegmans JP. Patient-tailored modulation of the immune system may revolutionize future lung cancer treatment. BMC Cancer. 2012;12:580.</td><td><a href=http://dx.doi.org/10.1186/1471-2407-12-580>10.1186/1471-2407-12-580</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, ArticleTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR76</td><td>BibArticle</td><td>Liu H, Ma Q, Xu Q, Lei J, Li X, Wang Z, et al. Therapeutic potential of perineural invasion, hypoxia and desmoplasia in pancreatic cancer. Curr Pharm Des. 2012;18:2395&#8211;403.</td><td><a href=http://dx.doi.org/10.2174/13816128112092395>10.2174/13816128112092395</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR77</td><td>BibArticle</td><td>Riester M, Wei W, Waldron L, Culhane AC, Trippa L, Oliva E, et al. Risk prediction for late-stage ovarian cancer by meta-analysis of 1525 patient samples. J Natl Cancer Inst. 2014;106:dju048.</td><td><a href=http://dx.doi.org/10.1093/jnci/dju048>10.1093/jnci/dju048</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, VolumeID, ArticleTitle, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR78</td><td>BibArticle</td><td>Puisieux A, Brabletz T, Caramel J. Oncogenic roles of EMT-inducing transcription factors. Nat Cell Biol. 2014;16:488&#8211;94.</td><td><a href=http://dx.doi.org/10.1038/ncb2976>10.1038/ncb2976</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, VolumeID, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR79</td><td>BibArticle</td><td>Ili&#263; D, Furuta Y, Kanazawa S, Takeda N, Sobue K, Nakatsuji N, et al. Reduced cell motility and enhanced focal adhesion contact formation in cells from FAK-deficient mice. Nature. 1995;377:539&#8211;44.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR80</td><td>BibArticle</td><td>Chautard E, Fatoux-Ardore M, Ballut L, Thierry-Mieg N, Ricard-Blum S. MatrixDB, the extracellular matrix interaction database. Nucleic Acids Res. 2011;39:D235&#8211;40.</td><td><a href=http://dx.doi.org/10.1093/nar/gkq830>10.1093/nar/gkq830</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, FirstPage, Author_FamilyName_1, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR81</td><td>BibArticle</td><td>Barker TH, Baneyx G, Card&#243;-Vila M, Workman GA, Weaver M, Menon PM, et al. SPARC regulates extracellular matrix organization through its modulation of integrin-linked kinase activity. J Biol Chem. 2005;280:36483&#8211;93.</td><td><a href=http://dx.doi.org/10.1074/jbc.M504663200>10.1074/jbc.M504663200</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR82</td><td>BibArticle</td><td>Dong J, Grunstein J, Tejada M, Peale F, Frantz G, Liang W-C, et al. VEGF-null cells require PDGFR alpha signaling-mediated stromal fibroblast recruitment for tumorigenesis. EMBO J. 2004;23:2800&#8211;10.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR83</td><td>BibArticle</td><td>Singh A, Settleman J. EMT, cancer stem cells and drug resistance: an emerging axis of evil in the war on cancer. Oncogene. 2010;29:4741&#8211;51.</td><td><a href=http://dx.doi.org/10.1038/onc.2010.215>10.1038/onc.2010.215</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, FirstPage, Author_FamilyName_1, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR84</td><td>BibArticle</td><td>Seton-Rogers S. Layers of regulation. Nat Rev Cancer. 2011;11:689.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR85</td><td>BibArticle</td><td>Yang J, Weinberg RA. Epithelial-mesenchymal transition: at the crossroads of development and tumor metastasis. Dev Cell. 2008;14:818&#8211;29.</td><td><a href=http://dx.doi.org/10.1016/j.devcel.2008.05.009>10.1016/j.devcel.2008.05.009</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR86</td><td>BibArticle</td><td>Gentles AJ, Plevritis SK, Majeti R, Alizadeh AA. Association of a leukemic stem cell gene expression signature with clinical outcomes in acute myeloid leukemia. JAMA. 2010;304:2706&#8211;15.</td><td><a href=http://dx.doi.org/10.1001/jama.2010.1862>10.1001/jama.2010.1862</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, JournalTitle, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR87</td><td>BibArticle</td><td>L&#243;pez-Novoa JM, Nieto MA. Inflammation and EMT: an alliance towards organ fibrosis and cancer progression. EMBO Mol Med. 2009;1:303&#8211;14.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1002/emmm.200900043>10.1002/emmm.200900043</a></td><td><b>CrossRef-Web</b></td>
<td>FirstPage, ArticleTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR88</td><td>BibArticle</td><td>Steinmetz R, Wagoner HA, Zeng P, Hammond JR, Hannon TS, Meyers JL, et al. Mechanisms regulating the constitutive activation of the extracellular signal-regulated kinase (ERK) signaling pathway in ovarian cancer and the effect of ribonucleic acid interference for ERK1/2 on cancer cell proliferation. Mol Endocrinol. 2004;18:2570&#8211;82.</td><td><a href=http://dx.doi.org/10.1210/me.2004-0082>10.1210/me.2004-0082</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR89</td><td>BibArticle</td><td>Tamura RE, de Vasconcellos JF, Sarkar D, Libermann TA, Fisher PB, Zerbini LF. GADD45 proteins: central players in tumorigenesis. Curr Mol Med. 2012;12:634&#8211;51.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.2174/156652412800619978>10.2174/156652412800619978</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, ArticleTitle, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR90</td><td>BibArticle</td><td>Tetreault M-P, Yang Y, Katz JP. Kr&#252;ppel-like factors in cancer. Nat Rev Cancer. 2013;13:701&#8211;13.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/nrc3582>10.1038/nrc3582</a></td><td><b>CrossRef-Web</b></td>
<td>VolumeID, Author_FamilyName_2, Year, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR91</td><td>BibArticle</td><td>Gentles AJ, Newman AM, Liu CL, Bratman SV, Feng W, Kim D, et al. The prognostic landscape of genes and infiltrating immune cells across human cancers. Nat Med. 2015;21:938&#8211;45.</td><td><a href=http://dx.doi.org/10.1038/nm.3909>10.1038/nm.3909</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, VolumeID, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR92</td><td>BibArticle</td><td>The Cancer Genome Atlas Research Network. Comprehensive genomic characterization defines human glioblastoma genes and core pathways. Nature. 2008;455:1061&#8211;8.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR93</td><td>BibArticle</td><td>Gravendeel LAM, Kouwenhoven MCM, Gevaert O, de Rooi JJ, Stubbs AP, Duijm JE, et al. Intrinsic gene expression profiles of gliomas are a better predictor of survival than histology. Cancer Res. 2009;69:9065&#8211;72.</td><td><a href=http://dx.doi.org/10.1158/0008-5472.CAN-09-2307>10.1158/0008-5472.CAN-09-2307</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, Year, FirstPage, Author_FamilyName_1, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR94</td><td>BibArticle</td><td>Collisson EA, Campbell JD, Brooks AN, Berger AH, Lee W, Chmielecki J, et al. Comprehensive molecular profiling of lung adenocarcinoma. Nature. 2014;511:543&#8211;50.</td><td><a href=http://dx.doi.org/10.1038/nature13385>10.1038/nature13385</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, JournalTitle, ArticleTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR95</td><td>BibArticle</td><td>Gentles AJ, Alizadeh AA, Lee SI, Myklebust JH, Shachaf CM, Shahbaba B, et al. A pluripotency signature predicts histologic transformation and influences survival in follicular lymphoma patients. Blood. 2009;114:3158&#8211;66.</td><td><a href=http://dx.doi.org/10.1182/blood-2009-02-202465>10.1182/blood-2009-02-202465</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, ArticleTitle, JournalTitle, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR96</td><td>BibArticle</td><td>Chen F, Kook H, Milewski R, Gitler AD, Lu MM, Li J, et al. Hop is an unusual homeobox gene that modulates cardiac development. Cell. 2002;110:713&#8211;23.</td><td><a href=http://dx.doi.org/10.1016/S0092-8674(02)00932-7>10.1016/S0092-8674(02)00932-7</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR97</td><td>BibArticle</td><td>Kee HJ, Kim J-R, Nam K-I, Park HY, Shin S, Kim JC, et al. Enhancer of polycomb1, a novel homeodomain only protein-binding partner, induces skeletal muscle differentiation. J Biol Chem. 2007;282:7700&#8211;9.</td><td><a href=http://dx.doi.org/10.1074/jbc.M611198200>10.1074/jbc.M611198200</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR98</td><td>BibArticle</td><td>Kook H, Lepore JJ, Gitler AD, Lu MM, Yung WWM, Mackay J, et al. Cardiac hypertrophy and histone deacetylase-dependent transcriptional repression mediated by the atypical homeodomain protein Hop. J Clin Invest. 2003;112:863&#8211;71.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1172/JCI19137>10.1172/JCI19137</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR99</td><td>BibArticle</td><td>Katoh H, Yamashita K, Waraya M, Margalit O, Ooki A, Tamaki H, et al. Epigenetic silencing of HOPX promotes cancer progression in colorectal cancer. Neoplasia. 2012;14:559&#8211;IN6.</td><td><a href=http://dx.doi.org/10.1593/neo.12330>10.1593/neo.12330</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Year, FirstPage, Author_FamilyName_1, ArticleTitle, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR100</td><td>BibArticle</td><td>Waraya M, Yamashita K, Katoh H, Ooki A, Kawamata H, Nishimiya H, et al. Cancer specific promoter CpG Islands hypermethylation of HOP homeobox (HOPX) gene and its potential tumor suppressive role in pancreatic carcinogenesis. BMC Cancer. 2012;12:397.</td><td><a href=http://dx.doi.org/10.1186/1471-2407-12-397>10.1186/1471-2407-12-397</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, Year, FirstPage, Author_FamilyName_1, ArticleTitle, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR101</td><td>BibArticle</td><td>Chen Y, Yang L, Cui T, Pacyna-Gengelbach M, Petersen I. HOPX is methylated and exerts tumour suppressive function through Ras-induced senescence in human lung cancer. J Pathol. 2015;235:397&#8211;407.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1002/path.4469>10.1002/path.4469</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, VolumeID, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR102</td><td>BibArticle</td><td>Jain R, Li D, Gupta M, Manderfield LJ, Ifkovits JL, Wang Q, et al. Integration of Bmp and Wnt signaling by Hopx specifies commitment of cardiomyoblasts. Science. 2015;348:aaa6071&#8211;1.</td><td><a href=http://dx.doi.org/10.1126/science.aaa6071>10.1126/science.aaa6071</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, ArticleTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR103</td><td>BibArticle</td><td>Logsdon BA, Gentles AJ, Miller CP, Blau CA, Becker PS, Lee SI. Sparse expression bases in cancer reveal tumor drivers. Nucleic Acids Res. 2015;43:1332&#8211;44.</td><td><a href=http://dx.doi.org/10.1093/nar/gku1290>10.1093/nar/gku1290</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, ArticleTitle, Author_FamilyName_2, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR104</td><td>BibArticle</td><td>Kim J-A, Choi H-K, Kim T-M, Leem S-H, Oh I-H. Regulation of mesenchymal stromal cells through fine tuning of canonical Wnt signaling. Stem Cell Res. 2015;14:356&#8211;68.</td><td><a href=http://dx.doi.org/10.1016/j.scr.2015.02.007>10.1016/j.scr.2015.02.007</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, VolumeID, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR105</td><td>BibArticle</td><td>Macheda ML, Stacker SA. Importance of Wnt signaling in the tumor stroma microenvironment. Curr Cancer Drug Targets. 2008;8:454&#8211;65.</td><td><a href=http://dx.doi.org/10.2174/156800908785699324>10.2174/156800908785699324</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, Author_FamilyName_1, FirstPage, Year, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR106</td><td>BibArticle</td><td>Savagner P, Yamada KM, Thiery JP. The zinc-finger protein slug causes desmosome dissociation, an initial and necessary step for growth factor-induced epithelial-mesenchymal transition. J Cell Biol. 1997;137:1403&#8211;19.</td><td><a href=http://dx.doi.org/10.1083/jcb.137.6.1403>10.1083/jcb.137.6.1403</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Year, FirstPage, Author_FamilyName_1, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR107</td><td>BibArticle</td><td>Mishra PJ, Mishra PJ, Humeniuk R, Medina DJ, Alexe G, Mesirov JP, et al. Carcinoma-associated fibroblast-like differentiation of human mesenchymal stem cells. Cancer Res. 2008;68:4331&#8211;9.</td><td><a href=http://dx.doi.org/10.1158/0008-5472.CAN-08-0943>10.1158/0008-5472.CAN-08-0943</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, VolumeID, Author_FamilyName_2, ArticleTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR108</td><td>BibArticle</td><td>Li N, Yousefi M, Nakauka-Ddamba A, Jain R, Tobias J, Epstein JA, et al. Single-cell analysis of proxy reporter allele-marked epithelial cells establishes intestinal stem cell hierarchy. Stem Cell Rep. 2014;3:876&#8211;91.</td><td><a href=http://dx.doi.org/10.1016/j.stemcr.2014.09.011>10.1016/j.stemcr.2014.09.011</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, VolumeID, Year, FirstPage, Author_FamilyName_1, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR109</td><td>BibArticle</td><td>Jain R, Barkauskas CE, Takeda N, Bowie EJ, Aghajanian H, Wang Q, et al. Plasticity of Hopx(+) type I alveolar cells to regenerate type II cells in the lung. Nat Commun. 2015;6:6727.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/ncomms7727>10.1038/ncomms7727</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, VolumeID, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR110</td><td>BibArticle</td><td>Ma S, Xie N, Li W, Yuan B, Shi Y, Wang Y. Immunobiology of mesenchymal stem cells. Cell Death Differ. 2014;21:216&#8211;25.</td><td><a href=http://dx.doi.org/10.1038/cdd.2013.158>10.1038/cdd.2013.158</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR111</td><td>BibArticle</td><td>Karnoub AE, Dash AB, Vo AP, Sullivan A, Brooks MW, Bell GW, et al. Mesenchymal stem cells within tumour stroma promote breast cancer metastasis. Nature. 2007;449:557&#8211;63.</td><td><a href=http://dx.doi.org/10.1038/nature06188>10.1038/nature06188</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, VolumeID, ArticleTitle, JournalTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR112</td><td>BibArticle</td><td>Law CW, Chen Y, Shi W, voom Smyth GK. Precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biol. 2014;15:R29.</td><td><a href=http://dx.doi.org/10.1186/gb-2014-15-2-r29>10.1186/gb-2014-15-2-r29</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, VolumeID, JournalTitle, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR113</td><td>BibUnstructured</td><td><span style='background:#BDBAD6'>Gene Expression Omnibus (GEO)</span>: <span style='background:#FF3300'>http://www.ncbi.nlm.nih.gov/geo/</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></table>