<H1>Chapter DOI: 10.1007/11780441_23<br/>Cited-By Count: 0</H1><table border="1" width="30%"><tr><td>Total References</td><td>20</td></tr><tr><td>Springer references</td><td>3</td></tr><tr><td>Non Springer references</td><td>17</td></tr><tr><td>BibStructured Count</td><td width="10%">16</td></tr><tr><td>BibUnstructured Count</td><td width="10%">4</td></tr><tr><td>DOI already available in SpringerLink</td><td>16</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>1</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>1</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>0</td></tr><tr><td>DOI Obtained For Non Springer</td><td>2</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>0</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>2</td></tr><tr><td>Valid DOI Enriched</td><td>0</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibArticle</td><td>Alliali, J., Sagot, M.-F.: A new distance for high level RNA secondary structure comparison. IEEE/ACM Transactions on Computational Biology and Bioinformatics&nbsp;2(1), 4&#8211;14 (2005)</td><td><a href=http://dx.doi.org/10.1109/TCBB.2005.9>10.1109/TCBB.2005.9</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Arslan, A.N., E&#487;ecio&#287;lu, &#214;., Pevzner, P.A.: A new approach to sequence alignment: normalized sequence alignment. Bioinformatics&nbsp;17(4), 327&#8211;337 (2001)</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/17.4.327>10.1093/bioinformatics/17.4.327</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibArticle</td><td>Backofen, R., Will, S.: Local sequence-structure motifs in RNA. Journal of Bioinformatics and Computational Biology (JBCB)&nbsp;2(4), 681&#8211;698 (2004)</td><td><a href=http://dx.doi.org/10.1142/S0219720004000818>10.1142/S0219720004000818</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Chartrand, P., Meng, X.-H., Singer, R.H., Long, R.M.: Structural elements required for the localization of ASH1 mRNA and of a green fluorescent protein reporter particle in vivo. Current Biology&nbsp;9, 333&#8211;336 (1999)</td><td><a href=http://dx.doi.org/10.1016/S0960-9822(99)80144-4>10.1016/S0960-9822(99)80144-4</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR5</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Chen</span>, <span style='background:#DDDDDD'>S.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Wang</span>, <span style='background:#DDDDDD'>Z.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Zhang</span>, <span style='background:#DDDDDD'>K.</span></span></aug>: <span style='background:#FFD9B3'>Pattern matching and local alignment for RNA structures</span>. In: International Conference on Mathematics and Engineering Techniques in Medicine and Biological Sciences (METMBS), pp.&amp;nbsp;<span style='background:#D279FF'>55&#8211;61</span> (<span style='background:#66FF66'>2002</span>)</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Couzin, J.: Breakthrough of the year. Small RNAs make big splash. Science&nbsp;298(5602), 2296&#8211;2297 (2002)</td><td><a href=http://dx.doi.org/10.1126/science.298.5602.2296>10.1126/science.298.5602.2296</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Currey, K.M., Sasha, D., Shapiro, B.A., Wang, J., Zhang, K.: An algorithm for finding the largest approxemately common substructure of two trees. IEEE Transactions on Pattern Analysis and Machine Intelligence&nbsp;20(8), 889&#8211;895 (1998)</td><td><a href=http://dx.doi.org/10.1109/34.709622>10.1109/34.709622</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR8</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Demaine</span>, <span style='background:#DDDDDD'>E.D.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Mozes</span>, <span style='background:#DDDDDD'>S.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Rossman</span>, <span style='background:#DDDDDD'>B.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Weimann</span>, <span style='background:#DDDDDD'>O.</span></span></aug>: <span style='background:#FFD9B3'>An <i>O</i>(<i>n</i> &lt;Superscript&gt;3&lt;/Superscript&gt;)-time algorithm for tree edit distance. Technical Report arXiv:cs.DS/0604037</span>, <span style='background:#FFFF49'>Cornell University</span> (<span style='background:#66FF66'>2006</span>)</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR9</td><td>BibChapter</td><td>Efraty, N., Landau, G.M.: Sparse normalized local alignment. In: Sahinalp, S.C., Muthukrishnan, S.M., Dogrusoz, U. (eds.) CPM 2004. LNCS, vol.&nbsp;3109, pp. 333&#8211;346. Springer, Heidelberg (2004)</td><td><a href=http://dx.doi.org/10.1007/978-3-540-27801-6_25>10.1007/978-3-540-27801-6_25</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR10</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Giegerich</span>, <span style='background:#DDDDDD'>R.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>H&#246;chsmann</span>, <span style='background:#DDDDDD'>M.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Kurtz</span>, <span style='background:#DDDDDD'>S.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>T&#246;ller</span>, <span style='background:#DDDDDD'>T.</span></span></aug>: <span style='background:#FFD9B3'>Local similarity in RNA secondary structures</span>. In: Computational Systems Bioinformatics (CSB), pp.&amp;nbsp;<span style='background:#D279FF'>159&#8211;168</span> (<span style='background:#66FF66'>2003</span>)</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1109/csb.2003.1227315>10.1109/csb.2003.1227315</a></td><td><b>CrossRef-Web</b></td>
<td>BookTitle</td><td>CrossRef</td></tr><tr><td>CR11</td><td>BibChapter</td><td>Guignon, V., Chauve, C., Hamel, S.: An edit distance between RNA stem-loops. In: Consens, M.P., Navarro, G. (eds.) SPIRE 2005. LNCS, vol.&nbsp;3772, pp. 335&#8211;347. Springer, Heidelberg (2005)</td><td><a href=http://dx.doi.org/10.1007/11575832_38>10.1007/11575832_38</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR12</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Gusfield</span>, <span style='background:#DDDDDD'>D.</span></span></aug>: <span style='background:#FFD9B3'>Algorithms on Strings, Trees, and Sequences</span>. Computer Science and Computational Biology. Press Syndicate of the University of Cambridge (<span style='background:#66FF66'>1997</span>)</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1017/CBO9780511574931>10.1017/CBO9780511574931</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Jiang, T., Lin, G., Ma, B., Zhang, K.: A general edit distance between RNA structures. Journal of Computational Biology&nbsp;9(2), 371&#8211;388 (2002)</td><td><a href=http://dx.doi.org/10.1089/10665270252935511>10.1089/10665270252935511</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibChapter</td><td>Klein, P.N.: Computing the edit-distance between unrooted ordered trees. In: Bilardi, G., Pietracaprina, A., Italiano, G.F., Pucci, G. (eds.) ESA 1998. LNCS, vol.&nbsp;1461, pp. 91&#8211;102. Springer, Heidelberg (1998)</td><td><a href=http://dx.doi.org/10.1007/3-540-68530-8_8>10.1007/3-540-68530-8_8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Moore, P.B.: Structural motifs in RNA. Annual review of biochemistry&nbsp;68, 287&#8211;300 (1999)</td><td><a href=http://dx.doi.org/10.1146/annurev.biochem.68.1.287>10.1146/annurev.biochem.68.1.287</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Shasha, D., Zhang, K.: Simple fast algorithms for the editing distance between trees and related problems. SIAM Journal on Computing&nbsp;18(6), 1245&#8211;1262 (1989)</td><td><a href=http://dx.doi.org/10.1137/0218082>10.1137/0218082</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Smith, T.F., Waterman, M.S.: The identification of common molecular subsequences. Journal of Molecular Biology&nbsp;147, 195&#8211;197 (1981)</td><td><a href=http://dx.doi.org/10.1016/0022-2836(81)90087-5>10.1016/0022-2836(81)90087-5</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Wang, J., Zhang, K.: Identiffying consensus of trees through alignment. Information Sciences&nbsp;126, 165&#8211;189 (2000)</td><td><a href=http://dx.doi.org/10.1016/S0020-0255(00)00014-1>10.1016/S0020-0255(00)00014-1</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Zuker, M.: On finding all suboptimal foldings of an RNA molecule. Science&nbsp;244(4900), 48&#8211;52 (1989)</td><td><a href=http://dx.doi.org/10.1126/science.2468181>10.1126/science.2468181</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Zuker, M., Stiegler, P.: Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information. Nucleic Acids Research&nbsp;9(1), 133&#8211;148 (1981)</td><td><a href=http://dx.doi.org/10.1093/nar/9.1.133>10.1093/nar/9.1.133</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr></table>