<H1>Chapter DOI: 10.1186/s12934-016-0507-6<br/>Cited-By Count: 0</H1><table border="1" width="30%"><tr><td>Total References</td><td>174</td></tr><tr><td>Springer references</td><td>30</td></tr><tr><td>Non Springer references</td><td>144</td></tr><tr><td>BibStructured Count</td><td width="10%">173</td></tr><tr><td>BibUnstructured Count</td><td width="10%">1</td></tr><tr><td>DOI already available in SpringerLink</td><td>0</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>135</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>4</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>30</td></tr><tr><td>DOI Obtained For Non Springer</td><td>109</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>139</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>0</td></tr><tr><td>Valid DOI Enriched</td><td>82</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibArticle</td><td>Michel RH, McGovern PE, Badler V. Chemical evidence for ancient beer. Nature. 1992;360:24.</td><td><a href=http://dx.doi.org/10.1038/360024b0>10.1038/360024b0</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, JournalTitle, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR2</td><td>BibArticle</td><td>McGovern PE, Glusker DL, Exner LJ, Voigt MM. Neolithic resinated wines. Nature. 1996;281:480&#8211;1.</td><td><a href=http://dx.doi.org/10.1038/381480a0>10.1038/381480a0</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, ArticleTitle, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR3</td><td>BibArticle</td><td>Salque M, Bogucki PI, Pyzel J, Sobkowiak-Tabaka I, Grygiel R, Szmyt M, Evershed RP. Earliest evidence for cheese making in the sixth millennium BC in northern Europe. Nature. 2013;493:522&#8211;5.</td><td><a href=http://dx.doi.org/10.1038/nature11698>10.1038/nature11698</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Allen F, Andreotti R, Eveleigh DE, Nystrom J. Mary Elizabeth Hickox Mandels, 90, bioenergy leader. Biotechnol Biofuels. 2009;2:22.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-2-22>10.1186/1754-6834-2-22</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2, ArticleTitle, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR5</td><td>BibArticle</td><td>Jeffries T, Lindblad P. We march backwards into the future. Curr Opin Biotechnol. 2009;20:255&#8211;6.</td><td><a href=http://dx.doi.org/10.1016/j.copbio.2009.06.004>10.1016/j.copbio.2009.06.004</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Reese ET. A microbiological process report; enzymatic hydrolysis of cellulose. App Microbiol. 1956;4:39&#8211;45.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Mandels M, Reese ET. Induction of cellulase in <Emphasis Type="Italic">Trichoderma viride</Emphasis> as influenced by carbon sources and metals. J Bacteriol. 1957;73:269&#8211;78.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Mandels M, Reese ET. Induction of cellulase in fungi by cellobiose. J Bacteriol. 1960;79:816&#8211;26.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Mandels M, Parrish FW, Reese ET. Sophorose as an inducer of cellulase in <Emphasis Type="Italic">Trichoderma viride</Emphasis>. J Bacteriol. 1962;83:400&#8211;8.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Shoemaker S, Schweickart V, Ladner M, Gelfand D, Kwok S, Myambo K, Innis M. Molecular cloning of exo-cellobiohydrolase I derived from <Emphasis Type="Italic">Trichoderma reesei</Emphasis> strain L27. Nat Biotech. 1983;1:691&#8211;6.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/nbt1083-691>10.1038/nbt1083-691</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR11</td><td>BibArticle</td><td>Divne C, Stahlberg J, Reinikainen T, Ruohonen L, Pettersson G, Knowles JK, Teeri TT, Jones TA. The three-dimensional crystal structure of the catalytic core of cellobiohydrolase I from <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Science. 1994;265:524&#8211;8.</td><td><a href=http://dx.doi.org/10.1126/science.8036495>10.1126/science.8036495</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR12</td><td>BibArticle</td><td>Mandels M, Weber J, Parizek R. Enhanced cellulase production by a mutant of <Emphasis Type="Italic">Trichoderma viride</Emphasis>. App Microbiol. 1971;21:152&#8211;4.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Eveleigh DE, Montenecourt BS. Increasing yields of extracellular enzymes. Adv Appl Microbiol. 1979;25:57&#8211;74.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/S0065-2164(08)70146-1>10.1016/S0065-2164(08)70146-1</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Peterson R, Nevalainen H. <Emphasis Type="Italic">Trichoderma reesei</Emphasis> RUT-C30&#8211;thirty years of strain improvement. Microbiology. 2012;158:58&#8211;68.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Durand H, Clanet M, Tiraby G. Genetic improvement of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> for large scale cellulase production. Enzyme Microb Technol. 1988;10:341&#8211;6.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/0141-0229(88)90012-9>10.1016/0141-0229(88)90012-9</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR16</td><td>BibChapter</td><td>Galante Y, De Conti A, Monteverdi R. Applications of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> enzymes in the food and feed industry. In: Harman GE, Kubicek CP, editors. <Emphasis Type="Italic">Trichoderma</Emphasis> and <Emphasis Type="Italic">Gliocladium</Emphasis>, vol. 2. London: Taylor &amp; Francis Ltd; 1998. p. 338&#8211;42.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR17</td><td>BibChapter</td><td>Galante Y, De Conti A, Monteverdi R. Applications of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> enzymes in the textile industry. In: Harman GE, Kubicek CP, editors. <Emphasis Type="Italic">Trichoderma</Emphasis> and <Emphasis Type="Italic">Gliocladium</Emphasis>, vol. 2. London: Taylor &amp; Francis Ltd; 1998. p. 311&#8211;26.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR18</td><td>BibChapter</td><td>Buchert J, Oksanen J, Pere J, Siika-aho M, Suurn&#228;kki A, Viikari L. Application of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> enzymes in the pulp and paper industry. In: Harman GE, Kubicek CP, editors. <Emphasis Type="Italic">Trichoderma</Emphasis> and <Emphasis Type="Italic">Gliocladium</Emphasis>, vol. 2. London: Taylor &amp; Francis Ltd; 1998. p. 343&#8211;64.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Dekker RFH. Bioconversion of hemicellulose: aspects of hemicellulase production by <Emphasis Type="Italic">Trichoderma reesei</Emphasis> QM 9414 and enzymic saccharification of hemicellulose. Biotechnol Bioeng. 1983;25:1127&#8211;46.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1002/bit.260250419>10.1002/bit.260250419</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Biely P, Puls J, Schneider H. Acetyl xylan esterases in fungal cellulolytic systems. FEBS Lett. 1985;186:80&#8211;4.</td><td><a href=http://dx.doi.org/10.1016/0014-5793(85)81343-0>10.1016/0014-5793(85)81343-0</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR21</td><td>BibArticle</td><td>Saloheimo M, Lehtovaara P, Penttil&#228; M, Teeri TT, St&#229;hlberg J, Johansson G, Pettersson G, Claeyssens M, Tomme P, Knowles JKC. EGIII, a new endoglucanase from <Emphasis Type="Italic">Trichoderma reesei</Emphasis>: the characterization of both gene and enzyme. Gene. 1988;63:11&#8211;21.</td><td><a href=http://dx.doi.org/10.1016/0378-1119(88)90541-0>10.1016/0378-1119(88)90541-0</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR22</td><td>BibArticle</td><td>T&#246;rr&#246;nen A, Mach RL, Messner R, Gonzalez R, Kalkkinen N, Harkki A, Kubicek CP. The two major xylanases from <Emphasis Type="Italic">Trichoderma reesei</Emphasis>: characterization of both enzymes and genes. Nat Biotechnol. 1992;10:1461&#8211;5.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR23</td><td>BibArticle</td><td>Tenkanen M, Buchert J, Viikari L. Binding of hemicellulases on isolated polysaccharide substrates. Enzyme Microb Technol. 1995;17:499&#8211;505.</td><td><a href=http://dx.doi.org/10.1016/0141-0229(94)00050-2>10.1016/0141-0229(94)00050-2</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR24</td><td>BibArticle</td><td>Tenkanen M, Luonteri E, Teleman A. Effect of side groups on the action of &#946;-xylosidase from <Emphasis Type="Italic">Trichoderma reesei</Emphasis> against substituted xylo-oligosaccharides. FEBS Lett. 1996;399:303&#8211;6.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR25</td><td>BibArticle</td><td>Biely P, C&#244;t&#233; GL, Kremnick&#253; L, Greene RV, Tenkanen M. Action of acetylyxlan esterase from <Emphasis Type="Italic">Trichoderma reesei</Emphasis> on acetylated methyl glycosides. FEBS Lett. 1997;420:121&#8211;4.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR26</td><td>BibArticle</td><td>Tenkanen M, Vr&#353;ansk&#225; M, Siika-Aho M, Wong DW, Puchart V, Penttil&#228; M, Saloheimo M, Biely P. Xylanase XYN IV from <Emphasis Type="Italic">Trichoderma reesei</Emphasis> showing exo- and endo-xylanase activity. FEBS J. 2013;280:285&#8211;301.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR27</td><td>BibArticle</td><td>Biely P, Czisz&#225;rov&#225; M, Agger JW, Li XL, Puchart V, Vr&#353;ansk&#225; M, Eijsink VGH, Westereng B. <Emphasis Type="Italic">Trichoderma reesei</Emphasis> CE16 acetyl esterase and its role in enzymatic degradation of acetylated hemicellulose. Biochim Biophys Acta. 2014;1840:516&#8211;25.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.bbagen.2013.10.008>10.1016/j.bbagen.2013.10.008</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR28</td><td>BibArticle</td><td>Biely P, Puchart V, Stringer MA, Krogh KBRM. <Emphasis Type="Italic">Trichoderma reesei</Emphasis> XYN VI&#8212;a novel appendage-dependent eukaryotic glucuronoxylan hydrolase. FEBS J. 2014;281:3894&#8211;903.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR29</td><td>BibArticle</td><td>Igarashi K, Uchihashi T, Koivula A, Wada M, Kimura S, Okamoto T, Penttil&#228; M, Ando T, Samejima M. Traffic jams reduce hydrolytic efficiency of cellulase on cellulose surface. Science. 2011;333:1279&#8211;82.</td><td><a href=http://dx.doi.org/10.1126/science.1208386>10.1126/science.1208386</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR30</td><td>BibArticle</td><td>Gruber F, Visser J, Kubicek CP, De Graaff LH. The development of a heterologous transformation system for the cellulolytic fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis> based on a <Emphasis Type="Italic">pyrG</Emphasis>-negative mutant strain. Curr Genet. 1990;18:71&#8211;6.</td><td><a href=http://dx.doi.org/10.1007/BF00321118>10.1007/BF00321118</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR31</td><td>BibArticle</td><td>Penttil&#228; M, Nevalainen H, Ratto M, Salminen E, Knowles J. A versatile transformation system for the cellulolytic filamentous fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Gene. 1987;61:155&#8211;64.</td><td><a href=http://dx.doi.org/10.1016/0378-1119(87)90110-7>10.1016/0378-1119(87)90110-7</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR32</td><td>BibArticle</td><td>Kubicek CP, Messner R, Gruber F, Mach RL, Kubicek-Pranz EM. The <Emphasis Type="Italic">Trichoderma</Emphasis> cellulase regulatory puzzle: from the interior life of a secretory fungus. Enzyme Microb Technol. 1993;15:90&#8211;9.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/0141-0229(93)90030-6>10.1016/0141-0229(93)90030-6</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR33</td><td>BibArticle</td><td>Kubicek CP, Mikus M, Schuster A, Schmoll M, Seiboth B. Metabolic engineering strategies for the improvement of cellulase production by <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>. Biotechnol Biofuels. 2009;2:1&#8211;14.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-2-19>10.1186/1754-6834-2-19</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_1, Author_FamilyName_2, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR34</td><td>BibArticle</td><td>Harkki A, Uusitalo J, Bailey M, Penttil&#228; M, Knowles JKC. A novel fungal expression system: secretion of active calf chymosin from the filamentous fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Bio Technol. 1989;7:596&#8211;603.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/nbt0689-596>10.1038/nbt0689-596</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR35</td><td>BibArticle</td><td>Kuhls K, Lieckfeldt E, Samuels GJ, Kovacs W, Meyer W, Petrini O, Gams W, B&#246;rner T, Kubicek CP. Molecular evidence that the asexual industrial fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis> is a clonal derivative of the ascomycete <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>. Proc Natl Acad Sci USA. 1996;93:7755&#8211;60.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1073/pnas.93.15.7755>10.1073/pnas.93.15.7755</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage, VolumeID</td><td>CrossRef</td></tr><tr><td>CR36</td><td>BibArticle</td><td>Seidl V, Seibel C, Kubicek CP, Schmoll M. Sexual development in the industrial workhorse <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Proc Natl Acad Sci USA. 2009;106:13909&#8211;14.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1073/pnas.0904936106>10.1073/pnas.0904936106</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR37</td><td>BibArticle</td><td>Linke R, Thallinger GG, Haarmann T, Eidner J, Schreiter M, Lorenz P, Seiboth B, Kubicek CP. Restoration of female fertility in <Emphasis Type="Italic">Trichoderma reesei</Emphasis> QM6a provides the basis for inbreeding in this industrial cellulase producing fungus. Biotechnol Biofuels. 2015;8:155.</td><td><a href=http://dx.doi.org/10.1186/s13068-015-0311-2>10.1186/s13068-015-0311-2</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, Author_FamilyName_1, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR38</td><td>BibArticle</td><td>Foreman PK, Brown D, Dankmeyer L, Dean R, Diener S, Dunn-Coleman NS, Goedegebuur F, Houfek TD, England GJ, Kelley AS, et al. Transcriptional regulation of biomass-degrading enzymes in the filamentous fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. J Biol Chem. 2003;278:31988&#8211;97.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1074/jbc.M304750200>10.1074/jbc.M304750200</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR39</td><td>BibArticle</td><td>Martinez D, Berka RM, Henrissat B, Saloheimo M, Arvas M, Baker SE, Chapman J, Chertkov O, Coutinho PM, Cullen D, et al. Genome sequencing and analysis of the biomass-degrading fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis> (syn. <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>). Nat Biotechnol. 2008;26:553&#8211;60.</td><td><a href=http://dx.doi.org/10.1038/nbt1403>10.1038/nbt1403</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR40</td><td>BibArticle</td><td>Kubicek CP. Systems biological approaches towards understanding cellulase production by <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. J Biotechnol. 2013;163:133&#8211;42.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.jbiotec.2012.05.020>10.1016/j.jbiotec.2012.05.020</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, FirstPage, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR41</td><td>BibChapter</td><td>Bischof R, Seiboth B. Systems biology of carbohydrate active enzyme production in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. In: Hallenbeck PC, editor. Advances in enzymatic conversion of biomass to biofuels. Montreal: Future Medicine Ltd; 2015. p. 6&#8211;19.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR42</td><td>BibArticle</td><td>Le Crom S, Schackwitz W, Pennacchio L, Magnuson JK, Culley DE, Collett JR, Martin J, Druzhinina IS, Mathis H, Monot F, et al. Tracking the roots of cellulase hyperproduction by the fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis> using massively parallel DNA sequencing. Proc Natl Acad Sci USA. 2009;106:16151&#8211;6.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1073/pnas.0905848106>10.1073/pnas.0905848106</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR43</td><td>BibArticle</td><td>Levinson HS, Mandels GR, Reese ET. Products of enzymatic hydrolysis of cellulose and its derivatives. Arch Biochem Biophys. 1951;31:351&#8211;65.</td><td><a href=http://dx.doi.org/10.1016/0003-9861(51)90150-6>10.1016/0003-9861(51)90150-6</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR44</td><td>BibArticle</td><td>Wyman CE. What is (and is not) vital to advancing cellulosic ethanol. Trends Biotechnol. 2007;25:153&#8211;7.</td><td><a href=http://dx.doi.org/10.1016/j.tibtech.2007.02.009>10.1016/j.tibtech.2007.02.009</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Year, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR45</td><td>BibArticle</td><td>Lombard V, Ramulu HG, Drula E, Coutinho PM, Henrissat B. The carbohydrate-active enzymes database (CAZy) in 2013. Nucleic Acids Res. 2014;42:D490&#8211;5.</td><td><a href=http://dx.doi.org/10.1093/nar/gkt1178>10.1093/nar/gkt1178</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR46</td><td>BibArticle</td><td>Messner R, Hagspiel K, Kubicek CP. Isolation of a &#946;-glucosidase binding and activating polysaccharide from cell walls of <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Arch Microbiol. 1990;154:150&#8211;5.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1007/BF00423325>10.1007/BF00423325</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR47</td><td>BibArticle</td><td>Ryu DDY, Mandels M. Cellulases: biosynthesis and applications. Enzyme Microb Technol. 1980;2:91&#8211;102.</td><td><a href=http://dx.doi.org/10.1016/0141-0229(80)90063-0>10.1016/0141-0229(80)90063-0</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR48</td><td>BibArticle</td><td>Mach RL, Seiboth B, Myasnikov A, Gonzalez R, Strauss J, Harkki AM, Kubicek CP. The <Emphasis Type="Italic">bgl1</Emphasis> gene of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> QM 9414 encodes an extracellular, cellulose-inducible &#946;-glucosidase involved in cellulase induction by sophorose. Mol Microbiol. 1995;16:687&#8211;97.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1111/j.1365-2958.1995.tb02430.x>10.1111/j.1365-2958.1995.tb02430.x</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR49</td><td>BibArticle</td><td>S&#248;rensen A, L&#252;beck M, L&#252;beck PS, Ahring BK. Fungal beta-glucosidases: a bottleneck in industrial use of lignocellulosic materials. Biomolecules. 2013;3:612&#8211;31.</td><td><a href=http://dx.doi.org/10.3390/biom3030612>10.3390/biom3030612</a></td><td>&nbsp;</td><td><b>CrossRef-Web</b></td>
<td>Year, FirstPage, ArticleTitle, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR50</td><td>BibArticle</td><td>Qing Q, Yang B, Wyman CE. Xylooligomers are strong inhibitors of cellulose hydrolysis by enzymes. Bioresour Technol. 2010;101:9624&#8211;30.</td><td><a href=http://dx.doi.org/10.1016/j.biortech.2010.06.137>10.1016/j.biortech.2010.06.137</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR51</td><td>BibArticle</td><td>Xin D, Ge X, Sun Z, Viikari L, Zhang J. Competitive inhibition of cellobiohydrolase I by manno-oligosaccharides. Enzyme Microb Technol. 2015;68:62&#8211;8.</td><td><a href=http://dx.doi.org/10.1016/j.enzmictec.2014.09.009>10.1016/j.enzmictec.2014.09.009</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR52</td><td>BibArticle</td><td>Zhang J, Viikari L. Xylo-oligosaccharides are competitive inhibitors of cellobiohydrolase I from <Emphasis Type="Italic">Thermoascus aurantiacus</Emphasis>. Bioresour Technol. 2012;117:286&#8211;91.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.biortech.2012.04.072>10.1016/j.biortech.2012.04.072</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR53</td><td>BibArticle</td><td>Banerjee G, Car S, Scott-Craig JS, Borrusch MS, Aslam N, Walton JD. Synthetic enzyme mixtures for biomass deconstruction: production and optimization of a core set. Biotechnol Bioeng. 2010;106:707&#8211;20.</td><td><a href=http://dx.doi.org/10.1002/bit.22741>10.1002/bit.22741</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR54</td><td>BibArticle</td><td>Jacobson F, Karkehabadi S, Hansson H, Goedegebuur F, Wallace L, Mitchinson C, Piens K, Stals I, Sandgren M. The crystal structure of the core domain of a cellulose induced protein (Cip1) from <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>, at 1.5 &#197; resolution. PLoS One. 2013;8:e70562.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0070562>10.1371/journal.pone.0070562</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR55</td><td>BibArticle</td><td>Pokkuluri PR, Duke NEC, Wood SJ, Cotta MA, Li XL, Biely P, Schiffer M. Structure of the catalytic domain of glucuronoyl esterase Cip2 from <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>. Prot Struct Funct Bioinform. 2011;79:2588&#8211;92.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1002/prot.23088>10.1002/prot.23088</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR56</td><td>BibArticle</td><td>Saloheimo M, Paloheimo M, Hakola S, Pere J, Swanson B, Nyyssonen E, Bhatia A, Ward M, Penttil&#228; M. Swollenin, a <Emphasis Type="Italic">Trichoderma reesei</Emphasis> protein with sequence similarity to the plant expansins, exhibits disruption activity on cellulosic materials. Eur J Biochem. 2002;269:4202&#8211;11.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1046/j.1432-1033.2002.03095.x>10.1046/j.1432-1033.2002.03095.x</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR57</td><td>BibArticle</td><td>Andberg M, Penttil&#228; M, Saloheimo M. Swollenin from <Emphasis Type="Italic">Trichoderma reesei</Emphasis> exhibits hydrolytic activity against cellulosic substrates with features of both endoglucanases and cellobiohydrolases. Bioresour Technol. 2015;181:105&#8211;13.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.biortech.2015.01.024>10.1016/j.biortech.2015.01.024</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR58</td><td>BibArticle</td><td>J&#228;ger G, Girfoglio M, Dollo F, Rinaldi R, Bongard H, Commandeur U, Fischer R, Spiess AC, B&#252;chs J. How recombinant swollenin from <Emphasis Type="Italic">Kluyveromyces lactis</Emphasis> affects cellulosic substrates and accelerates their hydrolysis. Biotechnol Biofuels. 2011;4:33.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR59</td><td>BibArticle</td><td>Gourlay K, Hu J, Arantes V, Andberg M, Saloheimo M, Penttil&#228; M, Saddler J. Swollenin aids in the amorphogenesis step during the enzymatic hydrolysis of pretreated biomass. Bioresour Technol. 2013;142:498&#8211;503.</td><td><a href=http://dx.doi.org/10.1016/j.biortech.2013.05.053>10.1016/j.biortech.2013.05.053</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR60</td><td>BibArticle</td><td>Harris PV, Welner D, McFarland KC, Re E, Navarro Poulsen JC, Brown K, Salbo R, Ding H, Vlasenko E, Merino S, et al. Stimulation of lignocellulosic biomass hydrolysis by proteins of glycoside hydrolase family 61: structure and function of a large, enigmatic family. Biochemistry. 2010;49:3305&#8211;16.</td><td><a href=http://dx.doi.org/10.1021/bi100009p>10.1021/bi100009p</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR61</td><td>BibArticle</td><td>Levasseur A, Drula E, Lombard V, Coutinho PM, Henrissat B. Expansion of the enzymatic repertoire of the CAZy database to integrate auxiliary redox enzymes. Biotechnol Biofuels. 2013;6:41.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-6-41>10.1186/1754-6834-6-41</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, ArticleTitle, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR62</td><td>BibArticle</td><td>Eibinger M, Ganner T, Bubner P, Ro&#353;ker S, Kracher D, Haltrich D, Ludwig R, Plank H, Nidetzky B. Cellulose surface degradation by a lytic polysaccharide monooxygenase and its effect on cellulase hydrolytic efficiency. J Biol Chem. 2014;289:35929&#8211;38.</td><td><a href=http://dx.doi.org/10.1074/jbc.M114.602227>10.1074/jbc.M114.602227</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR63</td><td>BibArticle</td><td>Westereng B, Cannella D, Agger JW, Jorgensen H, Andersen ML, Eijsink VG, Felby C. Enzymatic cellulose oxidation is linked to lignin by long-range electron transfer. Sci Rep. 2015;5:18561.</td><td><a href=http://dx.doi.org/10.1038/srep18561>10.1038/srep18561</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR64</td><td>BibArticle</td><td>Kracher D, Scheiblbrandner S, Felice AK, Breslmayr E, Preims M, Ludwicka K, Haltrich D, Eijsink VG, Ludwig R. Extracellular electron transfer systems fuel cellulose oxidative degradation. Science. 2016;352(6289):1098&#8211;101.</td><td><a href=http://dx.doi.org/10.1126/science.aaf3165>10.1126/science.aaf3165</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, IssueID, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR65</td><td>BibArticle</td><td>Bischof R, Fourtis L, Limbeck A, Gamauf C, Seiboth B, Kubicek CP. Comparative analysis of the <Emphasis Type="Italic">Trichoderma reesei</Emphasis> transcriptome during growth on the cellulase inducing substrates wheat straw and lactose. Biotechnol Biofuels. 2013;6:127.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-6-127>10.1186/1754-6834-6-127</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR66</td><td>BibArticle</td><td>Hemsworth GR, Henrissat B, Davies GJ, Walton PH. Discovery and characterization of a new family of lytic polysaccharide monooxygenases. Nat Chem Biol. 2014;10:122&#8211;6.</td><td><a href=http://dx.doi.org/10.1038/nchembio.1417>10.1038/nchembio.1417</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR67</td><td>BibArticle</td><td>Vaaje-Kolstad G, Westereng B, Horn SJ, Liu Z, Zhai H, S&#248;rlie M, Eijsink VGH. An oxidative enzyme boosting the enzymatic conversion of recalcitrant polysaccharides. Science. 2010;330:219&#8211;22.</td><td><a href=http://dx.doi.org/10.1126/science.1192231>10.1126/science.1192231</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, JournalTitle, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR68</td><td>BibArticle</td><td>Agger JW, Isaksen T, V&#225;rnai A, Vidal-Melgosa S, Willats WGT, Ludwig R, Horn SJ, Eijsink VGH, Westereng B. Discovery of LPMO activity on hemicelluloses shows the importance of oxidative processes in plant cell wall degradation. Proc Natl Acad Sci USA. 2014;111:6287&#8211;92.</td><td><a href=http://dx.doi.org/10.1073/pnas.1323629111>10.1073/pnas.1323629111</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR69</td><td>BibArticle</td><td>Leggio LL, Simmons TJ, Poulsen JC, Frandsen KE, Hemsworth GR, Stringer MA, von Freiesleben P, Tovborg M, Johansen KS, De Maria L, et al. Structure and boosting activity of a starch-degrading lytic polysaccharide monooxygenase. Nat Commun. 2015;6:5961.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/ncomms6961>10.1038/ncomms6961</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR70</td><td>BibArticle</td><td>Harris PV, Xu F, Kreel NE, Kang C, Fukuyama S. New enzyme insights drive advances in commercial ethanol production. Curr Opin Chem Biol. 2014;19:162&#8211;70.</td><td><a href=http://dx.doi.org/10.1016/j.cbpa.2014.02.015>10.1016/j.cbpa.2014.02.015</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR71</td><td>BibArticle</td><td>Hu J, Arantes V, Pribowo A, Saddler JN. The synergistic action of accessory enzymes enhances the hydrolytic potential of a &#8220;cellulase mixture&#8221; but is highly substrate specific. Biotechnol Biofuels. 2013;6:112.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-6-112>10.1186/1754-6834-6-112</a></td><td>&nbsp;</td><td><b>CrossRef-Web</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR72</td><td>BibArticle</td><td>Jabbour D, Borrusch MS, Banerjee G, Walton JD. Enhancement of fermentable sugar yields by &#945;-xylosidase supplementation of commercial cellulases. Biotechnol Biofuels. 2013;6:58.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-6-58>10.1186/1754-6834-6-58</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR73</td><td>BibArticle</td><td>Ye Z, Zheng Y, Li B, Borrusch MS, Storms R, Walton JD. Enhancement of synthetic <Emphasis Type="Italic">Trichoderma</Emphasis>-based enzyme mixtures for biomass conversion with an alternative family 5 glycosyl hydrolase from <Emphasis Type="Italic">Sporotrichum thermophile</Emphasis>. PLoS One. 2014;9:e109885.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0109885>10.1371/journal.pone.0109885</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR74</td><td>BibArticle</td><td>Akel E, Metz B, Seiboth B, Kubicek CP. Molecular regulation of arabinan and <Emphasis Type="SmallCaps">l</Emphasis>-arabinose metabolism in <Emphasis Type="Italic">Hypocrea jecorina</Emphasis> (<Emphasis Type="Italic">Trichoderma reesei</Emphasis>). Eukaryot Cell. 2009;8:1837&#8211;44.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/EC.00162-09>10.1128/EC.00162-09</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR75</td><td>BibArticle</td><td>Ries L, Pullan ST, Delmas S, Malla S, Blythe MJ, Archer DB. Genome-wide transcriptional response of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> to lignocellulose using RNA sequencing and comparison with <Emphasis Type="Italic">Aspergillus niger</Emphasis>. BMC Genom. 2013;14:541.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1186/1471-2164-14-541>10.1186/1471-2164-14-541</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR76</td><td>BibArticle</td><td>Ravalason H, Grisel S, Chevret D, Favel A, Berrin JG, Sigoillot JC, Herpoel-Gimbert I. <Emphasis Type="Italic">Fusarium verticillioides</Emphasis> secretome as a source of auxiliary enzymes to enhance saccharification of wheat straw. Bioresour Technol. 2012;114:589&#8211;96.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.biortech.2012.03.009>10.1016/j.biortech.2012.03.009</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR77</td><td>BibArticle</td><td>Kracher D, Oros D, Yao W, Preims M, Rezic I, Haltrich D, Rezic T, Ludwig R. Fungal secretomes enhance sugar beet pulp hydrolysis. Biotechnol J. 2014;9:483&#8211;92.</td><td><a href=http://dx.doi.org/10.1002/biot.201300214>10.1002/biot.201300214</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage, ArticleTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR78</td><td>BibArticle</td><td>Laothanachareon T, Bunterngsook B, Suwannarangsee S, Eurwilaichitr L, Champreda V. Synergistic action of recombinant accessory hemicellulolytic and pectinolytic enzymes to <Emphasis Type="Italic">Trichoderma reesei</Emphasis> cellulase on rice straw degradation. Bioresour Technol. 2015;198:682&#8211;90.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.biortech.2015.09.053>10.1016/j.biortech.2015.09.053</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR79</td><td>BibArticle</td><td>Levasseur A, Saloheimo M, Navarro D, Andberg M, Pontarotti P, Kruus K, Record E. Exploring laccase-like multicopper oxidase genes from the ascomycete <Emphasis Type="Italic">Trichoderma reesei</Emphasis>: a functional, phylogenetic and evolutionary study. BMC Biochem. 2010;11:32.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1186/1471-2091-11-32>10.1186/1471-2091-11-32</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR80</td><td>BibArticle</td><td>Glass LN, Schmoll M, Cate JHD, Coradetti S. Plant cell wall deconstruction by ascomycete fungi. Annu Rev Microbiol. 2013;67:477&#8211;98.</td><td><a href=http://dx.doi.org/10.1146/annurev-micro-092611-150044>10.1146/annurev-micro-092611-150044</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2, ArticleTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR81</td><td>BibArticle</td><td>Ilm&#233;n M, Thrane C, Penttil&#228; M. The glucose repressor gene <Emphasis Type="Italic">cre1</Emphasis> of <Emphasis Type="Italic">Trichoderma</Emphasis>: isolation and expression of a full length and a truncated mutant form. Mol Gen Genet. 1996;251:451&#8211;60.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR82</td><td>BibArticle</td><td>Mello-de-Sousa TM, Gorsche R, Rassinger A, Pocas-Fonseca MJ, Mach RL, Mach-Aigner AR. A truncated form of the carbon catabolite repressor 1 increases cellulase production in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Biotechnol Biofuels. 2014;7:129.</td><td><a href=http://dx.doi.org/10.1186/s13068-014-0129-3>10.1186/s13068-014-0129-3</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, Year, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR83</td><td>BibArticle</td><td>Seidl V, Gamauf C, Druzhinina IS, Seiboth B, Hartl L, Kubicek CP. The <Emphasis Type="Italic">Hypocrea jecorina</Emphasis> (<Emphasis Type="Italic">Trichoderma reesei</Emphasis>) hypercellulolytic mutant RUT C30 lacks a 85&nbsp;kb (29 gene-encoding) region of the wild-type genome. BMC Genom. 2008;9:327.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR84</td><td>BibArticle</td><td>Zou G, Shi S, Jiang Y, van den Brink J, de Vries RP, Chen L, Zhang J, Ma L, Wang C, Zhou Z. Construction of a cellulase hyper-expression system in <Emphasis Type="Italic">Trichoderma reesei</Emphasis> by promoter and enzyme engineering. Microb Cell Fact. 2012;11:1.</td><td><a href=http://dx.doi.org/10.1186/1475-2859-11-21>10.1186/1475-2859-11-21</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR85</td><td>BibArticle</td><td>Stricker AR, Grosstessner-Hain K, W&#252;rleitner E, Mach RL. Xyr1 (Xylanase regulator 1) regulates both the hydrolytic enzyme system and <Emphasis Type="SmallCaps">d</Emphasis>-xylose metabolism in <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>. Eukaryot Cell. 2006;5:2128&#8211;37.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/EC.00211-06>10.1128/EC.00211-06</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR86</td><td>BibArticle</td><td>Lichius A, Bidard F, Buchholz F, Le Crom S, Martin J, Schackwitz W, Austerlitz T, Grigoriev IV, Baker SE, Margeot A, et al. Genome sequencing of the <Emphasis Type="Italic">Trichoderma reesei</Emphasis> QM9136 mutant identifies a truncation of the transcriptional regulator XYR1 as the cause for its cellulase-negative phenotype. BMC Genom. 2015;16:326.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1186/s12864-015-1526-0>10.1186/s12864-015-1526-0</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR87</td><td>BibArticle</td><td>Portnoy T, Margeot A, Seidl-Seiboth V, Le Crom S, Chaabane FB, Linke R, Seiboth B, Kubicek CP. Differential regulation of the cellulase transcription factors XYR1, ACE2, and ACE1 in <Emphasis Type="Italic">Trichoderma reesei</Emphasis> strains producing high and low levels of cellulase. Eukaryot Cell. 2011;10:262&#8211;71.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/EC.00208-10>10.1128/EC.00208-10</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, FirstPage, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR88</td><td>BibArticle</td><td>Seiboth B, Karimi RA, Phatale PA, Linke R, Hartl L, Sauer DG, Smith KM, Baker SE, Freitag M, Kubicek CP. The putative protein methyltransferase LAE1 controls cellulase gene expression in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Mol Microbiol. 2012;84:1150&#8211;64.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1111/j.1365-2958.2012.08083.x>10.1111/j.1365-2958.2012.08083.x</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR89</td><td>BibArticle</td><td>Lv X, Zheng F, Li C, Zhang W, Chen G, Liu W. Characterization of a copper responsive promoter and its mediated overexpression of the xylanase regulator 1 results in an induction-independent production of cellulases in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Biotechnol Biofuels. 2015;8:67.</td><td><a href=http://dx.doi.org/10.1186/s13068-015-0249-4>10.1186/s13068-015-0249-4</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR90</td><td>BibArticle</td><td>Derntl C, Gudynaite-Savitch L, Calixte S, White T, Mach RL, Mach-Aigner AR. Mutation of the Xylanase regulator 1 causes a glucose blind hydrolase expressing phenotype in industrially used <Emphasis Type="Italic">Trichoderma</Emphasis> strains. Biotechnol Biofuels. 2013;6:62.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-6-62>10.1186/1754-6834-6-62</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR91</td><td>BibArticle</td><td>Aro N, Saloheimo A, Ilm&#233;n M, Penttil&#228; M. ACEII, a novel transcriptional activator involved in regulation of cellulase and xvlanase genes of <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. J Biol Chem. 2001;276:24309&#8211;14.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR92</td><td>BibArticle</td><td>Aro N, Ilm&#233;n M, Saloheimo A, Penttil&#228; M. ACEI of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> is a repressor of cellulase and xylanase expression. Appl Environ Microbiol. 2003;69:56&#8211;65.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/AEM.69.1.56-65.2003>10.1128/AEM.69.1.56-65.2003</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR93</td><td>BibArticle</td><td>H&#228;kkinen M, Valkonen M, Westerholm-Parvinen A, Aro N, Arvas M, Vitikainen M, Penttil&#228; M, Saloheimo M, Pakula T. Screening of candidate regulators for cellulase and hemicellulase production in <Emphasis Type="Italic">Trichoderma reesei</Emphasis> and identification of a factor essential for cellulase production. Biotechnol Biofuels. 2014;7:14.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR94</td><td>BibArticle</td><td>Larschan E, Winston F. The <Emphasis Type="Italic">S. cerevisiae</Emphasis> SAGA complex functions in vivo as a coactivator for transcriptional activation by Gal4. Genes Dev. 2001;15:1946&#8211;56.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1101/gad.911501>10.1101/gad.911501</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR95</td><td>BibArticle</td><td>Bhaumik SR, Green MR. SAGA is an essential in vivo target of the yeast acidic activator Gal4p. Genes Dev. 2001;15:1935&#8211;45.</td><td><a href=http://dx.doi.org/10.1101/gad.911401>10.1101/gad.911401</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR96</td><td>BibArticle</td><td>Xin Q, Gong Y, Lv X, Chen G, Liu W. <Emphasis Type="Italic">Trichoderma reesei</Emphasis> histone acetyltransferase Gcn5 regulates fungal growth, conidiation, and cellulase gene expression. Curr Microbiol. 2013;67:580&#8211;9.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1007/s00284-013-0396-4>10.1007/s00284-013-0396-4</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR97</td><td>BibArticle</td><td>Li Z, Yao G, Wu R, Gao L, Kan Q, Liu M, Yang P, Liu G, Qin Y, Song X, et al. Synergistic and dose-controlled regulation of cellulase gene expression in <Emphasis Type="Italic">Penicillium oxalicum</Emphasis>. PLoS Genet. 2015;11:e1005509.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1005509>10.1371/journal.pgen.1005509</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR98</td><td>BibArticle</td><td>Bok JW, Keller NP. LaeA, a regulator of secondary metabolism in <Emphasis Type="Italic">Aspergillus spp</Emphasis>. Eukaryot Cell. 2004;3:527&#8211;35.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/EC.3.2.527-535.2004>10.1128/EC.3.2.527-535.2004</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR99</td><td>BibArticle</td><td>Sarikaya-Bayram &#214;, Palmer JM, Keller N, Braus GH, Bayram &#214;. One Juliet and four Romeos: VeA and its methyltransferases. Front Microbiol. 2015;6:1.</td><td><a href=http://dx.doi.org/10.3389/fmicb.2015.00001>10.3389/fmicb.2015.00001</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR100</td><td>BibArticle</td><td>Aghcheh RK, N&#233;meth Z, Atanasova L, Fekete E, Paholcsek M, S&#225;ndor E, Aquino B, Druzhinina IS, Karaffa L, Kubicek CP. The VELVET a orthologue VEL1 of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> regulates fungal development and is essential for cellulase gene expression. PLoS One. 2014;9:e112799.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR101</td><td>BibArticle</td><td>Linder MB, Qiao M, Laumen F, Selber K, Hyytia T, Nakari-Set&#228;l&#228; T, Penttil&#228; ME. Efficient purification of recombinant proteins using hydrophobins as tags in surfactant-based two-phase systems. Biochemistry. 2004;43:11873&#8211;82.</td><td><a href=http://dx.doi.org/10.1021/bi0488202>10.1021/bi0488202</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR102</td><td>BibArticle</td><td>Smith W, Jantti J, Oja M, Saloheimo M. Comparison of intracellular and secretion-based strategies for production of human alpha-galactosidase A in the filamentous fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. BMC Biotechnol. 2014;14:91.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR103</td><td>BibArticle</td><td>Landowski CP, Huuskonen A, Wahl R, Westerholm-Parvinen A, Kanerva A, Hanninen AL, Salovuori N, Penttil&#228; M, Natunen J, Ostermeier C, et al. Enabling low cost biopharmaceuticals: a systematic approach to delete proteases from a well-known protein production host <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. PLoS One. 2015;10:e0134723.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0134723>10.1371/journal.pone.0134723</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR104</td><td>BibArticle</td><td>Nevalainen H, Peterson R. Making recombinant proteins in filamentous fungi&#8212;are we expecting too much? Front Microbiol. 2014;5:75.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR105</td><td>BibArticle</td><td>Linder MB, Szilvay GR, Nakari-Set&#228;l&#228; T, Penttil&#228; ME. Hydrophobins: the protein-amphiphiles of filamentous fungi. FEMS Microbiol Rev. 2005;29:877&#8211;96.</td><td><a href=http://dx.doi.org/10.1016/j.femsre.2005.01.004>10.1016/j.femsre.2005.01.004</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR106</td><td>BibArticle</td><td>W&#246;sten HA, Scholtmeijer K. Applications of hydrophobins: current state and perspectives. Appl Microbiol Biotechnol. 2015;99:1587&#8211;97.</td><td><a href=http://dx.doi.org/10.1007/s00253-014-6319-x>10.1007/s00253-014-6319-x</a></td><td>&nbsp;</td><td><b>CrossRef-Web</b></td>
<td>Author_FamilyName_2, FirstPage, Year, JournalTitle, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR107</td><td>BibArticle</td><td>Gaderer R, Bonazza K, Seidl-Seiboth V. Cerato-platanins: a fungal protein family with intriguing properties and application potential. Appl Microbiol Biotechnol. 2014;98:4795&#8211;803.</td><td><a href=http://dx.doi.org/10.1007/s00253-014-5690-y>10.1007/s00253-014-5690-y</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID, JournalTitle, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR108</td><td>BibArticle</td><td>Thongekkaew J, Ikeda H, Masaki K, Iefuji H. Fusion of cellulose binding domain from <Emphasis Type="Italic">Trichoderma reesei</Emphasis> CBHI to <Emphasis Type="Italic">Cryptococcus sp.</Emphasis> S-2 cellulase enhances its binding affinity and its cellulolytic activity to insoluble cellulosic substrates. Enzyme Microb Technol. 2013;52:241&#8211;6.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.enzmictec.2013.02.002>10.1016/j.enzmictec.2013.02.002</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR109</td><td>BibArticle</td><td>Tang CD, Li JF, Wei XH, Min R, Gao SJ, Wang JQ, Yin X, Wu MC. Fusing a carbohydrate-binding module into the <Emphasis Type="Italic">Aspergillus usamii</Emphasis> &#946;-mannanase to improve its thermostability and cellulose-binding capacity by in silico design. PLoS One. 2013;8:e64766.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0064766>10.1371/journal.pone.0064766</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR110</td><td>BibArticle</td><td>Sugimoto N, Igarashi K, Samejima M. Cellulose affinity purification of fusion proteins tagged with fungal family 1 cellulose-binding domain. Protein Expr Purif. 2012;82:290&#8211;6.</td><td><a href=http://dx.doi.org/10.1016/j.pep.2012.01.007>10.1016/j.pep.2012.01.007</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR111</td><td>BibArticle</td><td>Oliveira C, Carvalho V, Domingues L, Gama FM. Recombinant CBM-fusion technology&#8212;applications overview. Biotechnol Adv. 2015;33:358&#8211;69.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR112</td><td>BibArticle</td><td>Brethauer S, Studer MH. Consolidated bioprocessing of lignocellulose by a microbial consortium. Energy Environ Sci. 2014;7:1446&#8211;53.</td><td><a href=http://dx.doi.org/10.1039/c3ee41753k>10.1039/c3ee41753k</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR113</td><td>BibArticle</td><td>Minty JJ, Singer ME, Scholz SA, Bae CH, Ahn JH, Foster CE, Liao JC, Lin XN. Design and characterization of synthetic fungal-bacterial consortia for direct production of isobutanol from cellulosic biomass. Proc Natl Acad Sci USA. 2013;110:14592&#8211;7.</td><td><a href=http://dx.doi.org/10.1073/pnas.1218447110>10.1073/pnas.1218447110</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR114</td><td>BibArticle</td><td>Zuroff TR, Curtis WR. Developing symbiotic consortia for lignocellulosic biofuel production. Appl Microbiol Biotechnol. 2012;93:1423&#8211;35.</td><td><a href=http://dx.doi.org/10.1007/s00253-011-3762-9>10.1007/s00253-011-3762-9</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR115</td><td>BibArticle</td><td>Yamada R, Hasunuma T, Kondo A. Endowing non-cellulolytic microorganisms with cellulolytic activity aiming for consolidated bioprocessing. Biotechnol Adv. 2013;31:754&#8211;63.</td><td><a href=http://dx.doi.org/10.1016/j.biotechadv.2013.02.007>10.1016/j.biotechadv.2013.02.007</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR116</td><td>BibArticle</td><td>den Haan R, van Rensburg E, Rose SH, G&#246;rgens JF, van Zyl WH. Progress and challenges in the engineering of non-cellulolytic microorganisms for consolidated bioprocessing. Curr Opin Biotechnol. 2015;33:15&#8211;22.</td><td><a href=http://dx.doi.org/10.1016/j.copbio.2014.10.003>10.1016/j.copbio.2014.10.003</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR117</td><td>BibChapter</td><td>van Zyl W, den Haan R, la Grange D. Developing cellulolytic organisms for consolidated bioprocessing of lignocellulosics. In: Gupta VK, Tuohy MG, editors. Biofuel Technologies. Berlin: Springer; 2013. p. 189&#8211;220.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1007/978-3-642-34519-7>10.1007/978-3-642-34519-7</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, BookTitle</td><td>CrossRef</td></tr><tr><td>CR118</td><td>BibArticle</td><td>Amore A, Faraco V. Potential of fungi as category I consolidated bioprocessing organisms for cellulosic ethanol production. Renew Sust Energ Rev. 2012;16:3286&#8211;301.</td><td><a href=http://dx.doi.org/10.1016/j.rser.2012.02.050>10.1016/j.rser.2012.02.050</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR119</td><td>BibArticle</td><td>Kroukamp H, den Haan R, van Wyk N, van Zyl WH. Overexpression of native <Emphasis Type="Italic">PSE</Emphasis>1 and <Emphasis Type="Italic">SOD</Emphasis>1 in <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis> improved heterologous cellulase secretion. Appl Energ. 2013;102:150&#8211;6.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.apenergy.2012.05.062>10.1016/j.apenergy.2012.05.062</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR120</td><td>BibArticle</td><td>Inokuma K, Hasunuma T, Kondo A. Efficient yeast cell-surface display of exo- and endo-cellulase using the <Emphasis Type="Italic">SED1</Emphasis> anchoring region and its original promoter. Biotechnol Biofuels. 2014;7:8.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-7-8>10.1186/1754-6834-7-8</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR121</td><td>BibArticle</td><td>Lambertz C, Garvey M, Klinger J, Heesel D, Klose H, Fischer R, Commandeur U. Challenges and advances in the heterologous expression of cellulolytic enzymes: a review. Biotechnol Biofuels. 2014;7:135.</td><td><a href=http://dx.doi.org/10.1186/s13068-014-0135-5>10.1186/s13068-014-0135-5</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, Year, JournalTitle, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR122</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>McBride</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Brevnova</span> <span style='background:#DDDDDD'>E</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Ghandi</span> <span style='background:#DDDDDD'>C</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Mellon</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Froehlich</span> <span style='background:#DDDDDD'>A</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Deleault</span> <span style='background:#DDDDDD'>K</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Rajgharia</span> <span style='background:#DDDDDD'>V</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Flatt</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Van</span> <span style='background:#DDDDDD'>ZE</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Den</span> <span style='background:#DDDDDD'>HR</span></span></aug>. <span style='background:#FFD9B3'>Yeast expressing cellulases for simultaneous saccharification and fermentation using cellulose</span>. <span style='background:#66FF66'>2010</span>. <span style='background:#FFFF0F'>International patent WO2010060056</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR123</td><td>BibArticle</td><td>Xu Q, Singh A, Himmel ME. Perspectives and new directions for the production of bioethanol using consolidated bioprocessing of lignocellulose. Curr Opin Biotechnol. 2009;20:364&#8211;71.</td><td><a href=http://dx.doi.org/10.1016/j.copbio.2009.05.006>10.1016/j.copbio.2009.05.006</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_2, Author_FamilyName_1, Year, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR124</td><td>BibChapter</td><td>Bischof R, Seiboth B. Molecular tools for strain improvement of <Emphasis Type="Italic">Trichoderma spp</Emphasis>. In: Gupta VG, Schmoll M, Herrera-Estrella A, Upadhyay RS, Druzhinina I, Tuohy M, editors. Biotechnology and biology of <Emphasis Type="Italic">Trichoderma</Emphasis>. Oxford: Elsevier; 2014. p. 179&#8211;91.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR125</td><td>BibArticle</td><td>Rautio JJ, Smit BA, Wiebe M, Pentill&#228; M, Saloheimo M. Transcriptional monitoring of steady state and effects of anaerobic phases in chemostat cultures of the filamentous fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. BMC Genom. 2006;7:247.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR126</td><td>BibArticle</td><td>Bonaccorsi ED, Ferreira AJS, Chambergo FS, Ramos ASP, Mantovani MC, Farah JPS, Sorio CS, Gombert AK, Tonso A, El-Dorry H. Transcriptional response of the obligatory aerobe <Emphasis Type="Italic">Trichoderma reesei</Emphasis> to hypoxia and transient anoxia: implications for energy production and survival in the absence of oxygen. Biochemistry. 2006;45:3912&#8211;24.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR127</td><td>BibArticle</td><td>Haab D, Gassner B, Kubicek CP. Protein hypersecretory <Emphasis Type="Italic">Trichoderma reesei</Emphasis> mutant RUT C-30 displays increased ethanol and polyene resistance. J Biotechnol. 1993;29:97&#8211;108.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/0168-1656(93)90043-M>10.1016/0168-1656(93)90043-M</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR128</td><td>BibArticle</td><td>Peciulyte A, Anasontzis GE, Karlstr&#246;m K, Larsson PT, Olsson L. Morphology and enzyme production of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> Rut C-30 are affected by the physical and structural characteristics of cellulosic substrates. Fungal Genet Biol. 2014;72:64&#8211;72.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.fgb.2014.07.011>10.1016/j.fgb.2014.07.011</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR129</td><td>BibArticle</td><td>Callow NV, Ju LK. Promoting pellet growth of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> Rut C30 by surfactants for easy separation and enhanced cellulase production. Enzyme Microb Technol. 2012;50:311&#8211;7.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.enzmictec.2012.02.006>10.1016/j.enzmictec.2012.02.006</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR130</td><td>BibArticle</td><td>Lynd LR, Van Zyl WH, McBride JE, Laser M. Consolidated bioprocessing of cellulosic biomass: an update. Curr Opin Biotechnol. 2005;16:577&#8211;83.</td><td><a href=http://dx.doi.org/10.1016/j.copbio.2005.08.009>10.1016/j.copbio.2005.08.009</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, ArticleTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR131</td><td>BibChapter</td><td>Steiger MG. Molecular tools in <Emphasis Type="Italic">Trichoderma</Emphasis> genetic studies. In: Mukherjee PK, Horwitz BA, Singh US, Mukherjee M, Schmoll M, editors. <Emphasis Type="Italic">Trichoderma</Emphasis>: biology and applications. Oxfordshire: CABI; 2013. p. 128&#8211;43.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1079/9781780642475.0000>10.1079/9781780642475.0000</a></td><td><b>CrossRef-Meta</b></td>
<td>Year</td><td>CrossRef</td></tr><tr><td>CR132</td><td>BibArticle</td><td>Guangtao Z, Hartl L, Schuster A, Polak S, Schmoll M, Wang T, Seidl V, Seiboth B. Gene targeting in a nonhomologous end joining deficient <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>. J Biotechnol. 2009;139:146&#8211;51.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.jbiotec.2008.10.007>10.1016/j.jbiotec.2008.10.007</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage, VolumeID</td><td>CrossRef</td></tr><tr><td>CR133</td><td>BibArticle</td><td>Steiger MG, Vitikainen M, Uskonen P, Brunner K, Adam G, Pakula T, Penttil&#228; M, Saloheimo M, Mach RL, Mach-Aigner AR. Transformation system for <Emphasis Type="Italic">Hypocrea jecorina</Emphasis> (<Emphasis Type="Italic">Trichoderma reesei</Emphasis>) that favors homologous integration and employs reusable bidirectionally selectable markers. Appl Environ Microbiol. 2011;77:114&#8211;21.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/AEM.02100-10>10.1128/AEM.02100-10</a></td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, VolumeID</td><td>CrossRef</td></tr><tr><td>CR134</td><td>BibArticle</td><td>Schuster A, Bruno KS, Collett JR, Baker SE, Seiboth B, Kubicek CP, Schmoll M. A versatile toolkit for high throughput functional genomics with <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Biotechnol Biofuels. 2012;5:1.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-5-1>10.1186/1754-6834-5-1</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR135</td><td>BibArticle</td><td>J&#248;rgensen M, Skovlund D, Johannesen P, Mortensen U. A novel platform for heterologous gene expression in <Emphasis Type="Italic">Trichoderma reesei</Emphasis> (Teleomorph <Emphasis Type="Italic">Hypocrea jecorina</Emphasis>). Microb Cell Fact. 2014;13:33.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR136</td><td>BibArticle</td><td>Derntl C, Kiesenhofer DP, Mach RL, Mach-Aigner AR. Novel strategies for genomic manipulation of <Emphasis Type="Italic">Trichoderma reesei</Emphasis> with the purpose of strain engineering. Appl Environ Microbiol. 2015;81:6314&#8211;23.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/AEM.01545-15>10.1128/AEM.01545-15</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR137</td><td>BibArticle</td><td>Ouedraogo JP, Arentshorst M, Nikolaev I, Barends S, Ram AF. I-<Emphasis Type="Italic">Sce</Emphasis>I-mediated double-strand DNA breaks stimulate efficient gene targeting in the industrial fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Appl Microbiol Biotechnol. 2015;99:10083&#8211;95.</td><td><a href=http://dx.doi.org/10.1007/s00253-015-6829-1>10.1007/s00253-015-6829-1</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR138</td><td>BibArticle</td><td>Ouedraogo JP, Arentshorst M, Nikolaev I, Barends S, Ram AF. I-<Emphasis Type="Italic">Sce</Emphasis>I enzyme mediated integration (SEMI) for fast and efficient gene targeting in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. J Biotechnol. 2016;222:25&#8211;8.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.jbiotec.2016.02.012>10.1016/j.jbiotec.2016.02.012</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR139</td><td>BibArticle</td><td>Sander JD, Joung JK. CRISPR-Cas systems for editing, regulating and targeting genomes. Nat Biotechnol. 2014;32:347&#8211;55.</td><td><a href=http://dx.doi.org/10.1038/nbt.2842>10.1038/nbt.2842</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID, JournalTitle, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR140</td><td>BibArticle</td><td>Liu R, Chen L, Jiang Y, Zhou Z, Zou G. Efficient genome editing in filamentous fungus <Emphasis Type="Italic">Trichoderma reesei</Emphasis> using the CRISPR/Cas9 system. Cell Discov. 2015;1:15007.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/celldisc.2015.7>10.1038/celldisc.2015.7</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR141</td><td>BibArticle</td><td>N&#248;dvig CS, Nielsen JB, Kogle ME, Mortensen UH. A CRISPR-Cas9 system for genetic engineering of filamentous fungi. PLoS One. 2015;10:e0133085.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0133085>10.1371/journal.pone.0133085</a></td><td>&nbsp;</td><td><b>CrossRef-Web</b></td>
<td>FirstPage, Author_FamilyName_2, Year, VolumeID, JournalTitle, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR142</td><td>BibArticle</td><td>Banerjee G, Car S, Scott-Craig JS, Borrusch MS, Walton JD. Rapid optimization of enzyme mixtures for deconstruction of diverse pretreatment/biomass feedstock combinations. Biotechnol Biofuels. 2010;3:22.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-3-22>10.1186/1754-6834-3-22</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage, ArticleTitle, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR143</td><td>BibArticle</td><td>Dettmann A, Heilig Y, Valerius O, Ludwig S, Seiler S. Fungal communication requires the MAK-2 pathway elements STE-20 and RAS-2, the NRC-1 adapter STE-50 and the MAP kinase scaffold HAM-5. PLoS Genet. 2014;10:e1004762.</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1004762>10.1371/journal.pgen.1004762</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, ArticleTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR144</td><td>BibArticle</td><td>Jonkers W, Leeder AC, Ansong C, Wang Y, Yang F, Starr TL, Camp DG 2nd, Smith RD, Glass NL. HAM-5 functions as a MAP kinase scaffold during cell fusion in <Emphasis Type="Italic">Neurospora crassa</Emphasis>. PLoS Genet. 2014;10:e1004783.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1371/journal.pgen.1004783>10.1371/journal.pgen.1004783</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR145</td><td>BibArticle</td><td>Chuang YC, Li WC, Chen CL, Hsu PW, Tung SY, Kuo HC, Schmoll M, Wang TF. <Emphasis Type="Italic">Trichoderma reesei</Emphasis> meiosis generates segmentally aneuploid progeny with higher xylanase-producing capability. Biotechnol Biofuels. 2015;8:30.</td><td><a href=http://dx.doi.org/10.1186/s13068-015-0202-6>10.1186/s13068-015-0202-6</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, VolumeID, Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR146</td><td>BibArticle</td><td>Nitta M, Furukawa T, Shida Y, Mori K, Kuhara S, Morikawa Y, Ogasawara W. A new Zn(II)2Cys6-type transcription factor BglR regulates &#946;-glucosidase expression in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. Fungal Genet Biol. 2012;49:388&#8211;97.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.fgb.2012.02.009>10.1016/j.fgb.2012.02.009</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR147</td><td>BibArticle</td><td>Pomraning KR, Smith KM, Freitag M. Bulk segregant analysis followed by high-throughput sequencing reveals the <Emphasis Type="Italic">Neurospora</Emphasis> cell cycle gene, <Emphasis Type="Italic">ndc</Emphasis>-<Emphasis Type="Italic">1</Emphasis>, to be allelic with the gene for ornithine decarboxylase, <Emphasis Type="Italic">spe</Emphasis>-<Emphasis Type="Italic">1</Emphasis>. Eukaryot Cell. 2011;10:724&#8211;33.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/EC.00016-11>10.1128/EC.00016-11</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year, VolumeID</td><td>CrossRef</td></tr><tr><td>CR148</td><td>BibArticle</td><td>Niu J, Arentshorst M, Nair PDS, Dai Z, Baker SE, Frisvad JC, Nielsen KF, Punt PJ, Ram AFJ. Identification of a classical mutant in the industrial host <Emphasis Type="Italic">Aspergillus niger</Emphasis> by systems genetics: LaeA is required for citric acid production and regulates the formation of some secondary metabolites. G3 (Bethesda). 2016;6:193&#8211;204.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1534/g3.115.024067>10.1534/g3.115.024067</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR149</td><td>BibArticle</td><td>Nowrousian M, Teichert I, Masloff S, K&#252;ck U. Whole-genome sequencing of <Emphasis Type="Italic">Sordaria macrospora</Emphasis> mutants identifies developmental genes. G3 (Bethesda). 2012;2:261&#8211;70.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1534/g3.111.001479>10.1534/g3.111.001479</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage, VolumeID</td><td>CrossRef</td></tr><tr><td>CR150</td><td>BibArticle</td><td>Bok JW, Wiemann P, Garvey GS, Lim FY, Haas B, Wortman J, Keller NP. Illumina identification of RsrA, a conserved C2H2 transcription factor coordinating the NapA mediated oxidative stress signaling pathway in <Emphasis Type="Italic">Aspergillus</Emphasis>. BMC Genom. 2014;15:1011.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1186/1471-2164-15-1011>10.1186/1471-2164-15-1011</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR151</td><td>BibArticle</td><td>Hasunuma T, Hori Y, Sakamoto T, Ochiai M, Hatanaka H, Kondo A. Development of a GIN11/FRT-based multiple-gene integration technique affording inhibitor-tolerant, hemicellulolytic, xylose-utilizing abilities to industrial <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis> strains for ethanol production from undetoxified lignocellulosic hemicelluloses. Microb Cell Fact. 2014;13:145.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR152</td><td>BibArticle</td><td>Cherry JR, Fidantsef AL. Directed evolution of industrial enzymes: an update. Curr Opin Biotechnol. 2003;14:438&#8211;43.</td><td><a href=http://dx.doi.org/10.1016/S0958-1669(03)00099-5>10.1016/S0958-1669(03)00099-5</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_1, Author_FamilyName_2, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR153</td><td>BibArticle</td><td>Freier D, Mothershed CP, Wiegel J. Characterization of <Emphasis Type="Italic">Clostridium thermocellum</Emphasis> JW20. Appl Environ Microbiol. 1988;54:204&#8211;11.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR154</td><td>BibArticle</td><td>Goshima T, Negi K, Tsuji M, Inoue H, Yano S, Hoshino T, Matsushika A. Ethanol fermentation from xylose by metabolically engineered strains of <Emphasis Type="Italic">Kluyveromyces marxianus</Emphasis>. J Biosci Bioeng. 2013;116:551&#8211;4.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.jbiosc.2013.05.010>10.1016/j.jbiosc.2013.05.010</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage, VolumeID</td><td>CrossRef</td></tr><tr><td>CR155</td><td>BibArticle</td><td>Rodrussamee N, Lertwattanasakul N, Hirata K, Suprayogi, Limtong S, Kosaka T, Yamada M. Growth and ethanol fermentation ability on hexose and pentose sugars and glucose effect under various conditions in thermotolerant yeast <Emphasis Type="Italic">Kluyveromyces marxianus</Emphasis>. Appl Microbiol Biotechnol. 2011;90:1573&#8211;86.</td><td><a href=http://dx.doi.org/10.1007/s00253-011-3218-2>10.1007/s00253-011-3218-2</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR156</td><td>BibBook</td><td>Kubicek CP. Lignocellulose biorefinery. Hoboken: Wiley-Blackwell; 2013.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1002/9781118414514.ch11>10.1002/9781118414514.ch11</a></td><td><b>CrossRef-Meta</b></td>
<td>BookTitle, PublisherName</td><td>CrossRef</td></tr><tr><td>CR157</td><td>BibArticle</td><td>Maehara T, Ichinose H, Furukawa T, Ogasawara W, Takabatake K, Kaneko S. Ethanol production from high cellulose concentration by the basidiomycete fungus <Emphasis Type="Italic">Flammulina velutipes</Emphasis>. Fungal Biol. 2013;117:220&#8211;6.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.funbio.2013.02.002>10.1016/j.funbio.2013.02.002</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1, FirstPage</td><td>CrossRef</td></tr><tr><td>CR158</td><td>BibArticle</td><td>El-Batal AI, Farahat LM, El-Rehim HA. Ethanol production by <Emphasis Type="Italic">Kluyveromyces lactis</Emphasis> immobilized cells in copolymer carriers produced by radiation polymerization. Acta Microbiol Pol. 2000;49:157&#8211;66.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR159</td><td>BibArticle</td><td>Ohta K, Beall DS, Mejia JP, Shanmugam KT, Ingram LO. Genetic improvement of <Emphasis Type="Italic">Escherichia coli</Emphasis> for ethanol production: chromosomal integration of <Emphasis Type="Italic">Zymomonas mobilis</Emphasis> genes encoding pyruvate decarboxylase and alcohol dehydrogenase II. Appl Environ Microbiol. 1991;57:893&#8211;900.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR160</td><td>BibArticle</td><td>Fujita Y, Takahashi S, Ueda M, Tanaka A, Okada H, Morikawa Y, Kawaguchi T, Arai M, Fukuda H, Kondo A. Direct and efficient production of ethanol from cellulosic material with a yeast strain displaying cellulolytic enzymes. Appl Environ Microbiol. 2002;68:5136&#8211;41.</td><td><a href=http://dx.doi.org/10.1128/AEM.68.10.5136-5141.2002>10.1128/AEM.68.10.5136-5141.2002</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, FirstPage, Author_FamilyName_1, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR161</td><td>BibArticle</td><td>Katahira S, Fujita Y, Mizuike A, Fukuda H, Kondo A. Construction of a xylan-fermenting yeast strain through codisplay of xylanolytic enzymes on the surface of xylose-utilizing <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis> cells. Appl Environ Microbiol. 2004;70:5407&#8211;14.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1128/AEM.70.9.5407-5414.2004>10.1128/AEM.70.9.5407-5414.2004</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, FirstPage, Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR162</td><td>BibArticle</td><td>Fujita Y, Ito J, Ueda M, Fukuda H, Kondo A. Synergistic saccharification, and direct fermentation to ethanol, of amorphous cellulose by use of an engineered yeast strain codisplaying three types of cellulolytic enzyme. Appl Environ Microbiol. 2004;70:1207&#8211;12.</td><td><a href=http://dx.doi.org/10.1128/AEM.70.2.1207-1212.2004>10.1128/AEM.70.2.1207-1212.2004</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR163</td><td>BibArticle</td><td>Den Haan R, McBride JE, Grange DCL, Lynd LR, Van Zyl WH. Functional expression of cellobiohydrolases in <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis> towards one-step conversion of cellulose to ethanol. Enzyme Microb Technol. 2007;40:1291&#8211;9.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/j.enzmictec.2006.09.022>10.1016/j.enzmictec.2006.09.022</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR164</td><td>BibArticle</td><td>Yamada R, Taniguchi N, Tanaka T, Ogino C, Fukuda H, Kondo A. Direct ethanol production from cellulosic materials using a diploid strain of <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis> with optimized cellulase expression. Biotechnol Biofuels. 2011;4:1&#8211;8.</td><td><a href=http://dx.doi.org/10.1186/1754-6834-4-8>10.1186/1754-6834-4-8</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_1, Author_FamilyName_2, JournalTitle, VolumeID</td><td>CrossRef</td></tr><tr><td>CR165</td><td>BibArticle</td><td>Ilm&#233;n M, den Haan R, Brevnova E, McBride J, Wiswall E, Froehlich A, Koivula A, Voutilainen SP, Siika-aho M, la Grange DC, et al. High level secretion of cellobiohydrolases by <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis>. Biotechnol Biofuels. 2011;4:1&#8211;15.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR166</td><td>BibArticle</td><td>Nakatani Y, Yamada R, Ogino C, Kondo A. Synergetic effect of yeast cell-surface expression of cellulase and expansin-like protein on direct ethanol production from cellulose. Microb Cell Fact. 2013;12:66.</td><td><a href=http://dx.doi.org/10.1186/1475-2859-12-66>10.1186/1475-2859-12-66</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR167</td><td>BibArticle</td><td>Matano Y, Hasunuma T, Kondo A. Simultaneous improvement of saccharification and ethanol production from crystalline cellulose by alleviation of irreversible adsorption of cellulase with a cell surface-engineered yeast strain. Appl Microbiol Biotechnol. 2013;97:2231&#8211;7.</td><td><a href=http://dx.doi.org/10.1007/s00253-012-4587-x>10.1007/s00253-012-4587-x</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, JournalTitle, ArticleTitle, Author_FamilyName_1, Author_FamilyName_2, FirstPage</td><td>CrossRef</td></tr><tr><td>CR168</td><td>BibArticle</td><td>Liu Z, Inokuma K, Ho S-H, Haan Rd, Hasunuma T, Zyl WH, Kondo A. Combined cell-surface display- and secretion-based strategies for production of cellulosic ethanol with <Emphasis Type="Italic">Saccharomyces cerevisiae</Emphasis>. Biotechnol Biofuels. 2015;8:1&#8211;12.</td><td><a href=http://dx.doi.org/10.1186/s13068-015-0344-6>10.1186/s13068-015-0344-6</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR169</td><td>BibArticle</td><td>Tao L, Dong HJ, Chen X, Chen SF, Wang TH. Expression of ethylene-forming enzyme (EFE) of <Emphasis Type="Italic">Pseudomonas syringae pv. glycinea</Emphasis> in <Emphasis Type="Italic">Trichoderma viride</Emphasis>. Appl Microbiol Biotechnol. 2008;80:573&#8211;8.</td><td><a href=http://dx.doi.org/10.1007/s00253-008-1562-7>10.1007/s00253-008-1562-7</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, FirstPage, Year, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR170</td><td>BibArticle</td><td>Chen X, Liang Y, Hua J, Tao L, Qin W, Chen S. Overexpression of bacterial ethylene-forming enzyme gene in <Emphasis Type="Italic">Trichoderma reesei</Emphasis> enhanced the production of ethylene. Int J Biol Sci. 2010;6:96&#8211;106.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR171</td><td>BibArticle</td><td>Steiger MG, Mach-Aigner AR, Gorsche R, Rosenberg EE, Mihovilovic MD, Mach RL. Synthesis of an antiviral drug precursor from chitin using a saprophyte as a whole-cell catalyst. Microb Cell Fact. 2011;10:1.</td><td><a href=http://dx.doi.org/10.1186/1475-2859-10-102>10.1186/1475-2859-10-102</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2, Year, JournalTitle, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR172</td><td>BibArticle</td><td>Dashtban M, Kepka G, Seiboth B, Qin W. Xylitol production by genetically engineered <Emphasis Type="Italic">Trichoderma reesei</Emphasis> strains using barley straw as feedstock. Appl Biochem Biotechnol. 2013;169:554&#8211;69.</td><td><a href=http://dx.doi.org/10.1007/s12010-012-0008-y>10.1007/s12010-012-0008-y</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, Author_FamilyName_2, VolumeID, JournalTitle</td><td>CrossRef</td></tr><tr><td>CR173</td><td>BibArticle</td><td>Hong Y, Dashtban M, Kepka G, Chen S, Qin W. Overexpression of <Emphasis Type="SmallCaps">d</Emphasis>-xylose reductase (<Emphasis Type="Italic">xyl1</Emphasis>) gene and antisense inhibition of <Emphasis Type="SmallCaps">d</Emphasis>-xylulokinase (<Emphasis Type="Italic">xyiH</Emphasis>) gene increase xylitol production in <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. BioMed Res Int. 2014;2014:8.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1155/2014/169705>10.1155/2014/169705</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Year, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR174</td><td>BibArticle</td><td>Jovanovi&#263; B, Mach RL, Mach-Aigner AR. Erythritol production on wheat straw using <Emphasis Type="Italic">Trichoderma reesei</Emphasis>. AMB Expr. 2014;4:1&#8211;12.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1186/s13568-014-0034-y>10.1186/s13568-014-0034-y</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, VolumeID, JournalTitle</td><td>CrossRef</td></tr></table>