<H1>Chapter DOI: 10.1186/s12864-016-2774-3<br/>Cited-By Count: 0</H1><table border="1" width="30%"><tr><td>Total References</td><td>51</td></tr><tr><td>Springer references</td><td>21</td></tr><tr><td>Non Springer references</td><td>30</td></tr><tr><td>BibStructured Count</td><td width="10%">51</td></tr><tr><td>BibUnstructured Count</td><td width="10%">0</td></tr><tr><td>DOI already available in SpringerLink</td><td>49</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>1</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>0</td></tr><tr><td>DOI Obtained For Non Springer</td><td>1</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>1</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>0</td></tr><tr><td>Valid DOI Enriched</td><td>0</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibArticle</td><td>Lewin HA, Larkin DM, Pontius J, O&#8217;Brien SJ. Every genome sequence needs a good map. Genome Res. 2009;19:1925&#8211;8.</td><td><a href=http://dx.doi.org/10.1101/gr.094557.109>10.1101/gr.094557.109</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Ferguson-Smith MA, Trifonov V. Mammalian karyotype evolution. Nat Rev Genet. 2007;8:950&#8211;62.</td><td><a href=http://dx.doi.org/10.1038/nrg2199>10.1038/nrg2199</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibArticle</td><td>Griffin DK, Robertson LBW, Tempest HG, Skinner BM. The evolution of the avian genome as revealed by comparative molecular cytogenetics. Cytogenet Genome Res. 2007;117:64&#8211;77.</td><td><a href=http://dx.doi.org/10.1159/000103166>10.1159/000103166</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Deakin JE, Delbridge ML, Koina E, Harley N, Alsop AE, Wang C, et al. Reconstruction of the ancestral marsupial karyotype from comparative gene maps. BMC Evol Biol. 2013;13:258.</td><td><a href=http://dx.doi.org/10.1186/1471-2148-13-258>10.1186/1471-2148-13-258</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR5</td><td>BibArticle</td><td>Uno Y, Nishida C, Tarui H, Ishishita S, Takagi C, Nishimura O, et al. Inference of the protokaryotypes of amniotes and tetrapods and the evolutionary processes of microchromosomes from comparative gene mapping. PLoS One. 2012;7:e53027.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0053027>10.1371/journal.pone.0053027</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Deakin JE, Ezaz T. Tracing the evolution of amniote chromosomes. Chromosoma. 2014;123:201&#8211;16.</td><td><a href=http://dx.doi.org/10.1007/s00412-014-0456-y>10.1007/s00412-014-0456-y</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Giovannotti M, Caputo V, O&#8217;Brien PCM, Lovell FL, Trifonov V, Nisi Cerioni P, et al. Skinks (reptilia: Scincidae) have highly conserved karyotypes as revealed by chromosome painting. Cytogenet Genome Res. 2010;127:224&#8211;31.</td><td><a href=http://dx.doi.org/10.1159/000295002>10.1159/000295002</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Trifonov VA, Giovannotti M, O&#8217;Brien PCM, Wallduck M, Lovell F, Rens W, et al. Chromosomal evolution in Gekkonidae. I. Chromosome painting between Gekko and Hemidactylus species reveals phylogenetic relationships within the group. Chromosome Res. 2011;19:843&#8211;55.</td><td><a href=http://dx.doi.org/10.1007/s10577-011-9241-4>10.1007/s10577-011-9241-4</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Pokorn&#225; M, Giovannotti M, Kratochv&#237;l L, Kasai F, Trifonov VA, O&#8217;Brien PCM, et al. Strong conservation of the bird Z chromosome in reptilian genomes is revealed by comparative painting despite 275 million years divergence. Chromosoma. 2011;120:455&#8211;68.</td><td><a href=http://dx.doi.org/10.1007/s00412-011-0322-0>10.1007/s00412-011-0322-0</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Pokorn&#225; M, Giovannotti M, Kratochv&#237;l L, Caputo V, Olmo E, Ferguson-Smith MA, et al. Conservation of chromosomes syntenic with avian autosomes in squamate reptiles revealed by comparative chromosome painting. Chromosoma. 2012;121:409&#8211;18.</td><td><a href=http://dx.doi.org/10.1007/s00412-012-0371-z>10.1007/s00412-012-0371-z</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR11</td><td>BibArticle</td><td>Young MJ, Meally DO, Sarre SD, Georges A, Ezaz T. Molecular cytogenetic map of the central bearded dragon, <Emphasis Type="Italic">Pogona vitticeps</Emphasis> (Squamata : Agamidae). Chromosom Res. 2013;21:361&#8211;74.</td><td><a href=http://dx.doi.org/10.1007/s10577-013-9362-z>10.1007/s10577-013-9362-z</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR12</td><td>BibArticle</td><td>Srikulnath K, Uno Y, Nishida C, Matsuda Y. Karyotype evolution in monitor lizards: Cross-species chromosome mapping of cDNA reveals highly conserved synteny and gene order in the Toxicofera clade. Chromosom Res. 2013;21:805&#8211;19.</td><td><a href=http://dx.doi.org/10.1007/s10577-013-9398-0>10.1007/s10577-013-9398-0</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Srikulnath K, Nishida C, Matsubara K, Uno Y, Thongpan A, Suputtitada S, et al. Karyotypic evolution in squamate reptiles: comparative gene mapping revealed highly conserved linkage homology between the butterfly lizard (<Emphasis Type="Italic">Leiolepis reevesii rubritaeniata</Emphasis>, Agamidae, Lacertilia) and the Japanese four-striped rat snake (<Emphasis Type="Italic">Elaphe quadrivirgata</Emphasis>, Colubridae, Serpentes). Chromosome Res. 2009;17:975&#8211;86.</td><td><a href=http://dx.doi.org/10.1007/s10577-009-9101-7>10.1007/s10577-009-9101-7</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Srikulnath K, Matsubara K, Uno Y, Nishida C, Olsson M, Matsuda Y. Identification of the linkage group of the Z sex chromosomes of the sand lizard (<Emphasis Type="Italic">Lacerta agilis</Emphasis>, Lacertidae) and elucidation of karyotype evolution in lacertid lizards. Chromosoma. 2014;123:563&#8211;75.</td><td><a href=http://dx.doi.org/10.1007/s00412-014-0467-8>10.1007/s00412-014-0467-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Srikulnath K, Uno Y, Nishida C, Ota H, Matsuda Y. Karyotype reorganization in the Hokou gecko (<Emphasis Type="Italic">Gekko hokouensis</Emphasis>, Gekkonidae): The process of microchromosome disappearance in Gekkota. PLoS One. 2015;10:e0134829.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0134829>10.1371/journal.pone.0134829</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Vonk FJ, Casewell NR, Henkel CV, Heimberg AM, Jansen HJ, McCleary RJR, et al. The king cobra genome reveals dynamic gene evolution and adaptation in the snake venom system. Proc Natl Acad Sci U S A. 2013;110:20651&#8211;6.</td><td><a href=http://dx.doi.org/10.1073/pnas.1314702110>10.1073/pnas.1314702110</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Castoe TA, de Koning APJ, Hall KT, Card DC, Schield DR, Fujita MK, et al. The Burmese python genome reveals the molecular basis for extreme adaptation in snakes. Proc Natl Acad Sci U S A. 2013;110:20645&#8211;50.</td><td><a href=http://dx.doi.org/10.1073/pnas.1314475110>10.1073/pnas.1314475110</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Alf&#246;ldi J, Di Palma F, Grabherr M, Williams C, Kong L, Mauceli E, et al. The genome of the green anole lizard and a comparative analysis with birds and mammals. Nature. 2011;477:587&#8211;91.</td><td><a href=http://dx.doi.org/10.1038/nature10390>10.1038/nature10390</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Song B, Cheng S, Sun Y, Zhong X, Jin J, Guan R, et al. A genome draft of the legless anguid lizard, <Emphasis Type="Italic">Ophisaurus gracilis</Emphasis>. Gigascience. 2015;4:17.</td><td><a href=http://dx.doi.org/10.1186/s13742-015-0056-7>10.1186/s13742-015-0056-7</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Liu Y, Zhou Q, Wang Y, Luo L, Yang J, Yang L, et al. <Emphasis Type="Italic">Gekko japonicus</Emphasis> genome reveals evolution of adhesive toe pads and tail regeneration. Nat Commun. 2015;6:1&#8211;11.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1038/ncomms10033>10.1038/ncomms10033</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR21</td><td>BibArticle</td><td>Bickham JW. Two-hundred-million-year-old chromosomes: deceleration of the rate of karyotypic evolution in turtles. Science. 1981;212:1291&#8211;3.</td><td><a href=http://dx.doi.org/10.1126/science.212.4500.1291>10.1126/science.212.4500.1291</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR22</td><td>BibArticle</td><td>Olmo E. Trends in the evolution of reptilian chromosomes. Integr Comp Biol. 2008;48:486&#8211;93.</td><td><a href=http://dx.doi.org/10.1093/icb/icn049>10.1093/icb/icn049</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR23</td><td>BibArticle</td><td>Badenhorst D, Hillier LW, Literman R, Montiel EE, Radhakrishnan S, Shen Y, et al. Physical mapping and refinement of the painted turtle genome (<Emphasis Type="Italic">Chrysemys picta</Emphasis>) inform amniote genome evolution and challenge turtle-bird chromosomal conservation. Genome Biol Evol. 2015;7:2038&#8211;50.</td><td><a href=http://dx.doi.org/10.1093/gbe/evv119>10.1093/gbe/evv119</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR24</td><td>BibArticle</td><td>Quinn AE, Georges A, Sarre SD, Guarino F, Ezaz T, Graves JAM. Temperature sex reversal implies sex gene dosage in a reptile. Science. 2007;316:411.</td><td><a href=http://dx.doi.org/10.1126/science.1135925>10.1126/science.1135925</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR25</td><td>BibArticle</td><td>Radder RS, Quinn AE, Georges A, Sarre SD, Shine R. Genetic evidence for co-occurrence of chromosomal and thermal sex-determining systems in a lizard. Biol Lett. 2008;4:176&#8211;8.</td><td><a href=http://dx.doi.org/10.1098/rsbl.2007.0583>10.1098/rsbl.2007.0583</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR26</td><td>BibArticle</td><td>Holleley CE, O&#8217;Meally D, Sarre SD, Marshall-Graves JA, Ezaz T, Matsubara K, et al. Sex reversal triggers the rapid transition from genetic to temperature-dependent sex. Nature. 2015;523:79&#8211;82.</td><td><a href=http://dx.doi.org/10.1038/nature14574>10.1038/nature14574</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR27</td><td>BibArticle</td><td>Kawai A, Ishijima J, Nishida C, Kosaka A, Ota H, Kohno S, et al. The ZW sex chromosomes of <Emphasis Type="Italic">Gekko hokouensis</Emphasis> (Gekkonidae, Squamata) represent highly conserved homology with those of avian species. Chromosoma. 2009;118:43&#8211;51.</td><td><a href=http://dx.doi.org/10.1007/s00412-008-0176-2>10.1007/s00412-008-0176-2</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR28</td><td>BibArticle</td><td>Matsubara K, Kuraku S, Tarui H, Nishimura O, Nishida C, Agata K, et al. Intra-genomic GC heterogeneity in sauropsids: evolutionary insights from cDNA mapping and GC(3) profiling in snake. BMC Genomics. 2012;13:604.</td><td><a href=http://dx.doi.org/10.1186/1471-2164-13-604>10.1186/1471-2164-13-604</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR29</td><td>BibArticle</td><td>Matsubara K, Tarui H, Toriba M, Yamada K, Nishida-Umehara C, Agata K, et al. Evidence for different origin of sex chromosomes in snakes, birds, and mammals and step-wise differentiation of snake sex chromosomes. Proc Natl Acad Sci U S A. 2006;103:18190&#8211;5.</td><td><a href=http://dx.doi.org/10.1073/pnas.0605274103>10.1073/pnas.0605274103</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR30</td><td>BibArticle</td><td>Vicoso B, Emerson JJ, Zektser Y, Mahajan S, Bachtrog D. Comparative sex chromosome genomics in snakes: differentiation, evolutionary strata, and lack of global dosage compensation. PLoS Biol. 2013;11:e1001643.</td><td><a href=http://dx.doi.org/10.1371/journal.pbio.1001643>10.1371/journal.pbio.1001643</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR31</td><td>BibArticle</td><td>Rovatsos M, Altmanova M, Pokorna MJ, Kratochvil L. Novel X-linked genes revealed by quantitative polymerase chain reaction in the green anole, <Emphasis Type="Italic">Anolis carolinensis</Emphasis>. G3. 2014;4:2107&#8211;13.</td><td><a href=http://dx.doi.org/10.1534/g3.114.014084>10.1534/g3.114.014084</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR32</td><td>BibArticle</td><td>Ezaz T, Azad B, O&#8217;Meally D, Young MJ, Matsubara K, Edwards MJ, et al. Sequence and gene content of a large fragment of a lizard sex chromosome and evaluation of candidate sex differentiating gene R-spondin 1. BMC Genomics. 2013;14:899.</td><td><a href=http://dx.doi.org/10.1186/1471-2164-14-899>10.1186/1471-2164-14-899</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR33</td><td>BibArticle</td><td>Rovatsos M, Altmanov&#225; M, Pokorn&#225; M, Kratochv&#237;l L. Conserved sex chromosomes across adaptively radiated <Emphasis Type="Italic">Anolis</Emphasis> lizards. Evolution. 2014;68:2079&#8211;85.</td><td><a href=http://dx.doi.org/10.1111/evo.12357>10.1111/evo.12357</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR34</td><td>BibArticle</td><td>Georges A, Li Q, Lian J, Meally DO, Deakin J, Wang Z, et al. High-coverage sequencing and annotated assembly of the genome of the Australian dragon lizard <Emphasis Type="Italic">Pogona vitticeps</Emphasis>. Gigascience. 2015;4:45.</td><td><a href=http://dx.doi.org/10.1186/s13742-015-0085-2>10.1186/s13742-015-0085-2</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR35</td><td>BibArticle</td><td>Witten G. Some karyotypes of Australian agamids (Reptilia : Lacertilia). Aust J Zool. 1983;31:533&#8211;40.</td><td><a href=http://dx.doi.org/10.1071/ZO9830533>10.1071/ZO9830533</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR36</td><td>BibArticle</td><td>Ezaz T, Quinn AE, Miura I, Sarre SD, Georges A, Marshall Graves JA. The dragon lizard <Emphasis Type="Italic">Pogona vitticep</Emphasis>s has ZZ/ZW micro-sex chromosomes. Chromosom Res. 2005;13:763&#8211;76.</td><td><a href=http://dx.doi.org/10.1007/s10577-005-1010-9>10.1007/s10577-005-1010-9</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR37</td><td>BibArticle</td><td>Deakin JE, Koina E, Waters PD, Doherty R, Patel VS, Delbridge ML, et al. Physical map of two tammar wallaby chromosomes: A strategy for mapping in non-model mammals. Chromosom Res. 2008;16:1159&#8211;75.</td><td><a href=http://dx.doi.org/10.1007/s10577-008-1266-y>10.1007/s10577-008-1266-y</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR38</td><td>BibArticle</td><td>Renfree MB, Papenfuss AT, Deakin JE, Lindsay J, Heider T, Belov K, et al. Genome sequence of an Australian kangaroo, <Emphasis Type="Italic">Macropus eugenii</Emphasis>, provides insight into the evolution of mammalian reproduction and development. Genome Biol. 2011;12:R81.</td><td><a href=http://dx.doi.org/10.1186/gb-2011-12-8-r81>10.1186/gb-2011-12-8-r81</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR39</td><td>BibArticle</td><td>O&#8217;Meally D, Ezaz T, Georges A, Sarre SD, Graves JAM. Are some chromosomes particularly good at sex? Insights from amniotes. Chromosom Res. 2012;20:7&#8211;19.</td><td><a href=http://dx.doi.org/10.1007/s10577-011-9266-8>10.1007/s10577-011-9266-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR40</td><td>BibArticle</td><td>Kawai A, Nishida-Umehara C, Ishijima J, Tsuda Y, Ota H, Matsuda Y. Different origins of bird and reptile sex chromosomes inferred from comparative mapping of chicken Z-linked genes. Cytogenet Genome Res. 2007;117:92&#8211;102.</td><td><a href=http://dx.doi.org/10.1159/000103169>10.1159/000103169</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR41</td><td>BibArticle</td><td>Ezaz T, Moritz B, Waters P, Graves JAM, Georges A, Sarre SD. The ZW sex microchromosomes of an Australian dragon lizard share no homology with those of other reptiles or birds. Chromosom Res. 2009;17:965&#8211;73.</td><td><a href=http://dx.doi.org/10.1007/s10577-009-9102-6>10.1007/s10577-009-9102-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR42</td><td>BibArticle</td><td>Tsuda Y, Nishida-Umehara C, Ishijima J, Yamada K, Matsuda Y. Comparison of the Z and W sex chromosomal architectures in elegant crested tinamou (<Emphasis Type="Italic">Eudromia elegans</Emphasis>) and ostrich (<Emphasis Type="Italic">Struthio camelus</Emphasis>) and the process of sex chromosome differentiation in palaeognathous birds. Chromosoma. 2007;116:159&#8211;73.</td><td><a href=http://dx.doi.org/10.1007/s00412-006-0088-y>10.1007/s00412-006-0088-y</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR43</td><td>BibArticle</td><td>Valenzuela N, Neuwald JL, Literman R. Transcriptional evolution underlying vertebrate sexual development. Dev Dyn. 2013;242:307&#8211;19.</td><td><a href=http://dx.doi.org/10.1002/dvdy.23897>10.1002/dvdy.23897</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR44</td><td>BibArticle</td><td>Lin L, Achermann JC. Steroidogenic factor-1 (SF-1, Ad4BP, NR5A1) and disorders of testis development. Sex Dev. 2008;2:200&#8211;9.</td><td><a href=http://dx.doi.org/10.1159/000152036>10.1159/000152036</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR45</td><td>BibArticle</td><td>Luo X, Ikeda Y, Lala D, Rice D, Wong M, Parker KL. Steroidogenic factor 1 (SF-1) is essential for endocrine development and function. J Steroid Biochem Mol Biol. 1999;69:13&#8211;8.</td><td><a href=http://dx.doi.org/10.1016/S0960-0760(98)00146-0>10.1016/S0960-0760(98)00146-0</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR46</td><td>BibArticle</td><td>Vilella AJ, Severin J, Ureta-Vidal A, Heng L, Durbin R, Birney E. EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Res. 2009;19:327&#8211;35.</td><td><a href=http://dx.doi.org/10.1101/gr.073585.107>10.1101/gr.073585.107</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR47</td><td>BibArticle</td><td>Untergasser A, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, et al. Primer3-new capabilities and interfaces. Nucleic Acids Res. 2012;40:e115.</td><td><a href=http://dx.doi.org/10.1093/nar/gks596>10.1093/nar/gks596</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR48</td><td>BibArticle</td><td>Schuler GD. Sequence mapping by electronic PCR. Genome Res. 1997;7:541&#8211;50.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR49</td><td>BibArticle</td><td>Jurka J, Kapitonov VV, Pavlicek A, Klonowski P, Kohany O, Walichiewicz J. Repbase Update, a database of eukaryotic repetitive elements. Cytogenet Genome Res. 2005;110:462&#8211;7.</td><td><a href=http://dx.doi.org/10.1159/000084979>10.1159/000084979</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR50</td><td>BibArticle</td><td>Alsop AE, Miethke P, Rofe R, Koina E, Sankovic N, Deakin JE, et al. Characterizing the chromosomes of the Australian model marsupial <Emphasis Type="Italic">Macropus eugenii</Emphasis> (tammar wallaby). Chromosom Res. 2005;13:627&#8211;36.</td><td><a href=http://dx.doi.org/10.1007/s10577-005-0989-2>10.1007/s10577-005-0989-2</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR51</td><td>BibArticle</td><td>Lichter P, Tang C-JC, Call K, Hermanson G, Evans GA, Housman D, et al. High-resolution mapping of human chromosome 11 by in situ hybridization with cosmid clones. Science. 1990;247:64&#8211;9.</td><td><a href=http://dx.doi.org/10.1126/science.2294592>10.1126/science.2294592</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr></table>