<H1>Chapter DOI: 10.1007/11780441_17<br/>Cited-By Count: 1</H1><table border="1" width="30%"><tr><td>Total References</td><td>16</td></tr><tr><td>Springer references</td><td>3</td></tr><tr><td>Non Springer references</td><td>13</td></tr><tr><td>BibStructured Count</td><td width="10%">14</td></tr><tr><td>BibUnstructured Count</td><td width="10%">2</td></tr><tr><td>DOI already available in SpringerLink</td><td>10</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>5</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>1</td></tr><tr><td>DOI Obtained For Non Springer</td><td>4</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>3</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>2</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibChapter</td><td>Arvestad, L., Berglund, A.C., Lagergren, J., Sennblad, B.: Gene tree reconstruction and orthology analysis based on an integrated model for duplication and sequence evolution. In: Proc. RECOMB 2004, pp. 326&#8211;335. ACM Press, New York (2004)</td><td><a href=http://dx.doi.org/10.1145/974614.974657>10.1145/974614.974657</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Arvestad, L., Berglund, A.C., Lagergren, J., Sennblad, B.: Bayesian gene/species tree reconciliation and orthology analysis using MCMC. Bioinformatics&nbsp;19(Suppl. 1), 7&#8211;15 (2003)</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btg1000>10.1093/bioinformatics/btg1000</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibChapter</td><td>Chor, B., Tuller, T.: Maximum likelihood of evolutionary trees is hard. In: Miyano, S., Mesirov, J., Kasif, S., Istrail, S., Pevzner, P.A., Waterman, M. (eds.) RECOMB 2005. LNCS (LNBI), vol.&nbsp;3500, pp. 296&#8211;310. Springer, Heidelberg (2005)</td><td><a href=http://dx.doi.org/10.1007/11415770_23>10.1007/11415770_23</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Day, W.H.: Computational complexity of inferring phylogenies from dissimilarity matrices. Bull. Math. Biol.&nbsp;49(4), 461&#8211;467 (1987)</td><td><a href=http://dx.doi.org/10.1007/BF02458863>10.1007/BF02458863</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, IssueID, ArticleTitle, Author_FamilyName_1, FirstPage, VolumeID</td><td>CrossRef</td></tr><tr><td>CR5</td><td>BibChapter</td><td>Durand, D., Halld&#243;rsson, B.V., Vernot, B.: A Hybrid Micro-Macroevolutionary Approach to Gene Tree Reconstruction. In: Miyano, S., Mesirov, J., Kasif, S., Istrail, S., Pevzner, P.A., Waterman, M. (eds.) RECOMB 2005. LNCS (LNBI), vol.&nbsp;3500, pp. 250&#8211;264. Springer, Heidelberg (2005)</td><td><a href=http://dx.doi.org/10.1007/11415770_19>10.1007/11415770_19</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Eulenstein, O., Mikrin, B., Vingron, M.: Duplication-based measures of difference between gene and species trees. Journal of Computational Biology&nbsp;5, 135&#8211;148 (1998)</td><td><a href=http://dx.doi.org/10.1089/cmb.1998.5.135>10.1089/cmb.1998.5.135</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Felsenstein, J.: Phylogenies from molecular sequences: Inference and reliability. Annu. Rev. Genet.&nbsp;22, 521&#8211;565 (1988)</td><td><a href=http://dx.doi.org/10.1146/annurev.ge.22.120188.002513>10.1146/annurev.ge.22.120188.002513</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Fitch, W., Margoliash, E.: Construction of phylogenetic trees. Science&nbsp;155, 279&#8211;284 (1967)</td><td><a href=http://dx.doi.org/10.1126/science.155.3760.279>10.1126/science.155.3760.279</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Goodman, M., Czelusniak, J., Moore, G.W., Romero-Herrera, A.E., Matsuda, G.: Fitting the gene lineage into its species lineage, a parsimony strategy illustrated by cladograms constructed from globin sequences. Syst. Zool.&nbsp;28, 138&#8211;163 (1979)</td><td><a href=http://dx.doi.org/10.1093/sysbio/28.2.132>10.1093/sysbio/28.2.132</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, ArticleTitle, VolumeID, Year, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Guigo, R., Muchnik, I., Smith, T.F.: Reconstruction of ancient phylogenies. Molecular Phylogenetics and Evolution&nbsp;6, 189&#8211;213 (1996)</td><td><a href=http://dx.doi.org/10.1006/mpev.1996.0071>10.1006/mpev.1996.0071</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR11</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hallett</span>, <span style='background:#DDDDDD'>M.T.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Lagergren</span>, <span style='background:#DDDDDD'>J.</span></span></aug>: <span style='background:#FFD9B3'>New algorithms for the duplication-loss model</span>. In: Proc. RECOMB <span style='background:#66FF66'>2000</span> <span style='background:#FFFF0F'>(2000)</span></td><td><a href=http://dx.doi.org/10.1145/332306.332359>10.1145/332306.332359</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, BookTitle, Author_FamilyName_2, Year</td><td>CrossRef</td></tr><tr><td>CR12</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Ma</span>, <span style='background:#DDDDDD'>B.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Li</span>, <span style='background:#DDDDDD'>M.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Zhang</span>, <span style='background:#DDDDDD'>L.</span></span></aug>: <span style='background:#FFD9B3'>From gene trees to species trees</span>. SIAM J. on comput. (<span style='background:#66FF66'>2000</span>)</td><td><a href=http://dx.doi.org/10.1137/S0097539798343362>10.1137/S0097539798343362</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, Author_FamilyName_2, BookTitle, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Mirkin, B., Muchnik, I., Smith, T.F.: A biologically consistent model for comparing molecular phylogenies. Journal of Computational Biology&nbsp;2, 493&#8211;507 (1995)</td><td><a href=http://dx.doi.org/10.1089/cmb.1995.2.493>10.1089/cmb.1995.2.493</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Page, R.D.M.: Maps between trees and cladistic analysis of historical associations among genes, organisms and areas. Syst. Zool.&nbsp;43, 58&#8211;77 (1994)</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.2307/2413581>10.2307/2413581</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, VolumeID, Year</td><td>CrossRef</td></tr><tr><td>CR15</td><td>BibBook</td><td>Nei, M.: Molecular Evolution Genetics. Columbia University Press, New York (1987)</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Zmasek, C.M., Eddy, S.R.: A simple algorithm to infer gene duplication and speciation events on a gene tree. Bioinformatics&nbsp;17(9), 821&#8211;828 (2001)</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/17.9.821>10.1093/bioinformatics/17.9.821</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr></table>