<H1>Chapter DOI: 10.1007/11780441_22<br/>Cited-By Count: 1</H1><table border="1" width="30%"><tr><td>Total References</td><td>23</td></tr><tr><td>Springer references</td><td>2</td></tr><tr><td>Non Springer references</td><td>21</td></tr><tr><td>BibStructured Count</td><td width="10%">20</td></tr><tr><td>BibUnstructured Count</td><td width="10%">3</td></tr><tr><td>DOI already available in SpringerLink</td><td>17</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>3</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>1</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>2</td></tr><tr><td>DOI Obtained For Non Springer</td><td>2</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>3</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>1</td></tr><tr><td>Valid DOI Enriched</td><td>3</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Akutsu</span>, <span style='background:#DDDDDD'>T.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Arimura</span>, <span style='background:#DDDDDD'>H.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Shimozono</span>, <span style='background:#DDDDDD'>S.</span></span></aug>: <span style='background:#FFD9B3'>On approximation algorithms for local multiple alignment</span>. In: RECOMB, pp.&amp;nbsp;<span style='background:#D279FF'>1&#8211;7</span> (<span style='background:#66FF66'>2000</span>)</td><td><a href=http://dx.doi.org/10.1145/332306.332311>10.1145/332306.332311</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, BookTitle, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Bafna, V., Lawler, E., Pevzner, P.A.: Approximation algorithms for multiple alignment. Theoretical Computer Science&nbsp;182, 233&#8211;244 (1997)</td><td><a href=http://dx.doi.org/10.1016/S0304-3975(97)00023-6>10.1016/S0304-3975(97)00023-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibArticle</td><td>Bailey, T., Elkan, C.: Unsupervised learning of multiple motifs in biopolymers using expectation maximization. Machine Learning&nbsp;21, 51&#8211;80 (1995)</td><td><a href=http://dx.doi.org/10.1007/BF00993379>10.1007/BF00993379</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Year, VolumeID, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Chazelle, B., Kingsford, C., Singh, M.: A semidefinite programming approach to side-chain positioning with new rounding strategies. INFORMS J. on Computing&nbsp;16, 380&#8211;392 (2004)</td><td><a href=http://dx.doi.org/10.1287/ijoc.1040.0096>10.1287/ijoc.1040.0096</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR5</td><td>BibBook</td><td>Cook, W., Cunningham, W., Pulleyblank, W., Schrijver, A.: Combinatorial Optimization. Wiley-Interscience, New York (1997)</td><td><a href=http://dx.doi.org/10.1002/9781118033142>10.1002/9781118033142</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Author_FamilyName_1, BookTitle, Year</td><td>CrossRef</td></tr><tr><td>CR6</td><td>BibBook</td><td>Gr&#246;tschel, M., Lov&#225;sz, L., Schrijver, A.: Geometric Algorithms and Combinatorial Optimization, 2nd edn. Springer, Berlin (1993)</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1007/978-3-642-78240-4>10.1007/978-3-642-78240-4</a></td><td><b>CrossRef-Web</b></td>
<td>Year, BookTitle</td><td>CrossRef</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Hertz, G., Stormo, G.: Identifying DNA and protein patterns with statistically significant alignments of multiple sequences. Bioinf.&nbsp;15, 563&#8211;577 (1999)</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/15.7.563>10.1093/bioinformatics/15.7.563</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Kellis, M., Patterson, N., Endrizzi, M., Birren, B., Lander, E.: Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature&nbsp;423, 241&#8211;254 (2003)</td><td><a href=http://dx.doi.org/10.1038/nature01644>10.1038/nature01644</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Kingsford, C., Chazelle, B., Singh, M.: Solving and analyzing side-chain positioning problems using linear and integer programming. Bioinf.&nbsp;21, 1028&#8211;1039 (2005)</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/bti144>10.1093/bioinformatics/bti144</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Lawrence, C., Altschul, S., Boguski, M., Liu, J., Neuwald, A., Wootton, J.: Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment. Science&nbsp;262, 208&#8211;214 (1993)</td><td><a href=http://dx.doi.org/10.1126/science.8211139>10.1126/science.8211139</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR11</td><td>BibArticle</td><td>Lee, T., Rinaldi, N., Robert, F., Odom, D., Bar-Joseph, Z., Gerber, G., et al.: Transcriptional regulatory networks in <Emphasis Type="Italic"> S. cerevisiae</Emphasis>. Science&nbsp;298, 799&#8211;804 (2002)</td><td><a href=http://dx.doi.org/10.1126/science.1075090>10.1126/science.1075090</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR12</td><td>BibArticle</td><td>Li, M., Ma, B., Wang, L.: Finding similar regions in many strings. J. Computer and Systems Sciences&nbsp;65(1), 73&#8211;96 (2002)</td><td><a href=http://dx.doi.org/10.1006/jcss.2002.1823>10.1006/jcss.2002.1823</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Marsan, L., Sagot, M.F.: Algorithms for extracting structured motifs using a suffix tree with an application to promoter and regulatory site consensus identification. J. Comp. Bio.&nbsp;7, 345&#8211;362 (2000)</td><td><a href=http://dx.doi.org/10.1089/106652700750050826>10.1089/106652700750050826</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibArticle</td><td>McGuire, A., Hughes, J., Church, G.: Conservation of DNA regulatory motifs and discovery of new motifs in microbial genomes. Genome Res.&nbsp;10, 744&#8211;757 (2000)</td><td><a href=http://dx.doi.org/10.1101/gr.10.6.744>10.1101/gr.10.6.744</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Osada, R., Zaslavsky, E., Singh, M.: Comparative analysis of methods for representing and searching for transcription factor binding sites. Bioinf.&nbsp;20, 3516&#8211;3525 (2004)</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/bth438>10.1093/bioinformatics/bth438</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR16</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Pevzner</span>, <span style='background:#DDDDDD'>P.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Sze</span>, <span style='background:#DDDDDD'>S.</span></span></aug>: <span style='background:#FFD9B3'>Combinatorial approaches to finding subtle signals in DNA sequences</span>. In: ISMB, pp.&amp;nbsp;<span style='background:#D279FF'>269&#8211;278</span> (<span style='background:#66FF66'>2000</span>)</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Robison, K., McGuire, A., Church, G.: A comprehensive library of DNA-binding site matrices for 55 proteins applied to the complete <Emphasis Type="Italic">Escherichia coli</Emphasis> K-12 Genome. J. Mol. Biol.&nbsp;284, 241&#8211;254 (1998)</td><td><a href=http://dx.doi.org/10.1006/jmbi.1998.2160>10.1006/jmbi.1998.2160</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Schuler, G., Altschul, S., Lipman, D.: A workbench for multiple alignment construction and analysis. Proteins&nbsp;9(3), 180&#8211;190 (1991)</td><td><a href=http://dx.doi.org/10.1002/prot.340090304>10.1002/prot.340090304</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Tavazoie, S., Hughes, J., Campbell, M., Cho, R., Church, G.: Systematic determination of genetic network architecture. Nat. Genetics&nbsp;22(3), 281&#8211;285 (1999)</td><td><a href=http://dx.doi.org/10.1038/10343>10.1038/10343</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Thompson, W., Rouchka, E., Lawrence, C.: Gibbs Recursive Sampler: finding transcription factor binding sites. Nucleic Acids Res.&nbsp;31, 3580&#8211;3585 (2003)</td><td><a href=http://dx.doi.org/10.1093/nar/gkg608>10.1093/nar/gkg608</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR21</td><td>BibArticle</td><td>Tompa, M., Li, N., Bailey, T., Church, G., De Moor, B., Eskin, E., et al.: Assessing computational tools for the discovery of transcription factor binding sites. Nat. Biotech.&nbsp;23, 137&#8211;144 (2005)</td><td><a href=http://dx.doi.org/10.1038/nbt1053>10.1038/nbt1053</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR22</td><td>BibArticle</td><td>Wang, L., Jiang, T.: On the complexity of multiple sequence alignment. J. Comp. Bio.&nbsp;1, 337&#8211;348 (1994)</td><td><a href=http://dx.doi.org/10.1089/cmb.1994.1.337>10.1089/cmb.1994.1.337</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR23</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Zaslavsky</span>, <span style='background:#DDDDDD'>E.</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Singh</span>, <span style='background:#DDDDDD'>M.</span></span></aug>: Combinatorial Optimization Approaches to Motif Finding (<span style='background:#66FF66'>submitted</span>), also available as Princeton University Computer Science Dept. Technical Report TR-728-05</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></table>