<H1>Chapter DOI: 10.1208/aapsj070243<br/>Cited-By Count: 43</H1><table border="1" width="30%"><tr><td>Total References</td><td>159</td></tr><tr><td>Springer references</td><td>7</td></tr><tr><td>Non Springer references</td><td>152</td></tr><tr><td>BibStructured Count</td><td width="10%">157</td></tr><tr><td>BibUnstructured Count</td><td width="10%">2</td></tr><tr><td>DOI already available in SpringerLink</td><td>130</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>7</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>1</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>0</td></tr><tr><td>DOI Obtained For Non Springer</td><td>8</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>7</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>1</td></tr><tr><td>Valid DOI Enriched</td><td>3</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibArticle</td><td>Kieffer BL. Recent advances in molecular recognition and signal transduction of active peptides: receptors for opioid peptides.<Emphasis Type="Italic">Cell Mol Neurobiol</Emphasis>. 1995;15:615&#8211;635.</td><td><a href=http://dx.doi.org/10.1007/BF02071128>10.1007/BF02071128</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Waldhoer M, Bartlett SE, Whistler JL. Opioid receptors.<Emphasis Type="Italic">Annu Rev Biochem</Emphasis>. 2004;73:953&#8211;990.</td><td><a href=http://dx.doi.org/10.1146/annurev.biochem.73.011303.073940>10.1146/annurev.biochem.73.011303.073940</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibArticle</td><td>Pasternak GW. Multiple opiate receptors: d&#233;j&#224; vu all over again.<Emphasis Type="Italic">Neuropharmacology</Emphasis>. 2004;47:312&#8211;323.</td><td><a href=http://dx.doi.org/10.1016/j.neuropharm.2004.07.004>10.1016/j.neuropharm.2004.07.004</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR4</td><td>BibArticle</td><td>Vaccarino AL, Kastin AJ. Endogenous opiates: 2000.<Emphasis Type="Italic">Peptides</Emphasis>. 2001;22:2257&#8211;2328.</td><td><a href=http://dx.doi.org/10.1016/S0196-9781(01)00566-6>10.1016/S0196-9781(01)00566-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR5</td><td>BibArticle</td><td>Hruby VJ, Agnes RS. Conformation-activity relationships of opioid peptides with selective activities at opioid receptors.<Emphasis Type="Italic">Biopolymers</Emphasis>. 1999;51:391&#8211;410.</td><td><a href=http://dx.doi.org/10.1002/(SICI)1097-0282(1999)51:6&lt;391::AID-BIP3&gt;3.0.CO;2-X>10.1002/(SICI)1097-0282(1999)51:6&lt;391::AID-BIP3&gt;3.0.CO;2-X</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Mansour A, Hoversten MT, Taylor LP, Watson SJ, Akil H. The cloned mu, delta and kappa receptors and their endogenous ligands: evidence for two opioid peptide recognition cores.<Emphasis Type="Italic">Brain Res</Emphasis>. 1995;700:89&#8211;98.</td><td><a href=http://dx.doi.org/10.1016/0006-8993(95)00928-J>10.1016/0006-8993(95)00928-J</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR7</td><td>BibArticle</td><td>Mosberg HI, Hurst R, Hruby VJ, et al. Conformationally constrained cyclic enkephalin analogs with pronounced delta opioid receptor agonist selectivity.<Emphasis Type="Italic">Life Sci</Emphasis>. 1983;32:2565&#8211;2569.</td><td><a href=http://dx.doi.org/10.1016/0024-3205(83)90239-4>10.1016/0024-3205(83)90239-4</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Pelton JT, Gulya K, Hruby VJ, Duckles S, Yamamura HI. Somatostatin analogs with affinity for opiate receptors in rat brain binding assay.<Emphasis Type="Italic">Peptides</Emphasis>. 1985;6:159&#8211;163.</td><td><a href=http://dx.doi.org/10.1016/0196-9781(85)90026-9>10.1016/0196-9781(85)90026-9</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Pelton JT, Kazmierski W, Gulya K, Yamamura HI, Hruby VJ. Design and synthesis of conformationally constrained somatostatin analogues with high potency and specificity for mu opioid receptors.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1986;29:2370&#8211;2375.</td><td><a href=http://dx.doi.org/10.1021/jm00161a037>10.1021/jm00161a037</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Kawasaki AM, Knapp RJ, Walton A et al. Syntheses, opioid binding affinities, and potencies of dynorphin A analogues substituted in positions 1, 6, 7, 8 and 10.<Emphasis Type="Italic">Int J Pept Protein Res</Emphasis>. 1993;42:411&#8211;419.</td><td><a href=http://dx.doi.org/10.1111/j.1399-3011.1993.tb00148.x>10.1111/j.1399-3011.1993.tb00148.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR11</td><td>BibArticle</td><td>Meyer JP, Collins N, Lung FD, et al. Design, synthesis, and biological properties of highly potent cyclic dynorphin A analogues: analogues cyclized between positions 5 and 11<Emphasis Type="Italic">J Med Chem</Emphasis>. 1994;37:3910&#8211;3917.</td><td><a href=http://dx.doi.org/10.1021/jm00049a010>10.1021/jm00049a010</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR12</td><td>BibArticle</td><td>Arttamangkul S, Murray TF, DeLander GE, Aldrich JV. Synthesis and opioid activity of conformationally constrained dynorphin A analogues. 1. Conformational constraint in the &#8220;message&#8221; sequence.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1995;38:2410&#8211;2417.</td><td><a href=http://dx.doi.org/10.1021/jm00013a016>10.1021/jm00013a016</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Lung FD, Collins N, Stropova D, et al. Design, synthesis, and biological activities of cyclic lactam peptide analogues of dynorphin A(1-11)-NH2.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1996;39:1136&#8211;1141.</td><td><a href=http://dx.doi.org/10.1021/jm950369c>10.1021/jm950369c</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Eguchi M. Recent advances in selective opioid receptor agonists and antagonists.<Emphasis Type="Italic">Med Res Rev</Emphasis>. 2004;24:182&#8211;212.</td><td><a href=http://dx.doi.org/10.1002/med.10059>10.1002/med.10059</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Okada T, Ernst OP, Palczewski K, Hofmann KP. Activation of rhodopsin: new insights from structural and biochemical studies.<Emphasis Type="Italic">Trends Biochem Sci</Emphasis>. 2001;26:318&#8211;324.</td><td><a href=http://dx.doi.org/10.1016/S0968-0004(01)01799-6>10.1016/S0968-0004(01)01799-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Li J, Edwards PC, Burghammer M, Villa C, Schertler GF. Structure of bovine rhodopsin in a trigonal crystal form.<Emphasis Type="Italic">J Mol Biol</Emphasis>. 2004;343:1409&#8211;1438.</td><td><a href=http://dx.doi.org/10.1016/j.jmb.2004.08.090>10.1016/j.jmb.2004.08.090</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Flower DR. Modeling G-protein-coupled receptors for drug design.<Emphasis Type="Italic">Biochim Biophys Acta</Emphasis> 1999;1422:207&#8211;234.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Flippen-Anderson JL, George C, Bertha CM, Rice KC. X-ray structure of potent opioid receptor ligands: etonitazene, cis-(+)-3-methylfentanyl, etorphine, diprenorphine, and buprenorphine.<Emphasis Type="Italic">Heterocycles</Emphasis>. 1994;39:751&#8211;766.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Urbanczyk-Lipkowska Z, Etter MC, Lipkowski AW, Portoghese PS. The crystal structure of a bimorphinan with highly selective kappa opioid receptor antagonist activity.<Emphasis Type="Italic">J Mol Struct</Emphasis>. 1987;159:287&#8211;295.</td><td><a href=http://dx.doi.org/10.1016/0022-2860(87)80047-9>10.1016/0022-2860(87)80047-9</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Griffin JF, Larson DL, Portoghese PS. Crystal structures of alpha-and beta-funaltrexamine: conformational requirement of the fumaramate moiety in the irreversible blockage of mu opioid receptors<Emphasis Type="Italic">J Med Chem</Emphasis>. 1986;29:778&#8211;783.</td><td><a href=http://dx.doi.org/10.1021/jm00155a031>10.1021/jm00155a031</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR21</td><td>BibArticle</td><td>Calderon SN, Rice KC, Rothman RB, et al. Probes for narcotic receptor mediated phenomena. 23. Synthesis, opioid receptor binding, and bioassay of the highly selective delta agonist (+)-4-[(alpha R)-alpha-((2S,5R)-4-Allyl-2,5-dimethyl-1-piperazinyl)-3-methoxybenzyl]-N,N-diethylbenzamide (SNC 80) and related novel nonpeptide delta opioid receptor ligands.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1997;40:695&#8211;704.</td><td><a href=http://dx.doi.org/10.1021/jm960319n>10.1021/jm960319n</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR22</td><td>BibArticle</td><td>Lavecchia A, Greco G, Novellino E, Vittorio F, Ronsisvalle G. Modeling of kappa-opioid receptor/agonists interactions using pharmacophore-based and docking simulations.<Emphasis Type="Italic">J Med Chem</Emphasis>. 2000;43:2124&#8211;2134.</td><td><a href=http://dx.doi.org/10.1021/jm991161k>10.1021/jm991161k</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR23</td><td>BibArticle</td><td>Doi M, Ishida T, Inoue M. Conformational characteristics of opioid kappa-receptor agonist: crystal structure of (5S, 7S, 8S)-(-)-N-methyl-N-[7-(1-pyrrolidinyl)-1-oxaspiro[4.5] dec-8-yl]benzeneacetamide (U69,593), and conformational comparison with some kappa-agonists.<Emphasis Type="Italic">Chem Pharm Bull (Tokyo)</Emphasis>. 1990;38:1815&#8211;1818.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1248/cpb.38.1815>10.1248/cpb.38.1815</a></td><td><b>CrossRef-Meta</b></td>
<td>VolumeID, Author_FamilyName_2, FirstPage, Year, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR24</td><td>BibArticle</td><td>Subramanian G, Patrelini MG, Portoghese PS, Ferguson DM. Molecular docking reveals a novel binding site model for fentanyl at the mu-opioid receptor.<Emphasis Type="Italic">J Med Chem</Emphasis>. 2000;43:381&#8211;391.</td><td><a href=http://dx.doi.org/10.1021/jm9903702>10.1021/jm9903702</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR25</td><td>BibArticle</td><td>Mosberg HI, Fowler CB. Development and validation of opioid ligand-receptor interaction models: the structural basis of mu vs delta selectivity.<Emphasis Type="Italic">J Pept Res</Emphasis>. 2002;60:329&#8211;335.</td><td><a href=http://dx.doi.org/10.1034/j.1399-3011.2002.21061.x>10.1034/j.1399-3011.2002.21061.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR26</td><td>BibArticle</td><td>Przydzial MJ, Pogozheva ID, Andrews SM, et al. Roles of residues 3 and 4 in cyclic tetrapeptide ligand recognition by the &#954;-opioid receptor.<Emphasis Type="Italic">J Pept Res</Emphasis>. 2005;65:333&#8211;342.</td><td><a href=http://dx.doi.org/10.1111/j.1399-3011.2005.00220.x>10.1111/j.1399-3011.2005.00220.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR27</td><td>BibArticle</td><td>Mosberg HI, Omnaas JR, Medzihradsky F, Smith CB. Cyclic, disulfide- and dithioether-containing opioid tetrapeptides: development of a ligand with high delta opioid receptor selectivity and affinity.<Emphasis Type="Italic">Life Sci</Emphasis>. 1988;43:1013&#8211;1020.</td><td><a href=http://dx.doi.org/10.1016/0024-3205(88)90547-4>10.1016/0024-3205(88)90547-4</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR28</td><td>BibArticle</td><td>Mosberg HI, Lomize AL, Wang C, et al. Development of a model for the &#948;-opioid receptor pharmacophore. 1. Conformationally restricted Tyr<Superscript>1</Superscript> replacements in the cyclic &#948; receptor selective tetrapeptide Tyr-c[D-Cys-Phe-D-Pen]OH (JOM-13).<Emphasis Type="Italic">J Med Chem</Emphasis> 1994<Emphasis Type="Italic">a</Emphasis>;37:4371&#8211;4383.</td><td><a href=http://dx.doi.org/10.1021/jm00051a015>10.1021/jm00051a015</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR29</td><td>BibArticle</td><td>Mosberg HI, Omnaas JR, Lomize A, et al. Development of a model for the &#948;-opioid receptor pharmacophore. 2. Conformationally restricted Phe<Superscript>3</Superscript> replacements in the cyclic &#948;-receptor selective tetrapeptide Tyr-c[D-Cys-Phe-D-Pen]OH (JOM-13).<Emphasis Type="Italic">J Med Chem</Emphasis>. 1994<Emphasis Type="Italic">b</Emphasis>;37:4384&#8211;4391.</td><td><a href=http://dx.doi.org/10.1021/jm00051a016>10.1021/jm00051a016</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR30</td><td>BibArticle</td><td>Mosberg HI, Dua RK, Pogozheva ID, Lomize AL. Development of a model for the &#948;-opioid receptor pharmacophore. 4. Residue 3 dehydrophenyl-alanine analogs of Tyr-c[D-Cys-Phe-D-Pen]OH (JOM-13) confirm required gauche orientation of aromatic sidechain.<Emphasis Type="Italic">Biopolymers</Emphasis>. 1996;39:287&#8211;296.</td><td><a href=http://dx.doi.org/10.1002/(SICI)1097-0282(199609)39:3&lt;287::AID-BIP2&gt;3.0.CO;2-K>10.1002/(SICI)1097-0282(199609)39:3&lt;287::AID-BIP2&gt;3.0.CO;2-K</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR31</td><td>BibArticle</td><td>Lomize AL, Flippen-Anderson JL, George C, Mosberg HI. Conformational analysis of the &#948; receptor-selective, cyclic opioid peptide, Tyr-c[D-Cys-Phe-D-Pen]OH(JOM-13): comparison of X-ray crystallographic structures, molecular mechanics simulations and<Superscript>1</Superscript>H NMR data.<Emphasis Type="Italic">J Am Chem Soc</Emphasis>. 1994;116:429&#8211;436.</td><td><a href=http://dx.doi.org/10.1021/ja00081a001>10.1021/ja00081a001</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR32</td><td>BibBook</td><td>Ho JC.<Emphasis Type="Italic">Development of a Model for the &#948;-opioid Receptor Pharmacophore</Emphasis> [dissertation]. Ann Arbor, MI: University of Michigan; 1997.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR33</td><td>BibArticle</td><td>McFadyen IJ, Ho JC, Mosberg HI, Traynor JR. Modifications of the cyclic mu receptor selective tetrapeptide Tyr-c[D-Cys-Phe-D-Pen]NH2 (Et): effects on opioid receptor binding and activation.<Emphasis Type="Italic">J Pept Res</Emphasis>. 2000;55:255&#8211;261.</td><td><a href=http://dx.doi.org/10.1034/j.1399-3011.2000.00177.x>10.1034/j.1399-3011.2000.00177.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR34</td><td>BibArticle</td><td>Fowler CB III, Pogozheva ID III, Lomize AL III, LeVine H III, Mosberg HI. Complex of an active &#956;-opioid receptor with cyclic peptide agonist modeled from experimental constraints.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2004<Emphasis Type="Italic">a</Emphasis>;43:15796&#8211;15810.</td><td><a href=http://dx.doi.org/10.1021/bi048413q>10.1021/bi048413q</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR35</td><td>BibArticle</td><td>Mosberg HI. Complementarity of delta opioid ligand pharmacophore and receptor models.<Emphasis Type="Italic">Biopolymers</Emphasis>. 1999;51:426&#8211;439.</td><td><a href=http://dx.doi.org/10.1002/(SICI)1097-0282(1999)51:6&lt;426::AID-BIP5&gt;3.0.CO;2-G>10.1002/(SICI)1097-0282(1999)51:6&lt;426::AID-BIP5&gt;3.0.CO;2-G</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR36</td><td>BibArticle</td><td>Law PY, Loh HH. Regulation of opioid receptor activities.<Emphasis Type="Italic">J Pharmacol Exp Ther</Emphasis>. 1999;289:607&#8211;624.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR37</td><td>BibArticle</td><td>Chaturvedi K, Christoffers KH, Singh K, Howells RD. Structure and regulation of opioid receptors.<Emphasis Type="Italic">Biopolymers</Emphasis>. 2000;55:334&#8211;346.</td><td><a href=http://dx.doi.org/10.1002/1097-0282(2000)55:4&lt;334::AID-BIP1006&gt;3.0.CO;2-S>10.1002/1097-0282(2000)55:4&lt;334::AID-BIP1006&gt;3.0.CO;2-S</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR38</td><td>BibArticle</td><td>Chavkin C, McLaughlin JP, Celver JP. Regulation of opioid receptor function by chronic agonist exposure: constitutive activity and desensitization.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 2001;60:20&#8211;25.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR39</td><td>BibArticle</td><td>Coward P, Wada HG, Falk MS, et al. Controlling signaling with a specifically designed Gi-coupled receptor.<Emphasis Type="Italic">Proc Natl Acad Sci USA</Emphasis>. 1998;95:352&#8211;357.</td><td><a href=http://dx.doi.org/10.1073/pnas.95.1.352>10.1073/pnas.95.1.352</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR40</td><td>BibArticle</td><td>Pogozheva ID, Lomize AL, Mosberg HI. Opioid receptor 3-dimensional structures from distance geometry calculations with hydrogen bonding constraints.<Emphasis Type="Italic">Biophys J</Emphasis> 1998;75:612&#8211;634.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/S0006-3495(98)77552-6>10.1016/S0006-3495(98)77552-6</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, Year, FirstPage, Author_FamilyName_1, VolumeID</td><td>CrossRef</td></tr><tr><td>CR41</td><td>BibArticle</td><td>Surratt CK, Johnson PS, Moriwaki A, et al. Mu opiate receptor: charged transmembrane domain amino acids are critical for agonist recognition and intrinsic activity.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1994;269:20548&#8211;20553.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR42</td><td>BibArticle</td><td>Befort K, Tabbara L, Bausch S, Chavkin C, Evans C, Kieffer BL. The conserved aspartate residue in the third putative transmembrane domain of the delta-opioid receptor is not the anionic counterpart for cationic opiate binding but is a constituent of the receptor binding site.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 1996<Emphasis Type="Italic">a</Emphasis>;49:216&#8211;223.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR43</td><td>BibArticle</td><td>Befort K, Tabbara L, Kling D, Maigret B, Kieffer BL. Role of aromatic transmembrane residues of the delta-opioid receptor in ligand recognition.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1996<Emphasis Type="Italic">b</Emphasis>;271:10161&#8211;10168.</td><td><a href=http://dx.doi.org/10.1074/jbc.271.17.10161>10.1074/jbc.271.17.10161</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR44</td><td>BibArticle</td><td>Befort K, Zilliox C, Filliol D, Yue S, Kieffer BL. Constitutive activation of the delta opioid receptor by mutations in transmembrane domains III and VII.<Emphasis Type="Italic">J Biol Chem</Emphasis> 1999;274:18574&#8211;18581.</td><td><a href=http://dx.doi.org/10.1074/jbc.274.26.18574>10.1074/jbc.274.26.18574</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR45</td><td>BibArticle</td><td>Bot G, Blake AD, Li S, Reisine T. Mutagenesis of the mouse delta opioid receptor converts(-)-buprenorphine from a partial agonist to an antagonist.<Emphasis Type="Italic">J Pharmacol Exp Ther</Emphasis>. 1998<Emphasis Type="Italic">a</Emphasis>;284:283&#8211;290.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR46</td><td>BibArticle</td><td>Bot G, Blake AD, Li S, Reisine T. Mutagenesis of a single amino acid in the rat mu-opioid receptor discriminates ligand binding.<Emphasis Type="Italic">J Neurochem</Emphasis>. 1998<Emphasis Type="Italic">b</Emphasis>;70:358&#8211;365.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR47</td><td>BibArticle</td><td>Spivak CE, Beglan CL, Seidleck BK, et al. Naloxone activation of mu-opioid receptors mutated at a histidine residue lining the opioid binding cavity.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 1997;52:983&#8211;992.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR48</td><td>BibArticle</td><td>Mansour A, Taylor LP, Fine JL, et al. Key residues defining the mu-opioid receptor binding pocket: a site-directed mutagenesis study.<Emphasis Type="Italic">J Neurochem</Emphasis> 1997;68:344&#8211;353.</td><td><a href=http://dx.doi.org/10.1046/j.1471-4159.1997.68010344.x>10.1046/j.1471-4159.1997.68010344.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR49</td><td>BibArticle</td><td>Meng F, Ueda Y, Hoversten MT, et al. Creating a functional opioid alkaloid binding site in the orphanin FQ receptor through site-directed mutagenesis.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 1998;53:772&#8211;777.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR50</td><td>BibArticle</td><td>Li JG, Chen C, Yin J, et al. ASP147 in the third transmembrane helix of the rat mu opioid receptor forms ion-pairing with morphine and naltrexone.<Emphasis Type="Italic">Life Sci</Emphasis>. 1999;65:175&#8211;185.</td><td><a href=http://dx.doi.org/10.1016/S0024-3205(99)00234-9>10.1016/S0024-3205(99)00234-9</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR51</td><td>BibArticle</td><td>Fukuda K, Terasako K, Kato S, Mori K. Identification of the amino acid residues involved in selective agonist binding in the first extracellular loop of the delta-and mu-opioid receptors.<Emphasis Type="Italic">FEBS Lett</Emphasis>. 1995;373:177&#8211;181.</td><td><a href=http://dx.doi.org/10.1016/0014-5793(95)01034-C>10.1016/0014-5793(95)01034-C</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR52</td><td>BibArticle</td><td>Minami M, Onogi T, Nakagawa T, et al. DAMGO, a mu-opioid receptor selective ligand, distinguishes between mu-and kappa-opioid receptors at a different region from that for the distinction between mu-and delta-opioid receptors.<Emphasis Type="Italic">FEBS Lett</Emphasis>. 1995;364:23&#8211;27.</td><td><a href=http://dx.doi.org/10.1016/0014-5793(95)00340-F>10.1016/0014-5793(95)00340-F</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR53</td><td>BibArticle</td><td>Wang JB, Johnson PS, Wu JM, Wang WF, Uhl GR. Human kappa opiate receptor second extracellular loop elevates dynorphin's affinity for human mu/kappa chimeras.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1994;269:2966&#8211;25969.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR54</td><td>BibArticle</td><td>Xue JC, Chen C, Zhu J, et al. Differential binding domains of peptide and non-peptide ligands in the cloned rat kappa opioid receptor.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1994;269:30195&#8211;30199.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR55</td><td>BibArticle</td><td>Ferguson DM, Kramer S, Metzger TG, Law PY, Portoghese PS. Isosteric replacement of acidic with neutral residues in extracellular loop-2 of the kappa-opioid receptor does not affect dynorphin A(1&#8211;13) affinity and function.<Emphasis Type="Italic">J Med Chem</Emphasis>. 2000;43:1251&#8211;1252.</td><td><a href=http://dx.doi.org/10.1021/jm990583r>10.1021/jm990583r</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR56</td><td>BibArticle</td><td>Bonner G, Meng F, Akil H. Selectivity of mu-opioid receptor determined by interfacial residues near third extracellular loop.<Emphasis Type="Italic">Eur J Pharmacol</Emphasis>. 2000;403:37&#8211;44.</td><td><a href=http://dx.doi.org/10.1016/S0014-2999(00)00578-1>10.1016/S0014-2999(00)00578-1</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR57</td><td>BibArticle</td><td>Xu H, Lu YF, Partilla JS, et al. Opioid peptide receptor studies. 11. Involvement of Tyr148, Trp318 and His319 of the rat mu-opioid receptor in binding of mu-selective ligands.<Emphasis Type="Italic">Synapse</Emphasis>. 1999;32:23&#8211;28.</td><td><a href=http://dx.doi.org/10.1002/(SICI)1098-2396(199904)32:1&lt;23::AID-SYN3&gt;3.0.CO;2-N>10.1002/(SICI)1098-2396(199904)32:1&lt;23::AID-SYN3&gt;3.0.CO;2-N</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR58</td><td>BibArticle</td><td>Ulens C, Van Boven M, Daenens P, Tytgat J. Interaction of p-fluoro fentanyl on cloned human opioid receptors and exploration of the role of Trp-318 and His-319 in mu-opioid receptor selectivity.<Emphasis Type="Italic">J Pharmacol Exp Ther</Emphasis>. 2000;294:1024&#8211;1033.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR59</td><td>BibArticle</td><td>Pepin MC, Yue SY, Roberts E, Wahlestedt C, Walker P. Novel &#8220;restoration of function&#8221; mutagenesis strategy to identify amino acids of the delta-opioid receptor involved in ligand binding.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1997;272:9260&#8211;9267.</td><td><a href=http://dx.doi.org/10.1074/jbc.272.14.9260>10.1074/jbc.272.14.9260</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR60</td><td>BibArticle</td><td>Valiquette M, Vu HK, Yue SY, Wahlestedt C, Walker P. Involvement of Trp-284, Val-296, and Val-297 of the human delta-opioid receptor in binding of delta-selective ligands.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1996;271:18789&#8211;18796.</td><td><a href=http://dx.doi.org/10.1074/jbc.271.31.18789>10.1074/jbc.271.31.18789</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR61</td><td>BibArticle</td><td>Hjorth SA, Thirstrup K, Grandy DK, Schwartz TW. Analysis of selective binding epitopes for the kappa-opioid receptor antagonist norbinaltorphime.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 1995;47:1089&#8211;1094.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR62</td><td>BibArticle</td><td>Cavalli A, Babey AM, Loh HH. Altered adenylyl cyclase responsiveness subsequent to point mutations of Asp 128 in the third transmembrane domain of the delta-opioid receptor.<Emphasis Type="Italic">Neuroscience</Emphasis>. 1999;93:1025&#8211;1031.</td><td><a href=http://dx.doi.org/10.1016/S0306-4522(99)00280-8>10.1016/S0306-4522(99)00280-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR63</td><td>BibArticle</td><td>Li J, Huang P, Chen C, de Riel JK, Weinstein H, Liu-Chen LY. Constitutive activation of the mu opioid receptor by mutation of D3.49(164), but not D3.32(147): D3.49(164) is critical for stabilization of the inactive form of the receptor and for its expression.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2001;40:12039&#8211;12050.</td><td><a href=http://dx.doi.org/10.1021/bi0100945>10.1021/bi0100945</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR64</td><td>BibArticle</td><td>Mouledous L, Topham CM, Moisand C, Mollereau C, Meunier JC. Functional inactivation of the nociceptin receptor by alanine substitution of glutamine 286 at the C terminus of transmembrane segment VI: evidence from a site-directed mutagenesis study of the ORLI receptor transmembrane-binding domain.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 2000;57:495&#8211;502.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR65</td><td>BibArticle</td><td>DeCaillot FM, Befort K, Filliol D, Yue S, Walker P, Kieffer BL. Opioid receptor random mutagenesis reveals a mechanism for G protein-coupled receptor activation.<Emphasis Type="Italic">Nat Struct Biol</Emphasis>. 2003;10:629&#8211;636.</td><td><a href=http://dx.doi.org/10.1038/nsb950>10.1038/nsb950</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR66</td><td>BibArticle</td><td>Spivak CE, Beglan CL, Zollner C, Surratt CK. Beta-Funaltrexamine, a gauge for the recognition site of wildtype and mutant H297Q mu-opioid receptors.<Emphasis Type="Italic">Synapse</Emphasis>. 2003;49:55&#8211;60.</td><td><a href=http://dx.doi.org/10.1002/syn.10205>10.1002/syn.10205</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR67</td><td>BibArticle</td><td>Chen C, Yin J, Riel JK, et al. Determination of the amino acid residue involved in [3H]beta-funaltrexamine covalent binding in the cloned rat mu-opioid receptor.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1996;271:21422&#8211;21429.</td><td><a href=http://dx.doi.org/10.1074/jbc.271.35.21422>10.1074/jbc.271.35.21422</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR68</td><td>BibArticle</td><td>Jones RM, Hjorth SA, Schwartz TW, Portoghese PS. Mutational evidence for a common kappa antagonist binding pocket in the wild-type kappa and mutant mu[K303E] opioid receptors.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1998;41:4911&#8211;4914.</td><td><a href=http://dx.doi.org/10.1021/jm9805182>10.1021/jm9805182</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR69</td><td>BibArticle</td><td>Larson DL, Jones RM, Hjorth SA, Schwartz TW, Portoghese PS. Binding of norbinaltorphimine (norBNI) congeners to wild-type and mutant mu and kappa opioid receptors: molecular recognition loci for the pharmacophore and address components of kappa antagonists.<Emphasis Type="Italic">J Med Chem</Emphasis>. 2000;43:1573&#8211;1576.</td><td><a href=http://dx.doi.org/10.1021/jm000059g>10.1021/jm000059g</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR70</td><td>BibArticle</td><td>Fowler CB III, Pogozheva ID III, LeVine H III, Mosberg HI. Refinement of a homology model of the &#956;-opiodd receptor using distance constraints from intrinsic and engineered zinc-binding sites.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2004<Emphasis Type="Italic">b</Emphasis>;43:8700&#8211;8710.</td><td><a href=http://dx.doi.org/10.1021/bi036067r>10.1021/bi036067r</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR71</td><td>BibArticle</td><td>Metzger TG, Paterlini MG, Portoghese PS, Ferguson DM. An analysis of the conserved residues between halobacterial retinal proteins and G-protein coupled receptors: implications for GPCR modeling.<Emphasis Type="Italic">J Chem Inf Comput Sci</Emphasis>. 1996;36:857&#8211;861.</td><td><a href=http://dx.doi.org/10.1021/ci950360j>10.1021/ci950360j</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR72</td><td>BibArticle</td><td>Strahs D, Weinstein H. Comparative modeling and molecular dynamics studies of the delta, kappa and mu opioid receptors.<Emphasis Type="Italic">Protein Eng</Emphasis>. 1997;10:1019&#8211;1038.</td><td><a href=http://dx.doi.org/10.1093/protein/10.9.1019>10.1093/protein/10.9.1019</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR73</td><td>BibArticle</td><td>Alkorta I, Loew GH. A 3D model of the delta opioid receptor and ligand-receptor complexes.<Emphasis Type="Italic">Protein Eng</Emphasis>. 1996;9:573&#8211;583.</td><td><a href=http://dx.doi.org/10.1093/protein/9.7.573>10.1093/protein/9.7.573</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR74</td><td>BibArticle</td><td>Subramanian G, Paterlini MG, Larson DL, Portoghese PS, Ferguson DM. Conformational analysis and automated receptor docking of selective arylacetamide-based kappa-opioid agonists.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1998;41:4777&#8211;4789.</td><td><a href=http://dx.doi.org/10.1021/jm9803166>10.1021/jm9803166</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR75</td><td>BibArticle</td><td>Paterlini G, Portoghese PS, Ferguson DM. Molecular simulation of dynorphin A-(1&#8211;10) binding to extracellular loop 2 of the kappa-opioid receptor: a model for receptor activation.<Emphasis Type="Italic">J Med Chem</Emphasis>. 1997;40:3254&#8211;3262.</td><td><a href=http://dx.doi.org/10.1021/jm970252j>10.1021/jm970252j</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR76</td><td>BibArticle</td><td>Filizola M, Caakkonen L, Loew GH. 3D modeling, ligand binding and activations studies of the cloned mouse delta, mu, and kappa opioid receptors.<Emphasis Type="Italic">Protein Eng.</Emphasis> 1999<Emphasis Type="Italic">a</Emphasis>, 12:927&#8211;942.</td><td><a href=http://dx.doi.org/10.1093/protein/12.11.927>10.1093/protein/12.11.927</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR77</td><td>BibArticle</td><td>Filizola M, Carteni-Farina M, Perez JJ. Molecular modeling study of the differential ligand-receptor interaction at the mu, delta and kappa opioid receptors.<Emphasis Type="Italic">J Comput Aided Mol Des</Emphasis>. 1999<Emphasis Type="Italic">b</Emphasis>;13:397&#8211;407.</td><td><a href=http://dx.doi.org/10.1023/A:1008079823736>10.1023/A:1008079823736</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR78</td><td>BibArticle</td><td>Pogozheva ID, Lomize AL, Mosberg HI. The transmembrane 7 alpha-bundle of rhodopsin: distance geometry calculations with hydrogen bonding constraints.<Emphasis Type="Italic">Biophys J</Emphasis>. 1997;72:1963&#8211;1985.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/S0006-3495(97)78842-8>10.1016/S0006-3495(97)78842-8</a></td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, VolumeID</td><td>CrossRef</td></tr><tr><td>CR79</td><td>BibArticle</td><td>Lomize AL, Pogozheva ID, Mosberg HI. Structural organization of G-protein-coupled receptors.<Emphasis Type="Italic">J Comput Aided Mol Des</Emphasis>. 1999;13:325&#8211;353.</td><td><a href=http://dx.doi.org/10.1023/A:1008050821744>10.1023/A:1008050821744</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR80</td><td>BibArticle</td><td>Vaidehi N, Floriano WB, Trabanino R, et al. Prediction of structure and function of G protein-coupled receptors.<Emphasis Type="Italic">Proc Natl Acad Sci USA</Emphasis>. 2002;99:12622&#8211;12627.</td><td><a href=http://dx.doi.org/10.1073/pnas.122357199>10.1073/pnas.122357199</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR81</td><td>BibArticle</td><td>Shacham S, Topf M, Avisar N, et al. Modeling the 3D structure of GPCRs from sequence.<Emphasis Type="Italic">Med Res Rev</Emphasis>. 2001;21:472&#8211;483.</td><td><a href=http://dx.doi.org/10.1002/med.1019>10.1002/med.1019</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR82</td><td>BibArticle</td><td>Shi L, Javitch JA. The binding site of aminergic G protein-coupled receptors: the transmembrane segments and second extracellular loop.<Emphasis Type="Italic">Annu Rev Pharmacol Toxicol</Emphasis>. 2002;42:437&#8211;467.</td><td><a href=http://dx.doi.org/10.1146/annurev.pharmtox.42.091101.144224>10.1146/annurev.pharmtox.42.091101.144224</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR83</td><td>BibArticle</td><td>Lawson Z, Wheatley M. The third extracellular loop of G-protein-coupled receptors: more than just a linker between 2 important transmembrane helices.<Emphasis Type="Italic">Biochem Soc Trans</Emphasis>. 2004;32:1048&#8211;1050.</td><td><a href=http://dx.doi.org/10.1042/BST0321048>10.1042/BST0321048</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR84</td><td>BibArticle</td><td>Baker D, Sali A. Protein structure prediction and structural genomics.<Emphasis Type="Italic">Science</Emphasis>. 2001;294:93&#8211;96.</td><td><a href=http://dx.doi.org/10.1126/science.1065659>10.1126/science.1065659</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR85</td><td>BibArticle</td><td>Palczewski K, Kumasaka T, Hori T, et al. Crystal structure of rhodopsin: a G protein-coupled receptor.<Emphasis Type="Italic">Science</Emphasis>. 2000;289:739&#8211;745.</td><td><a href=http://dx.doi.org/10.1126/science.289.5480.739>10.1126/science.289.5480.739</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR86</td><td>BibArticle</td><td>Ballesteros JA, Shi L, Javitch JA. Structural mimicry in G protein-coupled receptors: implications of the high-resolution structure of rhodopsin for structure-function analysis of rhodopsin-like receptors.<Emphasis Type="Italic">Mol Pharmacol</Emphasis>. 2001;60:1&#8211;19.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR87</td><td>BibArticle</td><td>Klabunde T, Hessler G. Drug design strategies for targeting G-protein-coupled receptors.<Emphasis Type="Italic">ChemBioChem</Emphasis>. 2002;3:928&#8211;944.</td><td><a href=http://dx.doi.org/10.1002/1439-7633(20021004)3:10&lt;928::AID-CBIC928&gt;3.0.CO;2-5>10.1002/1439-7633(20021004)3:10&lt;928::AID-CBIC928&gt;3.0.CO;2-5</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR88</td><td>BibArticle</td><td>Bissantz C, Bernard P, Hibert M, Rognan D. Protein-based virtual screening of chemical databases. II. Are homology models of G-protein coupled receptors suitable targets?.<Emphasis Type="Italic">Proteins</Emphasis>. 2003;50:5&#8211;25.</td><td><a href=http://dx.doi.org/10.1002/prot.10237>10.1002/prot.10237</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR89</td><td>BibArticle</td><td>Evers A, Klebe G. Ligand-supported homology modeling of g-protein-coupled receptor sites: models sufficient for successful virtual screening.<Emphasis Type="Italic">Angew Chem Int Ed Engl</Emphasis>. 2004<Emphasis Type="Italic">a</Emphasis>;43:248&#8211;251.</td><td><a href=http://dx.doi.org/10.1002/anie.200352776>10.1002/anie.200352776</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR90</td><td>BibArticle</td><td>Horn F, Bettler E, Oliveira L, Campagne F, Cohen FE, Vriend G. GPCRDB information system for G protein-coupled receptors.<Emphasis Type="Italic">Nucleic Acids Res</Emphasis>. 2003;31:294&#8211;297.</td><td><a href=http://dx.doi.org/10.1093/nar/gkg103>10.1093/nar/gkg103</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR91</td><td>BibArticle</td><td>Huang P, Li J, Chen C, Visiers I, Weinstein H, Liu-Chen LY. Functional role of a conserved motif in TM6 of the rat mu opioid receptor: constitutively active and inactive receptors result from substitutions of Thr6.34(279) with Lys and Asp.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2001;40:13501&#8211;13509.</td><td><a href=http://dx.doi.org/10.1021/bi010917q>10.1021/bi010917q</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR92</td><td>BibArticle</td><td>Sali A, Blundell TL. Comparative protein modeling by satisfaction of spatial restraints.<Emphasis Type="Italic">J Mol Biol</Emphasis>. 1993;234:779&#8211;815.</td><td><a href=http://dx.doi.org/10.1006/jmbi.1993.1626>10.1006/jmbi.1993.1626</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR93</td><td>BibArticle</td><td>Fiser A, Sali A. Modeller: generation and refinement of homology-based protein structure models.<Emphasis Type="Italic">Methods Enzymol</Emphasis>. 2003;374:461&#8211;491.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/S0076-6879(03)74020-8>10.1016/S0076-6879(03)74020-8</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR94</td><td>BibArticle</td><td>Peitsch MC. ProMod and Swiss-Model: internet-based tools for automated comparative protein modelling.<Emphasis Type="Italic">Biochem Soc Trans</Emphasis>. 1996;24:274&#8211;279.</td><td><a href=http://dx.doi.org/10.1042/bst0240274>10.1042/bst0240274</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>ArticleTitle, VolumeID, Author_FamilyName_1, FirstPage, Year</td><td>CrossRef</td></tr><tr><td>CR95</td><td>BibArticle</td><td>Lund O, Frimand K, Gorodkin J, et al. Protein distance constraints predicted by neural networks and probability density functions.<Emphasis Type="Italic">Protein Eng</Emphasis>. 1997;10:1241&#8211;1248.</td><td><a href=http://dx.doi.org/10.1093/protein/10.11.1241>10.1093/protein/10.11.1241</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR96</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Shindyalov</span> <span style='background:#DDDDDD'>IN</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bourne</span> <span style='background:#DDDDDD'>PE.</span></span>, <span style='background:#BDBAD6'>Improving alignments in HM protocol with intermediate sequences. In: Forth Meeting on the Critical Assessment of Techniques for Protein Structure Prediction;</span></aug> <span style='background:#66FF66'>2000</span>:<span style='background:#D279FF'>A&#8211;92</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR97</td><td>BibArticle</td><td>Lambert C, Leonard N, De Bolle X, Depiereux E. ESyPred3D: prediction of proteins 3D structures.<Emphasis Type="Italic">Bioinformatics</Emphasis>. 2002;18:1250&#8211;1256.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/18.9.1250>10.1093/bioinformatics/18.9.1250</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR98</td><td>BibArticle</td><td>Kim DE, Chivian D, Baker D. Protein structure prediction and analysis using the Robetta server.<Emphasis Type="Italic">Nucleic Acids Res</Emphasis>. 2004;32:W526-W531.</td><td><a href=http://dx.doi.org/10.1093/nar/gkh468>10.1093/nar/gkh468</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR99</td><td>BibArticle</td><td>Pieper U, Eswar N, Braberg H, et al. MODBASE, a database of annotated comparative protein structure models, and associated resources.<Emphasis Type="Italic">Nucleic Acids Res</Emphasis>. 2004;32:D217-D222.</td><td><a href=http://dx.doi.org/10.1093/nar/gkh095>10.1093/nar/gkh095</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR100</td><td>BibArticle</td><td>Eswar N, John B, Mirkovic N, et al. Tools for comparative protein structure modeling and analysis.<Emphasis Type="Italic">Nucleic Acids Res</Emphasis>. 2003; 31:3375&#8211;3380.</td><td><a href=http://dx.doi.org/10.1093/nar/gkg543>10.1093/nar/gkg543</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR101</td><td>BibArticle</td><td>John B, Sali A. Comparative protein structure modeling by iterative alignment, model building and model assessment.<Emphasis Type="Italic">Nucleic Acids Res</Emphasis>. 2003;31:3982&#8211;3992.</td><td><a href=http://dx.doi.org/10.1093/nar/gkg460>10.1093/nar/gkg460</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR102</td><td>BibArticle</td><td>Riek RP, Rigoutsos I, Novotny J, Graham RM. Non-alpha-helical elements modulate polytopic membrane protein architecture.<Emphasis Type="Italic">J Mol Biol</Emphasis>. 2001;306:349&#8211;362.</td><td><a href=http://dx.doi.org/10.1006/jmbi.2000.4402>10.1006/jmbi.2000.4402</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR103</td><td>BibArticle</td><td>Fiser A, Do RK, Sali A. Modeling of loops in protein structures.<Emphasis Type="Italic">Protein Sci</Emphasis>. 2000;9:1753&#8211;1773.</td><td><a href=http://dx.doi.org/10.1110/ps.9.9.1753>10.1110/ps.9.9.1753</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Year, FirstPage, Author_FamilyName_2, Author_FamilyName_1, VolumeID, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR104</td><td>BibArticle</td><td>Chothia C, Lesk AM. Helix movements and the reconstruction of the haem pocket during the evolution of the cytochrome c family.<Emphasis Type="Italic">J Mol Biol.</Emphasis> 1985;182:151&#8211;158.</td><td><a href=http://dx.doi.org/10.1016/0022-2836(85)90033-6>10.1016/0022-2836(85)90033-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR105</td><td>BibArticle</td><td>Meng EC, Bourne HR. Receptor activation: what does the rhodopsin structure tell us?<Emphasis Type="Italic">Trends Pharmacol Sci</Emphasis>. 2001;22:587&#8211;593.</td><td><a href=http://dx.doi.org/10.1016/S0165-6147(00)01825-3>10.1016/S0165-6147(00)01825-3</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR106</td><td>BibArticle</td><td>Karnik SS, Gogonea C, Patil S, Saad Y, Takezako T. Activation of G-protein-coupled receptors: a common molecular mechanism.<Emphasis Type="Italic">Trends Endocrinol Metab.</Emphasis> 2003;14:431&#8211;437.</td><td><a href=http://dx.doi.org/10.1016/j.tem.2003.09.007>10.1016/j.tem.2003.09.007</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR107</td><td>BibArticle</td><td>Ghanouni P, Steenhuis JJ, Farrens DL, Kobilka BK. Agonist-induced conformational changes in the G-protein-coupling domain of the beta 2 adrenergic receptor.<Emphasis Type="Italic">Proc Natl Acad Sci USA</Emphasis>. 2001<Emphasis Type="Italic">a</Emphasis>;98:5997&#8211;6002.</td><td><a href=http://dx.doi.org/10.1073/pnas.101126198>10.1073/pnas.101126198</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR108</td><td>BibArticle</td><td>Farrens DL, Altenbach C, Yang K, Hubbell WL, Khorana HG. Requirement of rigid-body motion of transmembrane helices for light activation of rhodopsin.<Emphasis Type="Italic">Science</Emphasis>. 1996;274:768&#8211;770.</td><td><a href=http://dx.doi.org/10.1126/science.274.5288.768>10.1126/science.274.5288.768</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR109</td><td>BibArticle</td><td>Gether U, Lin S, Ghanouni P, Ballesteros JA, Weinstein H, Kobilka BK. Agonists induce conformational changes in transmembrane domains III and VI of the beta2 adrenoceptor.<Emphasis Type="Italic">EMBO J.</Emphasis> 1997;16:6737&#8211;6747.</td><td><a href=http://dx.doi.org/10.1093/emboj/16.22.6737>10.1093/emboj/16.22.6737</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR110</td><td>BibArticle</td><td>Dunham TD, Farrens DL. Conformational changes in rhodopsin: movement of helix F detected by site-specific chemical labeling and fluorescence spectroscopy.<Emphasis Type="Italic">J Biol Chem.</Emphasis> 1999;274:1683&#8211;1690.</td><td><a href=http://dx.doi.org/10.1074/jbc.274.3.1683>10.1074/jbc.274.3.1683</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR111</td><td>BibArticle</td><td>Han M, Smith SO, Sakmar TP. Constitutive activation of opsin by mutation of methionine 257 on transmembrane helix 6.<Emphasis Type="Italic">Biochemistry</Emphasis>. 1998;37:8253&#8211;8261.</td><td><a href=http://dx.doi.org/10.1021/bi980147r>10.1021/bi980147r</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR112</td><td>BibArticle</td><td>Cai K, Klein-Seetharaman J, Hwa J, Hubbell WL, Khorana HG. Structure and function in rhodopsin: effects of disulfide cross-links in the cytoplasmic face of rhodopsin on transducin activation and phosphorylation by rhodopsin kinase.<Emphasis Type="Italic">Biochemistry</Emphasis>. 1999;38:12893&#8211;12898.</td><td><a href=http://dx.doi.org/10.1021/bi9912443>10.1021/bi9912443</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR113</td><td>BibArticle</td><td>Ghanouni P, Gryczynski Z, Steenhuis JJ, et al. Functionally different agonists induce distinct conformations in the G protein coupling domain of the beta 2 adrenergic receptor.<Emphasis Type="Italic">J Biol Chem.</Emphasis> 2001<Emphasis Type="Italic">b</Emphasis>;276:24433&#8211;24436.</td><td><a href=http://dx.doi.org/10.1074/jbc.C100162200>10.1074/jbc.C100162200</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR114</td><td>BibArticle</td><td>Janz JM, Farrens DL. Rhodopsin activation exposes a key hydrophobic binding site for the transducin alpha-subunit C terminus.<Emphasis Type="Italic">J Biol Chem.</Emphasis> 2004;279:29767&#8211;29773.</td><td><a href=http://dx.doi.org/10.1074/jbc.M402567200>10.1074/jbc.M402567200</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR115</td><td>BibArticle</td><td>Hubbell WL, Altenbach C, Hubbell CM, Khorana HG. Rhodopsin structure, dynamics, and activation: a perspective from crystallography, site-directed spin labeling, sulfhydryl reactivity, and disulfide crosslinking.<Emphasis Type="Italic">Adv Protein Chem.</Emphasis> 2003;63:243&#8211;290.</td><td><a href=http://dx.doi.org/10.1016/S0065-3233(03)63010-X>10.1016/S0065-3233(03)63010-X</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR116</td><td>BibArticle</td><td>Lin SW, Sakmar TP. Specific tryptophan UV-absorbance changes are probes of the transition of rhodopsin to its active state.<Emphasis Type="Italic">Biochemistry</Emphasis>. 1996;35:11149&#8211;11159.</td><td><a href=http://dx.doi.org/10.1021/bi960858u>10.1021/bi960858u</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR117</td><td>BibArticle</td><td>Altenbach C, Cai K, Khorana HG, Hubbell WL. Structural features and light-dependent changes in the sequence 306&#8211;322 extending from helix VII to the palmitoylation sites in rhodopsin: a sitedrected spin-labeling study.<Emphasis Type="Italic">Biochemistry</Emphasis>. 1999<Emphasis Type="Italic">a</Emphasis>;38:7931&#8211;7937.</td><td><a href=http://dx.doi.org/10.1021/bi9900121>10.1021/bi9900121</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR118</td><td>BibArticle</td><td>Altenbach C, Klein-Seetharaman J, Hwa J, Khorana HG, Hubbell WL. Structural features and light-dependent changes in the sequence 59&#8211;75 connecting helices I and II in rhodopsin: a site-directed spinlabeling study.<Emphasis Type="Italic">Biochemistry</Emphasis>. 1999<Emphasis Type="Italic">b</Emphasis>;38:7945&#8211;7949.</td><td><a href=http://dx.doi.org/10.1021/bi990014l>10.1021/bi990014l</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR119</td><td>BibArticle</td><td>Altenbach C, Cai K, Klein-Seetharaman J, Khorana HG, Hubbell WL. Structure and function in rhodopsin: mapping light-dependent changes in distance between residue 65 in helix TM1 and residues in the sequence 306&#8211;319 at the cytoplasmic end of helix TM7 and in helix H8.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2001<Emphasis Type="Italic">a</Emphasis>;40:15483&#8211;15492.</td><td><a href=http://dx.doi.org/10.1021/bi011546g>10.1021/bi011546g</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR120</td><td>BibArticle</td><td>Altenbach C, Klein-Seetharaman J, Cai K, Khorana HG, Hubbell WL. Structure and function in rhodopsin: mapping light-dependent changes in distance between residue 316 in helix 8 and residues in the sequence 60&#8211;75, covering the cytoplasmic end of helices TM1 and TM2 and their connection loop CL1.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2001<Emphasis Type="Italic">b</Emphasis>;40:15493&#8211;15500.</td><td><a href=http://dx.doi.org/10.1021/bi011545o>10.1021/bi011545o</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR121</td><td>BibArticle</td><td>Gether U, Kobilka BK. G protein-coupled receptors. II. Mechanism of agonist activation.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 1998;273:17979&#8211;17982.</td><td><a href=http://dx.doi.org/10.1074/jbc.273.29.17979>10.1074/jbc.273.29.17979</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR122</td><td>BibArticle</td><td>Devanathan S, Yao Z, Salamon Z, Kobilka B, Tollin G. Plasmon-waveguide resonance studies of ligand binding to the human beta 2-adrenergic receptor.<Emphasis Type="Italic">Biochemistry</Emphasis>. 2004;43:3280&#8211;3288.</td><td><a href=http://dx.doi.org/10.1021/bi035825a>10.1021/bi035825a</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR123</td><td>BibArticle</td><td>Salamon Z, Cowell S, Varga E, Yamamura HI, Hruby VJ, Tollin G. Plasmon resonance studies of agonist/antagonist binding to the human delta-opioid receptor: new structural insights into receptor-ligand interactions.<Emphasis Type="Italic">Biophys J.</Emphasis> 2000;79:2463&#8211;2474.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR124</td><td>BibArticle</td><td>Salamon Z, Hruby VJ, Tollin G, Cowell S. Binding of agonists, antagonists and inverse agonists to the human delta-opioid receptor produces distinctly different conformational states distinguishable by plasmon-waveguide resonance spectroscopy.<Emphasis Type="Italic">J Pept Res.</Emphasis> 2002;60:322&#8211;328.</td><td><a href=http://dx.doi.org/10.1034/j.1399-3011.2002.21060.x>10.1034/j.1399-3011.2002.21060.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR125</td><td>BibArticle</td><td>Alves ID, Ciano KA, Boguslavski V, et al. Selectivity, cooperativity, and reciprocity in the interactions between the delta-opioid receptor, its ligands, and G-proteins.<Emphasis Type="Italic">J Biol Chem.</Emphasis> 2004<Emphasis Type="Italic">a</Emphasis>;279:44673&#8211;44682.</td><td><a href=http://dx.doi.org/10.1074/jbc.M404713200>10.1074/jbc.M404713200</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR126</td><td>BibArticle</td><td>Alves ID, Cowell SM, Salamon Z, Devanathan S, Tollin G, Hruby VJ. Different structural states of the proteolipid membrane are produced by ligand binding to the human delta-opioid receptor as shown by plasmon-waveguide resonance spectroscopy.<Emphasis Type="Italic">Mol Pharmacol.</Emphasis> 2004<Emphasis Type="Italic">b</Emphasis>;65:1248&#8211;1257.</td><td><a href=http://dx.doi.org/10.1124/mol.65.5.1248>10.1124/mol.65.5.1248</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR127</td><td>BibArticle</td><td>Tramontano A, Morea V. Exploiting evolutionary relationships for predicting protein structures.<Emphasis Type="Italic">Biotechnol Bioeng.</Emphasis> 2003;84:756&#8211;763.</td><td><a href=http://dx.doi.org/10.1002/bit.10850>10.1002/bit.10850</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR128</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bates</span> <span style='background:#DDDDDD'>PA</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Kelley</span> <span style='background:#DDDDDD'>LA</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>MacCallum</span> <span style='background:#DDDDDD'>RM</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Sternberg</span> <span style='background:#DDDDDD'>MJ</span></span></aug>. <span style='background:#CCCCFF'>Enhancement of protein modeling by human intervention in applying the automatic programs 3D-JIGSAW and 3D-PSSM</span>. <span style='background:#CCFF99'><i>Proteins</i></span>. <span style='background:#66FF66'>2001</span>;<span style='background:#D279FF'>39&#8211;46</span>.</td><td><a href=http://dx.doi.org/10.1002/prot.1168>10.1002/prot.1168</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, FirstPage, Year, Author_FamilyName_1, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR129</td><td>BibArticle</td><td>Li X, Jacobson MP, Friesner RA. High-resolution prediction of protein helix positions and orientations.<Emphasis Type="Italic">Proteins</Emphasis>. 2004;55:368&#8211;382.</td><td><a href=http://dx.doi.org/10.1002/prot.20014>10.1002/prot.20014</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR130</td><td>BibArticle</td><td>Evers A, Gohlke H, Klebe G. Ligand-supported homology modelling of protein binding-sites using knowledge-based potentials.<Emphasis Type="Italic">J Mol Biol.</Emphasis> 2003;334:327&#8211;345.</td><td><a href=http://dx.doi.org/10.1016/j.jmb.2003.09.032>10.1016/j.jmb.2003.09.032</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR131</td><td>BibArticle</td><td>Ward SD, Hamdan FF, Bloodworth LM, Wess J. Conformational changes that occur during M3 muscarinic acetylcholine receptor activation probed by the use of an in situ disulfide cross-linking strategy.<Emphasis Type="Italic">J Biol Chem.</Emphasis> 2002;277:2247&#8211;2257.</td><td><a href=http://dx.doi.org/10.1074/jbc.M107647200>10.1074/jbc.M107647200</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR132</td><td>BibArticle</td><td>Swaminath G, Lee TW, Kobilka B. Identification of an allosteric binding site for Zn<Superscript>2+</Superscript> on the beta2 adrenergic receptor.<Emphasis Type="Italic">J Biol Chem.</Emphasis> 2003;278:352&#8211;356.</td><td><a href=http://dx.doi.org/10.1074/jbc.M206424200>10.1074/jbc.M206424200</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR133</td><td>BibArticle</td><td>Elling CE, Thirstrup K, Holst B, Schwartz TW. Conversion of agonist site to metal-ion chelator site in the beta(2)-adrenergic receptor.<Emphasis Type="Italic">Proc Natl Acad Sci USA</Emphasis>. 1999;96:12322&#8211;12327.</td><td><a href=http://dx.doi.org/10.1073/pnas.96.22.12322>10.1073/pnas.96.22.12322</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR134</td><td>BibArticle</td><td>Holst B, Elling CE, Schwartz TW. Partial agonism through a zincion switch constructed between transmembrane domains III and VII in the tachykinin NK(1) receptor.<Emphasis Type="Italic">Mol Pharmacol.</Emphasis> 2000;58:263&#8211;270.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR135</td><td>BibArticle</td><td>Lagerstrom MC, Klovins J, Fredriksson R, et al. High affinity agonistic metal ion binding sites within the melanocortin 4 receptor illustrate conformational change of transmembrane region 3.<Emphasis Type="Italic">J Biol Chem</Emphasis>. 2003;278:51521&#8211;51526.</td><td><a href=http://dx.doi.org/10.1074/jbc.M307683200>10.1074/jbc.M307683200</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR136</td><td>BibArticle</td><td>Ewing TJ, Makino S, Skillman AG, Kuntz ID. DOCK 4.0: search strategies for automated molecular docking of flexible molecule databases.<Emphasis Type="Italic">J Comput Aided Mol Des.</Emphasis> 2001;15:411&#8211;428.</td><td><a href=http://dx.doi.org/10.1023/A:1011115820450>10.1023/A:1011115820450</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR137</td><td>BibArticle</td><td>Jones G, Willett P, Glen RC, Leach AR, Taylor R. Development and validation of a genetic algorithm for flexible docking.<Emphasis Type="Italic">J Mol Biol</Emphasis>. 1997;267:727&#8211;748.</td><td><a href=http://dx.doi.org/10.1006/jmbi.1996.0897>10.1006/jmbi.1996.0897</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR138</td><td>BibArticle</td><td>Rarey M, Wefing S, Lengauer T. Placement of medium-sized molecular fragments into active sites of proteins.<Emphasis Type="Italic">J Comput Aided Mol Des.</Emphasis> 1996;10:41&#8211;54.</td><td><a href=http://dx.doi.org/10.1007/BF00124464>10.1007/BF00124464</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR139</td><td>BibArticle</td><td>Taylor RD, Jewsbury PJ, Essex JW. FDS: flexible ligand and receptor docking with a continuum solvent model and soft-core energy function.<Emphasis Type="Italic">J Comput Chem.</Emphasis> 2003;24:1637&#8211;1656.</td><td><a href=http://dx.doi.org/10.1002/jcc.10295>10.1002/jcc.10295</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR140</td><td>BibArticle</td><td>Halgren TA, Murphy RB, Friesner RA, et al. Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening.<Emphasis Type="Italic">J Med Chem.</Emphasis> 2004;47:1750&#8211;1759.</td><td><a href=http://dx.doi.org/10.1021/jm030644s>10.1021/jm030644s</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR141</td><td>BibArticle</td><td>Venkatachalam CM, Jiang X, Oldfield T, Waldman M. LigandFit: a novel method for the shape-directed rapid docking of ligands to protein active sites.<Emphasis Type="Italic">J Mol Graph Model.</Emphasis> 2003;21:289&#8211;307.</td><td><a href=http://dx.doi.org/10.1016/S1093-3263(02)00164-X>10.1016/S1093-3263(02)00164-X</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR142</td><td>BibArticle</td><td>Cavasotto CN, Abagan RA. Protein flexibility in ligand docking and virtual screening to protein kinases.<Emphasis Type="Italic">J Mol Biol.</Emphasis> 2004;337:209&#8211;225.</td><td><a href=http://dx.doi.org/10.1016/j.jmb.2004.01.003>10.1016/j.jmb.2004.01.003</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR143</td><td>BibArticle</td><td>Perola E, Walters WP, Charifson PS. A detailed comparison of current docking and scoring methods on systems of pharmaceutical relevance.<Emphasis Type="Italic">Proteins</Emphasis>. 2004;56:235&#8211;249.</td><td><a href=http://dx.doi.org/10.1002/prot.20088>10.1002/prot.20088</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR144</td><td>BibArticle</td><td>Halperin I, Ma B, Wolfson H, Nussinov R. Principles of docking: an overview of search algorithms and a guide to scoring functions.<Emphasis Type="Italic">Proteins.</Emphasis> 2004;47:409&#8211;443.</td><td><a href=http://dx.doi.org/10.1002/prot.10115>10.1002/prot.10115</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR145</td><td>BibArticle</td><td>Kontoyianni M, McClellan LM, Sokol GS. Evaluation of docking performance: comparative data on docking algorithms.<Emphasis Type="Italic">J Med Chem.</Emphasis> 2004;47:558&#8211;565.</td><td><a href=http://dx.doi.org/10.1021/jm0302997>10.1021/jm0302997</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR146</td><td>BibArticle</td><td>Carlson HA, McCammon JA. Accommodating protein flexibility in computational drug design.<Emphasis Type="Italic">Mol Pharmacol.</Emphasis> 2000;57:213&#8211;218.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR147</td><td>BibArticle</td><td>Evers A, Klebe G. Successful virtual screening for a submicromolar antagonist of the neurokinin-1 receptor based on a ligand-supported homology model.<Emphasis Type="Italic">J Med Chem.</Emphasis> 2004<Emphasis Type="Italic">b</Emphasis>;47:5381&#8211;5392.</td><td><a href=http://dx.doi.org/10.1021/jm0311487>10.1021/jm0311487</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR148</td><td>BibArticle</td><td>Heyl DL, Mosberg HI. Modification of the Phe3 aromatic moiety in delta receptor-selective dermorphin/deltorphin-related tetrapeptides: effects on opioid receptor binding.<Emphasis Type="Italic">Int J Pept Protein Res.</Emphasis> 1992;39:450&#8211;457.</td><td><a href=http://dx.doi.org/10.1111/j.1399-3011.1992.tb01449.x>10.1111/j.1399-3011.1992.tb01449.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR149</td><td>BibArticle</td><td>Sebastian A, Bidlack JM, Jiang Q, et al. 14 beta-[(p-nitrocinnamoy 1)amino]morphinones, 14 beta-[(p-nitrocinnamoyl)amino]-7,8-dihydromorphinones, and their codeinone analogues: synthesis and receptor activity.<Emphasis Type="Italic">J Med Chem.</Emphasis> 1993;36:3154&#8211;3160.</td><td><a href=http://dx.doi.org/10.1021/jm00073a015>10.1021/jm00073a015</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR150</td><td>BibArticle</td><td>Sagara T, Egashira H, Okamura M, Fujii I, Shimohigashi Y, Kanematsu K. Ligand recognition in mu opioid receptor: experimentally based modeling of mu opioid receptor binding sites and their testing by ligand docking.<Emphasis Type="Italic">Bioorg Med Chem.</Emphasis> 1996;4:2151&#8211;2166.</td><td><a href=http://dx.doi.org/10.1016/S0968-0896(96)00219-2>10.1016/S0968-0896(96)00219-2</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR151</td><td>BibArticle</td><td>Chabre M, Breton J. Orientation of aromatic residues in rhodopsin: rotation of one tryptophan upon the meta I to meta II transition after illumination.<Emphasis Type="Italic">Photochem Photobiol</Emphasis>. 1979;30:295&#8211;299.</td><td><a href=http://dx.doi.org/10.1111/j.1751-1097.1979.tb07150.x>10.1111/j.1751-1097.1979.tb07150.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR152</td><td>BibArticle</td><td>Baneres JL, Martin A, Hullot P, Girard JP, Rossi JC, Parello J Structure-based analysis of GPCR function: conformational adaptation of both agonist and receptor upon leukotriene B4 binding to recombinant BLT1.<Emphasis Type="Italic">J Mol Biol.</Emphasis> 2003;329:801&#8211;814.</td><td><a href=http://dx.doi.org/10.1016/S0022-2836(03)00438-8>10.1016/S0022-2836(03)00438-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR153</td><td>BibArticle</td><td>Ruprecht JJ, Mielke T, Vogel R, Villa C, Schertler GF. Electron crystallography reveals the structure of metarhodopsin I.<Emphasis Type="Italic">EMBO J.</Emphasis> 2004;23:3609&#8211;3620.</td><td><a href=http://dx.doi.org/10.1038/sj.emboj.7600374>10.1038/sj.emboj.7600374</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR154</td><td>BibChapter</td><td>Sammes PG, Taylor JB. Opioid receptors. In: Hansch C, ed.<Emphasis Type="Italic">Comprehensive Medicinal Chemistry</Emphasis>. Oxford, UK: Pergamon Press; 1990;805&#8211;844.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1016/0223-5234(90)90012-r>10.1016/0223-5234(90)90012-r</a></td><td><b>CrossRef-Web</b></td>
<td>Year</td><td>CrossRef</td></tr><tr><td>CR155</td><td>BibArticle</td><td>Schiller PW, Weltrowska G, Nguyen TM-D, Lemieux C, Chung NN, Lu Y. Conversion of &#956;-, &#948;- and &#954;-receptor selective opioid peptide agonists into &#956;-, &#948;-, and &#954;-receptor selective antagonists.<Emphasis Type="Italic">Life Sci.</Emphasis> 2003;73:691&#8211;698.</td><td><a href=http://dx.doi.org/10.1016/S0024-3205(03)00389-8>10.1016/S0024-3205(03)00389-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR156</td><td>BibArticle</td><td>Huang P, Kim S, Loew G. Development of a common 3D pharmacophore for delta-opioid recognition from peptides and non-peptides using a novel computer program.<Emphasis Type="Italic">J Comput Aided Mol Des.</Emphasis> 1997;11:21&#8211;28.</td><td><a href=http://dx.doi.org/10.1023/A:1008067209563>10.1023/A:1008067209563</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR157</td><td>BibArticle</td><td>Filizola M, Villar HO, Loew GH. Molecular determinants of non-specific recognition of delta, mu, and kappa opioid receptors.<Emphasis Type="Italic">Bioorg Med Chem.</Emphasis> 2001<Emphasis Type="Italic">a</Emphasis>;9:69&#8211;76.</td><td><a href=http://dx.doi.org/10.1016/S0968-0896(00)00223-6>10.1016/S0968-0896(00)00223-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR158</td><td>BibArticle</td><td>Filizola M, Villar HO, Loew GH. Differentiation of delta, mu, and kappa opioid receptor agonists based on pharmacophore development and computed physicochemical properties.<Emphasis Type="Italic">J Comput Aided Mol Des.</Emphasis> 2001<Emphasis Type="Italic">b</Emphasis>;15:297&#8211;307.</td><td><a href=http://dx.doi.org/10.1023/A:1011187320095>10.1023/A:1011187320095</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR159</td><td>BibArticle</td><td>Bernard D Jr, Coop A Jr, MacKerell AD Jr. 2D conformationally sampled pharmacophore: a ligand-based pharmacophore to differentiate delta opioid agonists from antagonists.<Emphasis Type="Italic">J Am Chem Soc.</Emphasis> 2003;125:3101&#8211;3107.</td><td><a href=http://dx.doi.org/10.1021/ja027644m>10.1021/ja027644m</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr></table>