<H1>Chapter DOI: 10.1186/s12864-016-2817-9<br/>Cited-By Count: 0</H1><table border="1" width="30%"><tr><td>Total References</td><td>81</td></tr><tr><td>Springer references</td><td>13</td></tr><tr><td>Non Springer references</td><td>68</td></tr><tr><td>BibStructured Count</td><td width="10%">70</td></tr><tr><td>BibUnstructured Count</td><td width="10%">11</td></tr><tr><td>DOI already available in SpringerLink</td><td>57</td></tr><tr><td>Additional DOI extracted from CrossRef-Meta</td><td>8</td></tr><tr><td>Additional DOI extracted from CrossRef-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from ResearchGate-Web</td><td>0</td></tr><tr><td>Additional DOI extracted from SpringerLink-FUZZY-Search</td><td>0</td></tr><tr><td>Spelling Mistakes</td><td>NULL</td></tr><tr><td>Wrong DOI</td><td>0</td></tr><tr><td>ISBN</td><td>NULL</td></tr><tr><td>Web Link</td><td>NULL</td></tr><tr><td>DOI Obtained For Springer Published</td><td>0</td></tr><tr><td>DOI Obtained For Non Springer</td><td>8</td></tr><tr><td>DOI Obtained From Structured Ref</td><td>3</td></tr><tr><td>DOI Obtained From Unstructured Ref</td><td>5</td></tr></table></br></br><table border="1">
<tr><th>Bib. ID</th><th>Ref. Type</th><th>References</th><th>DOI</th><th>Suspected DOI</th><th>DOI Source</th><th>Validated Elements</th><th>Validation Source</th></tr><tr><td>CR1</td><td>BibArticle</td><td>D'Hont A, Grivet L, Feldmann P, Glaszmann JC, Rao S, Berding N. Characterisation of the double genome structure of modern sugarcane cultivars (Saccharum spp.) by molecular cytogenetics. Molecular and General Genetics MGG. 1996;250(4):405&#8211;13. %@ 0026&#8211;8925.</td><td><a href=http://dx.doi.org/10.1007/BF02174028>10.1007/BF02174028</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR2</td><td>BibArticle</td><td>Price S. Cytogenetics of modern sugar canes. Economic Botany. 1963;17(2):97&#8211;106. %@ 0013&#8211;0001.</td><td><a href=http://dx.doi.org/10.1007/BF02985359>10.1007/BF02985359</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR3</td><td>BibBook</td><td>Price S. Interspecific hybridization in sugarcane breeding, vol. 1965. 1965. p. 1021&#8211;6.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR4</td><td>BibBook</td><td>Berding N, Roach BT. Germplasm collection, maintenance, and use, vol. 143. Amsterdam: Sugarcane improvement through breeding Elsevier; 1987. p. 210.</td><td><a href=http://dx.doi.org/10.1016/B978-0-444-42769-4.50009-6>10.1016/B978-0-444-42769-4.50009-6</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>BookTitle, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR5</td><td>BibChapter</td><td>Grivet L, D'Hont A, Roques D, Feldmann P, Lanaud C, Glaszmann JC. RFLP mapping in cultivated sugarcane. In: Saccharum. 1996.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR6</td><td>BibArticle</td><td>Hoarau JY, Grivet L, Offmann B, Raboin LM, Diorflar JP, Payet J, Hellmann M, D'Hont A, Glaszmann JC. Genetic dissection of a modern sugarcane cultivar (Saccharum spp.). II. Detection of QTLs for yield components. Theor Appl Genet. 2002;105(6&#8211;7):1027&#8211;37. %@ 0040&#8211;5752.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR7</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Piperidis</span> <span style='background:#DDDDDD'>G</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>D'Hont</span> <span style='background:#DDDDDD'>A</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hogarth</span> <span style='background:#DDDDDD'>DM</span></span></aug>. <span style='background:#66FF66'>2001</span>: <span style='background:#FFD9B3'>Chromosome composition analysis of various Saccharum interspecific hybrids by genomic in situ hybridisation (GISH)</span>. <span style='background:#C0FFC0'>Brisbane, Australia</span>: <span style='background:#FFFF0F'>Australian Society of Sugar Cane Technologists; 2001. p.&amp;nbsp;565&#8211;6. %@ 0949678104</span></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR8</td><td>BibArticle</td><td>Cuadrado A, Acevedo R, De La Espina SMD, Jouve N, De La Torre C. Genome remodelling in three modern S. officinarum&#215; S. spontaneum sugarcane cultivars. J Exp Bot. 2004;55(398):847&#8211;54. %@ 0022&#8211;0957.</td><td><a href=http://dx.doi.org/10.1093/jxb/erh093>10.1093/jxb/erh093</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR9</td><td>BibArticle</td><td>Bremer G. Problems in breeding and cytology of sugar cane. Euphytica. 1961;10(1):59&#8211;78. %@ 0014&#8211;2336.</td><td><a href=http://dx.doi.org/10.1007/BF00037206>10.1007/BF00037206</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR10</td><td>BibArticle</td><td>Tomkins JP, Yu Y, Miller-Smith H, Frisch DA, Woo SS, Wing RA. A bacterial artificial chromosome library for sugarcane. Theor Appl Genet. 1999;99(3&#8211;4):419&#8211;24. %@ 0040&#8211;5752.</td><td><a href=http://dx.doi.org/10.1007/s001220051252>10.1007/s001220051252</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR11</td><td>BibArticle</td><td>D&#8217;hont A. Unraveling the genome structure of polyploids using FISH and GISH; examples of sugarcane and banana. Cytogenet Genome Res. 2005;109(1&#8211;3):27&#8211;33. %@ 1424-1859X.</td><td><a href=http://dx.doi.org/10.1159/000082378>10.1159/000082378</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR12</td><td>BibArticle</td><td>Zhang J, Nagai C, Yu Q, Pan Y-B, Ayala-Silva T, Schnell RJ, Comstock JC, Arumuganathan AK, Ming R. Genome size variation in three Saccharum species. Euphytica. 2012;185(3):511&#8211;9. %@ 0014&#8211;2336.</td><td><a href=http://dx.doi.org/10.1007/s10681-012-0664-6>10.1007/s10681-012-0664-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR13</td><td>BibArticle</td><td>Daugrois JH, Grivet L, Roques D, Hoarau JY, Lombard H, Glaszmann JC, d'Hont A. A putative major gene for rust resistance linked with a RFLP marker in sugarcane cultivar &#8216;R570&#8217;. Theor Appl Genet. 1996;92(8):1059&#8211;64. %@ 0040&#8211;5752.</td><td><a href=http://dx.doi.org/10.1007/BF00224049>10.1007/BF00224049</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR14</td><td>BibArticle</td><td>Asnaghi C, D'Hont A, Glaszmann JC, Rott P. Resistance of Sugarcane Cultivar R 570 to Puccinia melanocephala Isolatesfrom Different Geographic Locations. Plant Disease. 2001;85(3):282&#8211;6. %@ 0191&#8211;2917.</td><td><a href=http://dx.doi.org/10.1094/PDIS.2001.85.3.282>10.1094/PDIS.2001.85.3.282</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR15</td><td>BibArticle</td><td>Asnaghi C, Paulet F, Kaye C, Grivet L, Deu M, Glaszmann JC, D&#8217;Hont A. Application of synteny across Poaceae to determine the map location of a sugarcane rust resistance gene. Theor Appl Genet. 2000;101(5&#8211;6):962&#8211;9. %@ 0040&#8211;5752.</td><td><a href=http://dx.doi.org/10.1007/s001220051568>10.1007/s001220051568</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR16</td><td>BibArticle</td><td>Asnaghi C, Roques D, Ruffel S, Kaye C, Hoarau JY, Telismart H, Girard JC, Raboin LM, Risterucci AM, Grivet L. Targeted mapping of a sugarcane rust resistance gene (Bru1) using bulked segregant analysis and AFLP markers. Theor Appl Genet. 2004;108(4):759&#8211;64. %@ 0040&#8211;5752.</td><td><a href=http://dx.doi.org/10.1007/s00122-003-1487-6>10.1007/s00122-003-1487-6</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR17</td><td>BibArticle</td><td>Le Cunff L, Garsmeur O, Raboin LM, Pauquet J, Telismart H, Selvi A, Grivet L, Philippe R, Begum D, Deu M. Diploid/polyploid syntenic shuttle mapping and haplotype-specific chromosome walking toward a rust resistance gene (Bru1) in highly polyploid sugarcane (2n&#8764; 12x&#8764; 115). Genetics. 2008;180(1):649&#8211;60. %@ 0016&#8211;6731.</td><td><a href=http://dx.doi.org/10.1534/genetics.108.091355>10.1534/genetics.108.091355</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR18</td><td>BibArticle</td><td>Garsmeur O, Charron C, Bocs S, Jouffe V, Samain S, Couloux A, Droc G, Zini C, Glaszmann JC, Van Sluys MA. High homologous gene conservation despite extreme autopolyploid redundancy in sugarcane. New Phytologist. 2011;189(2):629&#8211;42. %@ 1469&#8211;8137.</td><td><a href=http://dx.doi.org/10.1111/j.1469-8137.2010.03497.x>10.1111/j.1469-8137.2010.03497.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR19</td><td>BibArticle</td><td>Fitch MM, Moore PH. Production of haploid Saccharum spontaneum L.-comparison of media for cold incubation of panicle branches and for float culture of anthers. J Plant Physiol. 1984;117(2):169&#8211;78. %@ 0176&#8211;1617.</td><td><a href=http://dx.doi.org/10.1016/S0176-1617(84)80031-0>10.1016/S0176-1617(84)80031-0</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR20</td><td>BibArticle</td><td>Ming R, Moore PH, Zee F, Abbey CA, Ma H, Paterson AH. Construction and characterization of a papaya BAC library as a foundation for molecular dissection of a tree-fruit genome. Theor Appl Genet. 2001;102(6&#8211;7):892&#8211;9. %@ 0040&#8211;5752.</td><td><a href=http://dx.doi.org/10.1007/s001220000448>10.1007/s001220000448</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR21</td><td>BibArticle</td><td>Yu Q, Guyot R, de Kochko A, Byers A, Navajas&#8208;P&#233;rez R, Langston BJ, Dubreuil&#8208;Tranchant C, Paterson AH, Poncet V, Nagai C. Micro&#8208;collinearity and genome evolution in the vicinity of an ethylene receptor gene of cultivated diploid and allotetraploid coffee species (Coffea). Plant J. 2011;67(2):305&#8211;17. %@ 1365-1313X.</td><td><a href=http://dx.doi.org/10.1111/j.1365-313X.2011.04590.x>10.1111/j.1365-313X.2011.04590.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR22</td><td>BibArticle</td><td>Ouyang S, Buell CR. The TIGR Plant Repeat Databases: a collective resource for the identification of repetitive sequences in plants. Nucleic Acids Res. 2004;32 suppl 1:D360&#8211;3. %@ 0305&#8211;1048.</td><td><a href=http://dx.doi.org/10.1093/nar/gkh099>10.1093/nar/gkh099</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR23</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Nussbaumer</span> <span style='background:#DDDDDD'>T</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Martis</span> <span style='background:#DDDDDD'>MM</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Roessner</span> <span style='background:#DDDDDD'>SK</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Pfeifer</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bader</span> <span style='background:#DDDDDD'>KC</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Sharma</span> <span style='background:#DDDDDD'>S</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Gundlach</span> <span style='background:#DDDDDD'>H</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Spannagl</span> <span style='background:#DDDDDD'>M.</span></span></aug> <span style='background:#CCCCFF'>MIPS a database framework for comparative plant genome research</span>. <span style='background:#CCFF99'>Nucleic Acids Res</span>. <span style='background:#66FF66'>2013</span>;41(D1):D1144&#8211;51. %@ <span style='background:#D279FF'>0305&#8211;1048</span>.</td><td>&nbsp;</td><td><a href=http://dx.doi.org/10.1093/nar/gks1153>10.1093/nar/gks1153</a></td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR24</td><td>BibArticle</td><td>Jurka J, Kapitonov VV, Pavlicek A, Klonowski P, Kohany O, Walichiewicz J. Repbase Update, a database of eukaryotic repetitive elements. Cytogenet Genome Res. 2005;110(1&#8211;4):462&#8211;7. %@ 1424-1859X.</td><td><a href=http://dx.doi.org/10.1159/000084979>10.1159/000084979</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR25</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Chen</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hu</span> <span style='background:#DDDDDD'>Q</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Zhang</span> <span style='background:#DDDDDD'>Y</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Lu</span> <span style='background:#DDDDDD'>C</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Kuang</span> <span style='background:#DDDDDD'>H</span></span></aug>: <span style='background:#FFD9B3'>P-MITE: a database for plant miniature inverted-repeat transposable elements</span>. Nucleic acids research <span style='background:#66FF66'>2013</span></td><td><a href=http://dx.doi.org/10.1093/nar/gkt1000>10.1093/nar/gkt1000</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, BookTitle, Year, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR26</td><td>BibArticle</td><td>Swarbreck D, Wilks C, Lamesch P, Berardini TZ, Garcia-Hernandez M, Foerster H, Li D, Meyer T, Muller R, Ploetz L. The Arabidopsis Information Resource (TAIR): gene structure and function annotation. Nucleic Acids Res. 2008;36 suppl 1:D1009&#8211;14. %@ 0305&#8211;1048.</td><td><a href=http://dx.doi.org/10.1093/nar/gkm965>10.1093/nar/gkm965</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>FirstPage, Author_FamilyName_1, VolumeID, Author_FamilyName_2, ArticleTitle</td><td>CrossRef</td></tr><tr><td>CR27</td><td>BibArticle</td><td>Vogel JP, Garvin DF, Mockler TC, Schmutz J, Rokhsar D, Bevan MW, Barry K, Lucas S, Harmon-Smith M, Lail K. Genome sequencing and analysis of the model grass Brachypodium distachyon. Nature. 2010;463(7282):763&#8211;8. %@ 0028&#8211;0836.</td><td><a href=http://dx.doi.org/10.1038/nature08747>10.1038/nature08747</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR28</td><td>BibArticle</td><td>Ouyang S, Zhu W, Hamilton J, Lin H, Campbell M, Childs K, Malek RL, Lee A, Zheng L, Orvis J. The TIGR Rice Genome Annotation Resource: improvements and new features. Nucleic Acids Res. 2007;35:D883&#8211;887.</td><td><a href=http://dx.doi.org/10.1093/nar/gkl976>10.1093/nar/gkl976</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR29</td><td>BibArticle</td><td>Bennetzen JL, Schmutz J, Wang H, Percifield R, Hawkins J, Pontaroli AC, Estep M, Feng L, Vaughn JN, Grimwood J. Reference genome sequence of the model plant Setaria. Nat Biotechnol. 2012;30(6):555&#8211;61. %@ 1087&#8211;0156.</td><td><a href=http://dx.doi.org/10.1038/nbt.2196>10.1038/nbt.2196</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR30</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Schnable</span> <span style='background:#DDDDDD'>PS</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Ware</span> <span style='background:#DDDDDD'>D</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Fulton</span> <span style='background:#DDDDDD'>RS</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Stein</span> <span style='background:#DDDDDD'>JC</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Wei</span> <span style='background:#DDDDDD'>F</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Pasternak</span> <span style='background:#DDDDDD'>S</span></span>, et al.</aug> <span style='background:#CCCCFF'>The B73 maize genome: complexity, diversity, and dynamics</span>. <span style='background:#CCFF99'>Science</span>. <span style='background:#66FF66'>2009</span>;326(5956):1112&#8211;5. %@ <span style='background:#D279FF'>0036&#8211;8075</span>.</td><td><a href=http://dx.doi.org/10.1126/science.1178534>10.1126/science.1178534</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>JournalTitle, ArticleTitle, Author_FamilyName_2, Author_FamilyName_1, Year</td><td>CrossRef</td></tr><tr><td>CR31</td><td>BibArticle</td><td>Paterson AH, Bowers JE, Bruggmann R, Dubchak I, Grimwood J, Gundlach H, Haberer G, Hellsten U, Mitros T, Poliakov A. The Sorghum bicolor genome and the diversification of grasses. Nature. 2009;457(7229):551&#8211;6. %@ 0028&#8211;0836.</td><td><a href=http://dx.doi.org/10.1038/nature07723>10.1038/nature07723</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR32</td><td>BibArticle</td><td>Cantarel BL, Korf I, Robb SMC, Parra G, Ross E, Moore B, Holt C, Alvarado AS, Yandell M. MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes. Genome Res. 2008;18(1):188&#8211;96. %@ 1088&#8211;9051.</td><td><a href=http://dx.doi.org/10.1101/gr.6743907>10.1101/gr.6743907</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR33</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Llorens</span> <span style='background:#DDDDDD'>C</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Futami</span> <span style='background:#DDDDDD'>R</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Covelli</span> <span style='background:#DDDDDD'>L</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Dom&#237;nguez-Escrib&#225;</span> <span style='background:#DDDDDD'>L</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Viu</span> <span style='background:#DDDDDD'>JM</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Tamarit</span> <span style='background:#DDDDDD'>D</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Aguilar-Rodr&#237;guez</span> <span style='background:#DDDDDD'>J</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Vicente-Ripolles</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Fuster</span> <span style='background:#DDDDDD'>G</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bernet</span> <span style='background:#DDDDDD'>GP</span></span></aug>: The Gypsy Database (GyDB) of mobile genetic elements: release 2.0. Nucleic acids research <span style='background:#66FF66'>2010</span>:gkq1061 %@ <span style='background:#D279FF'>0305&#8211;1048</span></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR34</td><td>BibBook</td><td>Smit AFA, Hubley R, Green P. RepeatMasker Open-3.0. 1996.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR35</td><td>BibArticle</td><td>Flutre T, Duprat E, Feuillet C, Quesneville H. Considering transposable element diversification in de novo annotation approaches. PLoS One. 2011;6(1):e16526. %@ 11932&#8211;16203.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0016526>10.1371/journal.pone.0016526</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR36</td><td>BibBook</td><td>Smit AFA, Hubley R. RepeatModeler Open-1.0. 2008.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR37</td><td>BibArticle</td><td>Li W, Godzik A. Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. Bioinformatics. 2006;22(13):1658&#8211;9. %@ 1367&#8211;4803.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btl158>10.1093/bioinformatics/btl158</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR38</td><td>BibArticle</td><td>Fu L, Niu B, Zhu Z, Wu S, Li W. CD-HIT: accelerated for clustering the next-generation sequencing data. Bioinformatics. 2012;28(23):3150&#8211;2. %@ 1367&#8211;4803.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/bts565>10.1093/bioinformatics/bts565</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR39</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Han</span> <span style='background:#DDDDDD'>Y</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Wessler</span> <span style='background:#DDDDDD'>SR</span></span></aug>: <span style='background:#FFD9B3'>MITE-Hunter: a program for discovering miniature inverted-repeat transposable elements from genomic sequences</span>. Nucleic acids research <span style='background:#66FF66'>2010</span>:gkq862 %@ <span style='background:#D279FF'>0305&#8211;1048</span></td><td><a href=http://dx.doi.org/10.1093/nar/gkq862>10.1093/nar/gkq862</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, BookTitle, Year, Author_FamilyName_1</td><td>CrossRef</td></tr><tr><td>CR40</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Marchler-Bauer</span> <span style='background:#DDDDDD'>A</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Zheng</span> <span style='background:#DDDDDD'>C</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Chitsaz</span> <span style='background:#DDDDDD'>F</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Derbyshire</span> <span style='background:#DDDDDD'>MK</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Geer</span> <span style='background:#DDDDDD'>LY</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Geer</span> <span style='background:#DDDDDD'>RC</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Gonzales</span> <span style='background:#DDDDDD'>NR</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Gwadz</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Hurwitz</span> <span style='background:#DDDDDD'>DI</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Lanczycki</span> <span style='background:#DDDDDD'>CJ</span></span></aug>: <span style='background:#FFD9B3'>CDD: conserved domains and protein three-dimensional structure</span>. Nucleic acids research <span style='background:#66FF66'>2012</span>:gks1243 %@ <span style='background:#D279FF'>0305&#8211;1048</span></td><td><a href=http://dx.doi.org/10.1093/nar/gks1243>10.1093/nar/gks1243</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_1, Year, BookTitle, Author_FamilyName_2</td><td>CrossRef</td></tr><tr><td>CR41</td><td>BibArticle</td><td>SanMiguel P, Gaut BS, Tikhonov A, Nakajima Y, Bennetzen JL. The paleontology of intergene retrotransposons of maize. Nat Genet. 1998;20(1):43&#8211;5.</td><td><a href=http://dx.doi.org/10.1038/1695>10.1038/1695</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR42</td><td>BibArticle</td><td>Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004;32(5):1792&#8211;7. %@ 0305&#8211;1048.</td><td><a href=http://dx.doi.org/10.1093/nar/gkh340>10.1093/nar/gkh340</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR43</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Tamura</span> <span style='background:#DDDDDD'>K</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Stecher</span> <span style='background:#DDDDDD'>G</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Peterson</span> <span style='background:#DDDDDD'>D</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Filipski</span> <span style='background:#DDDDDD'>A</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Kumar</span> <span style='background:#DDDDDD'>S</span></span></aug>: MEGA6: molecular evolutionary genetics analysis version 6.0. Molecular biology and evolution <span style='background:#66FF66'>2013</span>:mst197 %@ <span style='background:#D279FF'>0737&#8211;4038</span></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR44</td><td>BibArticle</td><td>Ma J, Bennetzen JL. Rapid recent growth and divergence of rice nuclear genomes. Proc Natl Acad Sci U S A. 2004;101(34):12404&#8211;10. %@ 10027&#8211;18424.</td><td><a href=http://dx.doi.org/10.1073/pnas.0403715101>10.1073/pnas.0403715101</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR45</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Milne</span> <span style='background:#DDDDDD'>I</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bayer</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Cardle</span> <span style='background:#DDDDDD'>L</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Shaw</span> <span style='background:#DDDDDD'>P</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Stephen</span> <span style='background:#DDDDDD'>G</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Wright</span> <span style='background:#DDDDDD'>F</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Marshall</span> <span style='background:#DDDDDD'>D.</span></span></aug> <span style='background:#CCCCFF'>Tablet&#8212;next generation sequence assembly visualization</span>. <span style='background:#CCFF99'>Bioinformatics</span>. <span style='background:#66FF66'>2010</span>;26(3):401&#8211;2. %@ <span style='background:#D279FF'>1367&#8211;4803</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR46</td><td>BibArticle</td><td>Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B. Mapping and quantifying mammalian transcriptomes by RNA-Seq. Nat Methods. 2008;5(7):621&#8211;8. %@ 1548&#8211;7091.</td><td><a href=http://dx.doi.org/10.1038/nmeth.1226>10.1038/nmeth.1226</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR47</td><td>BibArticle</td><td>Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, McWilliam H, Valentin F, Wallace IM, Wilm A, Lopez R. Clustal W and Clustal X version 2.0. Bioinformatics. 2007;23(21):2947&#8211;8. %@ 1367&#8211;4803.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btm404>10.1093/bioinformatics/btm404</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR48</td><td>BibArticle</td><td>Suyama M, Torrents D, Bork P. PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments. Nucleic Acids Res. 2006;34 suppl 2:W609&#8211;12. %@ 0305&#8211;1048.</td><td><a href=http://dx.doi.org/10.1093/nar/gkl315>10.1093/nar/gkl315</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR49</td><td>BibArticle</td><td>Wang D, Zhang Y, Zhang Z, Zhu J, Yu J. KaKs_Calculator 2.0: a toolkit incorporating gamma-series methods and sliding window strategies. Genomics Proteomics Bioinformatics. 2010;8(1):77&#8211;80. %@ 1672&#8211;0229.</td><td><a href=http://dx.doi.org/10.1016/S1672-0229(10)60008-3>10.1016/S1672-0229(10)60008-3</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR50</td><td>BibArticle</td><td>Yang Z, Nielsen R. Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models. Mol Biol Evol. 2000;17(1):32&#8211;&#8211;43. %@ 0737&#8211;4038.</td><td><a href=http://dx.doi.org/10.1093/oxfordjournals.molbev.a026236>10.1093/oxfordjournals.molbev.a026236</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR51</td><td>BibBook</td><td>F DM. Agricolae: statistical procedures for agricultural research.R package version 1. 2014.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR52</td><td>BibArticle</td><td>Sullivan MJ, Petty NK, Beatson SA. Easyfig: a genome comparison visualizer. Bioinformatics. 2011;27(7):1009&#8211;10. %@ 1367&#8211;4803.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/btr039>10.1093/bioinformatics/btr039</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR53</td><td>BibArticle</td><td>Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC, Haussler D, Miller W. Human&#8211;mouse alignments with BLASTZ. Genome Res. 2003;13(1):103&#8211;7. %@ 1088&#8211;9051.</td><td><a href=http://dx.doi.org/10.1101/gr.809403>10.1101/gr.809403</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR54</td><td>BibArticle</td><td>Carver TJ, Rutherford KM, Berriman M, Rajandream M-A, Barrell BG, Parkhill J. ACT: the Artemis comparison tool. Bioinformatics. 2005;21(16):3422&#8211;3. %@ 1367&#8211;4803.</td><td><a href=http://dx.doi.org/10.1093/bioinformatics/bti553>10.1093/bioinformatics/bti553</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR55</td><td>BibArticle</td><td>Darling ACE, Mau B, Blattner FR, Perna NT. Mauve: multiple alignment of conserved genomic sequence with rearrangements. Genome Res. 2004;14(7):1394&#8211;403. %@ 1088&#8211;9051.</td><td><a href=http://dx.doi.org/10.1101/gr.2289704>10.1101/gr.2289704</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR56</td><td>BibArticle</td><td>Tajima F. Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics. 1989;123(3):585&#8211;95. %@ 0016&#8211;6731.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR57</td><td>BibArticle</td><td>Ilic K, SanMiguel PJ, Bennetzen JL. A complex history of rearrangement in an orthologous region of the maize, sorghum, and rice genomes. Proc Natl Acad Sci. 2003;100(21):12265&#8211;70. %@ 10027&#8211;18424.</td><td><a href=http://dx.doi.org/10.1073/pnas.1434476100>10.1073/pnas.1434476100</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR58</td><td>BibArticle</td><td>Jannoo N, Grivet L, Chantret N, Garsmeur O, Glaszmann JC, Arruda P, D&#8217;Hont A. Orthologous comparison in a gene&#8208;rich region among grasses reveals stability in the sugarcane polyploid genome. Plant J. 2007;50(4):574&#8211;85. %@ 1365-1313X.</td><td><a href=http://dx.doi.org/10.1111/j.1365-313X.2007.03082.x>10.1111/j.1365-313X.2007.03082.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR59</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Aitken</span> <span style='background:#DDDDDD'>KS</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Jackson</span> <span style='background:#DDDDDD'>PA</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>McIntyre</span> <span style='background:#DDDDDD'>CL</span></span></aug>. <span style='background:#CCCCFF'>A combination of AFLP and SSR markers provides extensive map coverage and identification of homo (eo) logous linage groups in a sugarcane cultivar</span>. <span style='background:#CCFF99'>Theor Appl Genet</span>. <span style='background:#66FF66'>2005</span>;110(5):789&#8211;801. %@ <span style='background:#D279FF'>0040&#8211;5752</span>.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR60</td><td>BibBook</td><td>Ohno S. Evolution by gene duplicatio<Emphasis Type="Bold">n</Emphasis>. Berlin, New York: Springer-Verlag; 1970. p. 160.</td><td><a href=http://dx.doi.org/10.1007/978-3-642-86659-3>10.1007/978-3-642-86659-3</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR61</td><td>BibArticle</td><td>Prince VE, Pickett FB. Splitting pairs: the diverging fates of duplicated genes. Nat Rev Genet. 2002;3(11):827&#8211;37. %@ 1471&#8211;0056.</td><td><a href=http://dx.doi.org/10.1038/nrg928>10.1038/nrg928</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR62</td><td>BibArticle</td><td>Lynch M, Conery JS. The origins of genome complexity. Science. 2003;302(5649):1401&#8211;4. %@ 0036&#8211;8075.</td><td><a href=http://dx.doi.org/10.1126/science.1089370>10.1126/science.1089370</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR63</td><td>BibArticle</td><td>Flagel LE, Wendel JF. Gene duplication and evolutionary novelty in plants. New Phytologist. 2009;183(3):557&#8211;64. %@ 1469&#8211;8137.</td><td><a href=http://dx.doi.org/10.1111/j.1469-8137.2009.02923.x>10.1111/j.1469-8137.2009.02923.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR64</td><td>BibArticle</td><td>Blanc G, Wolfe KH. Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes. Plant Cell. 2004;16(7):1667&#8211;78. %@ 1532-1298X.</td><td><a href=http://dx.doi.org/10.1105/tpc.021345>10.1105/tpc.021345</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR65</td><td>BibArticle</td><td>Thomas BC, Pedersen B, Freeling M. Following tetraploidy in an Arabidopsis ancestor, genes were removed preferentially from one homeolog leaving clusters enriched in dose-sensitive genes. Genome Res. 2006;16(7):934&#8211;46. %@ 1088&#8211;9051.</td><td><a href=http://dx.doi.org/10.1101/gr.4708406>10.1101/gr.4708406</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR66</td><td>BibArticle</td><td>Freeling M. Bias in plant gene content following different sorts of duplication: tandem, whole-genome, segmental, or by transposition. Annu Rev Plant Biol. 2009;60:433&#8211;53. %@ 1543&#8211;5008.</td><td><a href=http://dx.doi.org/10.1146/annurev.arplant.043008.092122>10.1146/annurev.arplant.043008.092122</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR67</td><td>BibUnstructured</td><td><aug><span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Throude</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bolot</span> <span style='background:#DDDDDD'>S</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bosio</span> <span style='background:#DDDDDD'>M</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Pont</span> <span style='background:#DDDDDD'>C</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Sarda</span> <span style='background:#DDDDDD'>X</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Quraishi</span> <span style='background:#DDDDDD'>UM</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Bourgis</span> <span style='background:#DDDDDD'>F</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Lessard</span> <span style='background:#DDDDDD'>P</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Rogowsky</span> <span style='background:#DDDDDD'>P</span></span>, <span style='border:dotted windowtext 1.0pt;padding:0pt'><span style='background:#BCBCBC'>Ghesquiere</span> <span style='background:#DDDDDD'>A.</span></span></aug> Structure and expression analysis of rice paleo duplications. Nucleic acids research <span style='background:#66FF66'>2009</span>:gkn1048 %@ <span style='background:#D279FF'>0305&#8211;1048</span></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR68</td><td>BibArticle</td><td>Ozkan H, Levy AA, Feldman M. Allopolyploidy-induced rapid genome evolution in the wheat (Aegilops&#8211;Triticum) group. Plant Cell. 2001;13(8):1735&#8211;47. %@ 1532-1298X.</td><td><a href=http://dx.doi.org/10.1105/tpc.13.8.1735>10.1105/tpc.13.8.1735</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR69</td><td>BibArticle</td><td>Chantret N, Salse J, Sabot F, Rahman S, Bellec A, Laubin B, Dubois I, Dossat C, Sourdille P, Joudrier P. Molecular basis of evolutionary events that shaped the hardness locus in diploid and polyploid wheat species (Triticum and Aegilops). Plant Cell. 2005;17(4):1033&#8211;45. %@ 1532-1298X.</td><td><a href=http://dx.doi.org/10.1105/tpc.104.029181>10.1105/tpc.104.029181</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR70</td><td>BibArticle</td><td>Tate JA, Ni Z, Scheen A-C, Koh J, Gilbert CA, Lefkowitz D, Chen ZJ, Soltis PS, Soltis DE. Evolution and expression of homeologous loci in Tragopogon miscellus (Asteraceae), a recent and reciprocally formed allopolyploid. Genetics. 2006;173(3):1599&#8211;611. %@ 0016&#8211;6731.</td><td><a href=http://dx.doi.org/10.1534/genetics.106.057646>10.1534/genetics.106.057646</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR71</td><td>BibArticle</td><td>Buggs RJA, Chamala S, Wu WEI, Gao LU, May GD, Schnable PS, Soltis DE, Soltis PS, Barbazuk W. Characterization of duplicate gene evolution in the recent natural allopolyploid Tragopogon miscellus by next&#8208;generation sequencing and Sequenom iPLEX MassARRAY genotyping. Mol Ecol. 2010;19(s1):132&#8211;46. %@ 1365-1294X.</td><td><a href=http://dx.doi.org/10.1111/j.1365-294X.2009.04469.x>10.1111/j.1365-294X.2009.04469.x</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR72</td><td>BibArticle</td><td>Zhang J. Rates of conservative and radical nonsynonymous nucleotide substitutions in mammalian nuclear genes. J Mol Evol. 2000;50(1):56&#8211;68. %@ 0022&#8211;2844.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR73</td><td>BibArticle</td><td>Talamas E, Jackson L, Koeberl M, Jackson T, McElwee JL, Hawes NL, Chang B, Jablonski MM, Sidjanin DJ. Early transposable element insertion in intron 9 of the Hsf4 gene results in autosomal recessive cataracts in lop11 and ldis1 mice. Genomics. 2006;88(1):44&#8211;&#8211;51. %@ 0888&#8211;7543.</td><td><a href=http://dx.doi.org/10.1016/j.ygeno.2006.02.012>10.1016/j.ygeno.2006.02.012</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR74</td><td>BibArticle</td><td>Shimazaki M, Fujita K, Kobayashi H, Suzuki S. Pink-colored grape berry is the result of short insertion in intron of color regulatory gene. PLoS One. 2011;6(6):e21308. %@ 21932&#8211;26203.</td><td><a href=http://dx.doi.org/10.1371/journal.pone.0021308>10.1371/journal.pone.0021308</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR75</td><td>BibArticle</td><td>Babenko VN, Rogozin IB, Mekhedov SL, Koonin EV. Prevalence of intron gain over intron loss in the evolution of paralogous gene families. Nucleic Acids Res. 2004;32(12):3724&#8211;33. %@ 0305&#8211;1048.</td><td><a href=http://dx.doi.org/10.1093/nar/gkh686>10.1093/nar/gkh686</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR76</td><td>BibArticle</td><td>Chapman JA, Mascher M, Bulu&#231; A, Barry K, Georganas E, Session A, Strnadova V, Jenkins J, Sehgal S, Oliker L. A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome. Genome Biol. 2015;16(1):26.</td><td><a href=http://dx.doi.org/10.1186/s13059-015-0582-8>10.1186/s13059-015-0582-8</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR77</td><td>BibArticle</td><td>Ming R, Wai CM. Assembling allopolyploid genomes: no longer formidable. Genome Biol. 2015;16(1):27. %@ 1465&#8211;6906.</td><td><a href=http://dx.doi.org/10.1186/s13059-015-0585-5>10.1186/s13059-015-0585-5</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR78</td><td>BibBook</td><td>Burnquist WL, Sorrelles ME, Tanksley S. Characterization of genetic variability in Saccharum germplasm by means of restriction fragment length polymorphism (RFLP) analysis. 1995. p. 1995.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR79</td><td>BibArticle</td><td>Lu YH, D'Hont A, Walker DIT, Rao PS, Feldmann P, Glaszmann JC. Relationships among ancestral species of sugarcane revealed with RFLP using single copy maize nuclear probes. Euphytica. 1994;78(1&#8211;2):7&#8211;18. %@ 0014&#8211;2336.</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>CR80</td><td>BibArticle</td><td>Nair NV, Nair S, Sreenivasan TV, Mohan M. Analysis of genetic diversity and phylogeny in Saccharum and related genera using RAPD markers. Genet Resour Crop Ev. 1999;46(1):73&#8211;9. %@ 0925&#8211;9864.</td><td><a href=http://dx.doi.org/10.1023/A:1008696808645>10.1023/A:1008696808645</a></td><td>&nbsp;</td><td><b>SpringerLink</b></td>
<td align=center>---</td><td align=center>---</td></tr><tr><td>CR81</td><td>BibBook</td><td>Daniels J, Roach BT. Taxonomy and evolution. Amsterdam: Sugarcane improvement through breeding Elsevier; 1987. p. 7.</td><td><a href=http://dx.doi.org/10.1016/B978-0-444-42769-4.50007-2>10.1016/B978-0-444-42769-4.50007-2</a></td><td>&nbsp;</td><td><b>CrossRef-Meta</b></td>
<td>Author_FamilyName_2, BookTitle, Year, FirstPage, Author_FamilyName_1</td><td>CrossRef</td></tr></table>